[INFO ] 2021-11-01 17:00:48.404 [main] DataProviderCsvTest - AppendUsers test Success
[ERROR] 2021-11-01 17:00:48.413 [main] DataProviderCsvTest - AppendUsers test Success - Failure
[INFO ] 2021-11-01 17:02:55.979 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-01 17:14:23.113 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-14 16:58:22.775 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-14 17:02:08.421 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-14 18:21:49.589 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-14 18:21:49.734 [main] DataProviderCsvTest - getUserById test Success
[INFO ] 2021-11-14 18:21:49.755 [main] DataProviderCsvTest - null
[INFO ] 2021-11-14 18:25:44.114 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-14 18:25:44.284 [main] DataProviderCsvTest - getUserById test Success
[INFO ] 2021-11-14 18:25:44.301 [main] DataProviderCsvTest - null
[INFO ] 2021-11-21 13:56:45.544 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-21 13:56:45.553 [main] DataProviderCsv - createCsvWriter [start]
[INFO ] 2021-11-21 13:56:45.556 [main] DataProviderCsv - createCsvWriter [end]
[INFO ] 2021-11-21 13:56:45.695 [main] DataProviderCsvTest - getUserById test Success
[INFO ] 2021-11-21 13:56:45.722 [main] DataProviderCsvTest - null
[INFO ] 2021-11-21 13:59:03.063 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-21 13:59:03.073 [main] DataProviderCsv - createCsvWriter [start]
[INFO ] 2021-11-21 13:59:03.075 [main] DataProviderCsv - createCsvWriter [end]
[INFO ] 2021-11-21 14:00:55.337 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-21 14:00:55.346 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-21 14:00:55.349 [main] DataProviderCsv - createCsvWriter [start]
[INFO ] 2021-11-21 14:00:55.352 [main] DataProviderCsv - createCsvWriter [end]
[INFO ] 2021-11-21 14:00:55.443 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-21 14:24:35.201 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-21 14:24:35.229 [main] DataProviderCsvTest - getUserById test Success
[INFO ] 2021-11-21 14:24:35.356 [main] DataProviderCsvTest - null
[INFO ] 2021-11-21 14:27:01.683 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-21 14:27:01.708 [main] DataProviderCsvTest - getUserById test Success
[INFO ] 2021-11-21 14:27:01.823 [main] DataProviderCsvTest - Ivan
[INFO ] 2021-11-21 14:32:48.623 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-21 14:32:48.631 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-21 14:32:48.634 [main] DataProviderCsv - createCsvWriter [start]
[INFO ] 2021-11-21 14:32:48.635 [main] DataProviderCsv - createCsvWriter [end]
[INFO ] 2021-11-21 14:32:48.734 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-21 14:32:48.755 [main] DataProviderCsvTest - getUserById test Success
[INFO ] 2021-11-21 14:32:48.772 [main] DataProviderCsvTest - null
[INFO ] 2021-11-21 15:33:33.482 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-21 15:33:33.492 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-21 15:33:33.495 [main] DataProviderCsv - createCsvWriter [start]
[INFO ] 2021-11-21 15:33:33.497 [main] DataProviderCsv - createCsvWriter [end]
[INFO ] 2021-11-21 15:33:33.603 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-21 15:33:33.620 [main] DataProviderCsvTest - getUserById test Success
[INFO ] 2021-11-21 15:33:33.636 [main] DataProviderCsvTest - null
[INFO ] 2021-11-21 16:10:36.009 [main] DataProviderCsvTest - deleteUserById test Success
[ERROR] 2021-11-21 16:10:36.016 [main] DataProviderCsvTest - deleteUserById test Success - Failure
[INFO ] 2021-11-21 16:10:36.414 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-21 16:10:36.418 [main] DataProviderCsvTest - getUserById test Success
[INFO ] 2021-11-21 16:10:36.523 [main] DataProviderCsvTest - Ivan
[INFO ] 2021-11-21 16:12:13.313 [main] DataProviderCsvTest - deleteUserById test Success
[INFO ] 2021-11-21 16:12:13.337 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-21 16:12:13.342 [main] DataProviderCsvTest - getUserById test Success
[INFO ] 2021-11-21 16:12:13.462 [main] DataProviderCsvTest - Ivan
[INFO ] 2021-11-21 16:12:30.481 [main] DataProviderCsvTest - deleteUserById test Success
[ERROR] 2021-11-21 16:12:30.488 [main] DataProviderCsvTest - deleteUserById test Success - Failure
[INFO ] 2021-11-21 16:12:30.516 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-21 16:12:30.521 [main] DataProviderCsvTest - getUserById test Success
[INFO ] 2021-11-21 16:12:30.643 [main] DataProviderCsvTest - Ivan
[INFO ] 2021-11-21 16:12:55.132 [main] DataProviderCsvTest - deleteUserById test Success
[ERROR] 2021-11-21 16:12:55.141 [main] DataProviderCsvTest - deleteUserById test Success - Failure
[INFO ] 2021-11-21 16:26:17.152 [main] DataProviderCsvTest - deleteUserById test Success
[ERROR] 2021-11-21 16:26:17.160 [main] DataProviderCsvTest - deleteUserById test Success - Failure
[INFO ] 2021-11-21 16:26:31.775 [main] DataProviderCsvTest - deleteUserById test Success
[ERROR] 2021-11-21 16:26:31.791 [main] DataProviderCsvTest - deleteUserById test Success - Failure
[INFO ] 2021-11-21 16:26:49.799 [main] DataProviderCsvTest - deleteUserById test Success
[ERROR] 2021-11-21 16:27:19.620 [main] DataProviderCsvTest - deleteUserById test Success - Failure
[INFO ] 2021-11-21 16:27:23.757 [main] DataProviderCsvTest - deleteUserById test Success
[ERROR] 2021-11-21 16:28:06.163 [main] DataProviderCsvTest - deleteUserById test Success - Failure
[INFO ] 2021-11-21 16:29:41.629 [main] DataProviderCsvTest - deleteUserById test Success
[INFO ] 2021-11-21 16:30:11.874 [main] DataProviderCsvTest - deleteUserById test Success
[INFO ] 2021-11-27 12:52:19.879 [main] DataProviderCsvTest - deleteUserById test Success
[INFO ] 2021-11-27 12:52:19.914 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-27 12:52:19.915 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-27 12:52:19.919 [main] DataProviderCsv - createCsvWriter [start]
[INFO ] 2021-11-27 12:52:19.920 [main] DataProviderCsv - createCsvWriter [end]
[INFO ] 2021-11-27 12:52:20.021 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-27 12:52:20.024 [main] DataProviderCsvTest - getUserById test Success
[INFO ] 2021-11-27 12:52:20.041 [main] DataProviderCsvTest - null
[INFO ] 2021-11-27 12:52:54.048 [main] DataProviderCsvTest - deleteUserById test Success
[INFO ] 2021-11-27 12:53:55.965 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-27 12:53:55.967 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-27 12:53:55.969 [main] DataProviderCsv - createCsvWriter [start]
[INFO ] 2021-11-27 12:53:55.969 [main] DataProviderCsv - createCsvWriter [end]
[INFO ] 2021-11-27 12:53:56.109 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-27 12:53:56.111 [main] DataProviderCsvTest - getUserById test Success
[INFO ] 2021-11-27 12:53:56.129 [main] DataProviderCsvTest - null
[INFO ] 2021-11-27 12:54:03.208 [main] DataProviderCsvTest - deleteUserById test Success
[INFO ] 2021-11-27 12:54:03.250 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-27 12:54:13.721 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-27 12:54:13.725 [main] DataProviderCsv - createCsvWriter [start]
[INFO ] 2021-11-27 12:54:13.726 [main] DataProviderCsv - createCsvWriter [end]
[INFO ] 2021-11-27 12:54:28.360 [main] DataProviderCsvTest - deleteUserById test Success
[INFO ] 2021-11-27 12:54:28.409 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-27 12:54:57.198 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-27 12:54:57.202 [main] DataProviderCsv - createCsvWriter [start]
[INFO ] 2021-11-27 12:54:57.203 [main] DataProviderCsv - createCsvWriter [end]
[INFO ] 2021-11-27 12:55:11.171 [main] DataProviderCsvTest - deleteUserById test Success
[INFO ] 2021-11-27 12:55:11.219 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-27 12:55:38.337 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-27 12:55:47.135 [main] DataProviderCsv - createCsvWriter [start]
[INFO ] 2021-11-27 12:55:47.137 [main] DataProviderCsv - createCsvWriter [end]
[INFO ] 2021-11-27 12:56:26.317 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-27 12:56:30.322 [main] DataProviderCsvTest - getUserById test Success
[INFO ] 2021-11-27 12:56:30.349 [main] DataProviderCsvTest - null
[INFO ] 2021-11-27 13:04:39.620 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-27 13:04:39.627 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-27 13:04:39.721 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-27 13:16:19.392 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-27 13:16:19.400 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-27 13:16:19.500 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-27 13:17:15.622 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-27 13:17:15.632 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-27 13:17:15.737 [main] DataProviderCsv - appendUsers success
[INFO ] 2021-11-27 13:17:15.738 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-27 13:18:27.743 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-27 13:18:27.752 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-27 13:18:27.853 [main] DataProviderCsv - appendUsers success
[INFO ] 2021-11-27 13:18:27.854 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-27 13:21:33.073 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-27 13:21:33.082 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-27 13:21:33.183 [main] DataProviderCsv - appendUsers success
[INFO ] 2021-11-27 13:21:33.183 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-27 13:24:48.458 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-27 13:24:48.465 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-27 13:24:48.564 [main] DataProviderCsv - appendUsers success
[INFO ] 2021-11-27 13:24:48.564 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-27 13:59:19.114 [main] DataProviderCsvTest - getUserById test Success
[INFO ] 2021-11-27 13:59:19.123 [main] DataProviderCsv - getUserById [start]
[INFO ] 2021-11-27 13:59:19.295 [main] DataProviderCsv - getUserById [end]
[INFO ] 2021-11-27 13:59:19.295 [main] DataProviderCsvTest - Ivan
[INFO ] 2021-11-27 14:02:25.863 [main] DataProviderCsvTest - getUserById test Success
[INFO ] 2021-11-27 14:02:25.871 [main] DataProviderCsv - getUserById [start]
[INFO ] 2021-11-27 14:02:25.975 [main] DataProviderCsv - getUserById [end]
[INFO ] 2021-11-27 14:02:25.977 [main] DataProviderCsvTest - User(id=1, name=Ivan)
[INFO ] 2021-11-27 14:03:01.106 [main] DataProviderCsvTest - getUserById test Success
[INFO ] 2021-11-27 14:03:01.116 [main] DataProviderCsv - getUserById [start]
[INFO ] 2021-11-27 14:03:01.234 [main] DataProviderCsv - getUserById [end]
[INFO ] 2021-11-27 14:03:01.238 [main] DataProviderCsvTest - Result{message='null', entity=User(id=1, name=Ivan), code=SUCCESS}
[INFO ] 2021-11-27 14:30:42.907 [main] DataProviderCsvTest - getUserById test Fail
[INFO ] 2021-11-27 14:30:42.915 [main] DataProviderCsv - getUserById [start]
[INFO ] 2021-11-27 14:30:43.038 [main] DataProviderCsv - getUserById [end]
[INFO ] 2021-11-27 14:30:43.042 [main] DataProviderCsvTest - User(id=0, name=null)
[INFO ] 2021-11-27 14:31:42.705 [main] DataProviderCsvTest - getUserById test Fail
[INFO ] 2021-11-27 14:31:42.717 [main] DataProviderCsvTest - User(id=0, name=null)
[INFO ] 2021-11-27 14:31:42.717 [main] DataProviderCsv - getUserById [start]
[INFO ] 2021-11-27 14:31:42.856 [main] DataProviderCsv - getUserById [end]
[INFO ] 2021-11-27 14:31:42.856 [main] DataProviderCsvTest - User(id=0, name=null)
[INFO ] 2021-11-27 14:33:13.206 [main] DataProviderCsvTest - getUserById test Fail
[INFO ] 2021-11-27 14:33:13.216 [main] DataProviderCsv - getUserById [start]
[INFO ] 2021-11-27 14:33:13.351 [main] DataProviderCsv - getUserById [end]
[INFO ] 2021-11-27 14:33:13.355 [main] DataProviderCsvTest - User(id=0, name=Empty)
[INFO ] 2021-11-27 14:39:12.192 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-27 14:39:12.198 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-27 14:39:12.291 [main] DataProviderCsv - appendUsers success
[INFO ] 2021-11-27 14:39:12.291 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-27 14:40:24.906 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-27 14:40:24.918 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-27 14:40:24.946 [main] DataProviderCsv - [User(id=1, name=Ivan), User(id=2, name=Maria)]
[INFO ] 2021-11-27 14:40:25.046 [main] DataProviderCsv - appendUsers success
[INFO ] 2021-11-27 14:40:25.046 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-27 14:41:00.881 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-27 14:41:00.889 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-27 14:41:00.910 [main] DataProviderCsv - com.opencsv.bean.StatefulBeanToCsv@4f9a2c08
[INFO ] 2021-11-27 14:41:00.982 [main] DataProviderCsv - appendUsers success
[INFO ] 2021-11-27 14:41:00.982 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-27 14:42:02.436 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-27 14:42:02.444 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-27 14:42:02.449 [main] DataProviderCsv - ./src/main/resources/data/csv/user.csv
[INFO ] 2021-11-27 14:42:02.549 [main] DataProviderCsv - appendUsers success
[INFO ] 2021-11-27 14:42:02.550 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-27 14:44:15.764 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-27 14:44:15.774 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-27 14:44:15.899 [main] DataProviderCsv - appendUsers success
[INFO ] 2021-11-27 14:44:15.899 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-27 14:47:19.563 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-27 14:47:19.575 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-27 14:47:19.679 [main] DataProviderCsv - appendUsers success
[INFO ] 2021-11-27 14:47:19.680 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-27 14:47:32.799 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-27 14:47:32.807 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-27 14:47:32.916 [main] DataProviderCsv - appendUsers success
[INFO ] 2021-11-27 14:47:32.917 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-27 14:55:38.639 [main] DataProviderCsvTest - getUserById test Success
[INFO ] 2021-11-27 14:55:38.650 [main] DataProviderCsv - getUserById [start]
[INFO ] 2021-11-27 14:55:38.757 [main] DataProviderCsv - getUserById [end]
[INFO ] 2021-11-27 14:55:38.759 [main] DataProviderCsvTest - User(id=2, name=Maria)
[INFO ] 2021-11-27 14:59:05.562 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-27 14:59:05.568 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-27 14:59:05.669 [main] DataProviderCsv - appendUsers created:[User(id=1, name=Ivan), User(id=2, name=Maria)]
[INFO ] 2021-11-27 14:59:05.670 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-27 14:59:43.494 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-27 14:59:43.500 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-27 14:59:43.597 [main] DataProviderCsv - appendUsers created:[User(id=1, name=Ivan), User(id=2, name=Maria)]
[INFO ] 2021-11-27 14:59:43.598 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-27 14:59:57.489 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-27 14:59:57.497 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-27 14:59:57.610 [main] DataProviderCsv - appendUsers created:[User(id=1, name=Ivan), User(id=2, name=Maria)]
[INFO ] 2021-11-27 14:59:57.611 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-27 15:01:50.617 [main] DataProviderCsvTest - AppendUsers test Fail
[INFO ] 2021-11-27 15:01:50.625 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-27 15:01:50.653 [main] DataProviderCsv - appendUsers created:[]
[INFO ] 2021-11-27 15:01:50.654 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-27 15:02:33.693 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-27 15:02:33.700 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-27 15:02:33.795 [main] DataProviderCsv - appendUsers created:[User(id=1, name=Ivan), User(id=2, name=Maria)]
[INFO ] 2021-11-27 15:02:33.795 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-27 15:02:40.935 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-27 15:02:40.944 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-27 15:02:41.064 [main] DataProviderCsv - appendUsers created:[User(id=1, name=Ivan), User(id=2, name=Maria)]
[INFO ] 2021-11-27 15:02:41.065 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-27 15:03:00.605 [main] DataProviderCsvTest - AppendUsers test Fail
[INFO ] 2021-11-27 15:03:00.613 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-27 15:03:00.641 [main] DataProviderCsv - appendUsers created:[]
[INFO ] 2021-11-27 15:03:00.641 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-27 15:05:47.752 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-27 15:05:47.759 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-27 15:05:47.869 [main] DataProviderCsv - appendUsers created:[User(id=1, name=Ivan), User(id=2, name=Maria)]
[INFO ] 2021-11-27 15:05:47.870 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-27 15:05:53.308 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-27 15:05:53.316 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-27 15:05:53.441 [main] DataProviderCsv - appendUsers created:[User(id=1, name=Ivan), User(id=2, name=Maria)]
[INFO ] 2021-11-27 15:05:53.442 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-27 15:05:59.393 [main] DataProviderCsvTest - AppendUsers test Fail
[INFO ] 2021-11-27 15:05:59.401 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-27 15:19:04.237 [main] DataProviderCsvTest - deleteUserById test Success
[INFO ] 2021-11-27 15:20:29.463 [main] DataProviderCsvTest - deleteUserById test Success
[INFO ] 2021-11-27 15:21:49.830 [main] DataProviderCsvTest - deleteUserById test Success
[INFO ] 2021-11-27 15:24:29.720 [main] DataProviderCsvTest - deleteUserById test Success
[INFO ] 2021-11-27 15:27:25.468 [main] DataProviderCsvTest - deleteUserById test Success
[ERROR] 2021-11-27 15:27:25.482 [main] DataProviderCsvTest - deleteUserById test Success - Failure
[INFO ] 2021-11-27 15:28:17.645 [main] DataProviderCsvTest - deleteUserById test Success
[INFO ] 2021-11-27 16:07:51.913 [main] DataProviderCsvTest - deleteUserById test Success
[INFO ] 2021-11-27 16:07:51.923 [main] DataProviderCsv - ./src/main/resources/data/csv/newuser.csv
[INFO ] 2021-11-27 16:10:44.891 [main] DataProviderCsvTest - deleteUserById test Success
[INFO ] 2021-11-27 16:13:16.330 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-27 16:13:16.341 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-27 16:13:16.467 [main] DataProviderCsv - appendUsers created:[User(id=1, name=Ivan), User(id=2, name=Maria)]
[INFO ] 2021-11-27 16:13:16.468 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-27 16:13:27.604 [main] DataProviderCsvTest - deleteUserById test Success
[INFO ] 2021-11-27 16:13:43.075 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-27 16:13:43.084 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-27 16:13:43.189 [main] DataProviderCsv - appendUsers created:[User(id=1, name=Ivan), User(id=2, name=Maria)]
[INFO ] 2021-11-27 16:13:43.190 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-27 16:13:56.269 [main] DataProviderCsvTest - deleteUserById test Success
[INFO ] 2021-11-27 16:28:27.811 [main] DataProviderCsvTest - deleteUserById test Success
[INFO ] 2021-11-27 16:28:27.821 [main] DataProviderCsv - deleteUserById [start] id:2
[INFO ] 2021-11-27 16:28:27.841 [main] DataProviderCsv - ./src/main/resources/data/csv/newuser.csv has been written and renamed to ./src/main/resources/data/csv/user.csv
[INFO ] 2021-11-27 16:28:27.841 [main] DataProviderCsv - ./src/main/resources/data/csv/user.csv has been deleted
[INFO ] 2021-11-27 16:32:11.761 [main] DataProviderCsvTest - deleteUserById test Fail
[INFO ] 2021-11-27 16:32:11.771 [main] DataProviderCsv - deleteUserById [start] id:2
[INFO ] 2021-11-27 16:32:11.788 [main] DataProviderCsv - ./src/main/resources/data/csv/user.csv has been deleted
[INFO ] 2021-11-27 16:32:11.790 [main] DataProviderCsv - ./src/main/resources/data/csv/newuser.csv has been written and renamed to ./src/main/resources/data/csv/user.csv
[INFO ] 2021-11-28 15:47:51.882 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-28 15:47:51.890 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-28 15:47:52.014 [main] DataProviderCsv - appendUsers created:[User(id=1, name=Ivan), User(id=2, name=Maria)]
[INFO ] 2021-11-28 15:47:52.014 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-28 15:50:53.979 [main] DataProviderCsvTest - updateUser test Success
[INFO ] 2021-11-28 15:50:53.992 [main] DataProviderCsv - updateUser [start] user to update User(id=1, name=Boris)
[INFO ] 2021-11-28 15:50:53.993 [main] DataProviderCsv - getUserById [start] id:1
[INFO ] 2021-11-28 15:50:54.135 [main] DataProviderCsv - getUserById [end]
[INFO ] 2021-11-28 15:50:54.135 [main] DataProviderCsv - Entry found by given id Result{message='null', entity=User(id=1, name=Ivan), code=SUCCESS}
[INFO ] 2021-11-28 15:50:54.136 [main] DataProviderCsv - deleteUserById [start] id:1
[INFO ] 2021-11-28 15:50:54.140 [main] DataProviderCsv - ./src/main/resources/data/csv/user.csv has been deleted
[INFO ] 2021-11-28 15:50:54.141 [main] DataProviderCsv - ./src/main/resources/data/csv/newuser.csv has been written and renamed to ./src/main/resources/data/csv/user.csv
[INFO ] 2021-11-28 15:50:54.142 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-28 15:50:54.153 [main] DataProviderCsv - appendUsers created:[User(id=1, name=Boris)]
[INFO ] 2021-11-28 15:50:54.154 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-28 15:50:54.154 [main] DataProviderCsv - Entry successfully updated
[INFO ] 2021-11-28 15:51:33.275 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-28 15:51:33.285 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-28 15:51:33.399 [main] DataProviderCsv - appendUsers created:[User(id=1, name=Ivan), User(id=2, name=Maria)]
[INFO ] 2021-11-28 15:51:33.399 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-28 15:52:54.205 [main] DataProviderCsvTest - updateUser test Success
[INFO ] 2021-11-28 15:52:54.221 [main] DataProviderCsv - updateUser [start] user to update User(id=1, name=Boris)
[INFO ] 2021-11-28 15:52:54.222 [main] DataProviderCsv - getUserById [start] id:1
[INFO ] 2021-11-28 15:52:54.341 [main] DataProviderCsv - getUserById [end]
[INFO ] 2021-11-28 15:52:54.341 [main] DataProviderCsv - Entry found by given id Result{message='null', entity=User(id=1, name=Ivan), code=SUCCESS}
[INFO ] 2021-11-28 15:52:54.342 [main] DataProviderCsv - deleteUserById [start] id:1
[INFO ] 2021-11-28 15:52:54.346 [main] DataProviderCsv - ./src/main/resources/data/csv/user.csv has been deleted
[INFO ] 2021-11-28 15:52:54.348 [main] DataProviderCsv - ./src/main/resources/data/csv/newuser.csv has been written and renamed to ./src/main/resources/data/csv/user.csv
[INFO ] 2021-11-28 15:52:54.348 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-28 15:52:54.362 [main] DataProviderCsv - appendUsers created:[User(id=1, name=Boris)]
[INFO ] 2021-11-28 15:52:54.363 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-28 15:52:54.363 [main] DataProviderCsv - Entry successfully updated
[INFO ] 2021-11-28 15:53:07.259 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-28 15:53:07.266 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-28 15:53:07.374 [main] DataProviderCsv - appendUsers created:[User(id=1, name=Ivan), User(id=2, name=Maria)]
[INFO ] 2021-11-28 15:53:07.374 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-28 15:57:13.298 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-28 15:57:13.310 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-28 15:57:13.431 [main] DataProviderCsv - appendUsers created:[User(id=1, name=Ivan), User(id=2, name=Maria), User(id=3, name=Dima)]
[INFO ] 2021-11-28 15:57:13.432 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-28 15:57:25.857 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-28 15:57:25.863 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-28 15:57:25.969 [main] DataProviderCsv - appendUsers created:[User(id=1, name=Ivan), User(id=2, name=Maria)]
[INFO ] 2021-11-28 15:57:25.970 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-28 16:32:15.907 [main] DataProviderCsvTest - AppendUser test Success
[INFO ] 2021-11-28 16:32:15.923 [main] DataProviderCsv - appendUser [start] appending user User(id=3, name=Dmitriy)
[INFO ] 2021-11-28 16:32:15.923 [main] DataProviderCsv - getUserById [start] id:3
[INFO ] 2021-11-28 16:32:16.049 [main] DataProviderCsv - getUserById [end]
[INFO ] 2021-11-28 16:32:16.058 [main] DataProviderCsv - New user successfully appended User(id=3, name=Dmitriy)
[INFO ] 2021-11-28 16:32:37.888 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-28 16:32:37.897 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-28 16:32:38.004 [main] DataProviderCsv - appendUsers created:[User(id=1, name=Ivan), User(id=2, name=Maria), User(id=3, name=Dmitriy)]
[INFO ] 2021-11-28 16:32:38.005 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-28 16:32:59.260 [main] DataProviderCsvTest - AppendUser test Success
[INFO ] 2021-11-28 16:32:59.279 [main] DataProviderCsv - appendUser [start] appending user User(id=4, name=Daria)
[INFO ] 2021-11-28 16:32:59.279 [main] DataProviderCsv - getUserById [start] id:4
[INFO ] 2021-11-28 16:32:59.412 [main] DataProviderCsv - getUserById [end]
[INFO ] 2021-11-28 16:32:59.421 [main] DataProviderCsv - New user successfully appended User(id=4, name=Daria)
[INFO ] 2021-11-28 17:05:02.619 [main] DataProviderCsvTest - AppendUser test Fail
[INFO ] 2021-11-28 17:05:02.633 [main] DataProviderCsv - appendUser [start] appending user User(id=0, name=null)
[INFO ] 2021-11-28 17:05:02.633 [main] DataProviderCsv - Empty user
[INFO ] 2021-11-28 17:06:42.257 [main] DataProviderCsvTest - updateUser test Success
[INFO ] 2021-11-28 17:06:42.272 [main] DataProviderCsv - updateUser [start] user to update User(id=0, name=null)
[INFO ] 2021-11-28 17:06:42.272 [main] DataProviderCsv - getUserById [start] id:0
[ERROR] 2021-11-28 17:06:42.423 [main] DataProviderCsv - getUserById : Error parsing CSV line: 5. [ID,NAME]
[INFO ] 2021-11-28 17:06:42.426 [main] DataProviderCsv - getUserById [end]
[INFO ] 2021-11-28 17:06:42.428 [main] DataProviderCsv - Entry found by given id Result{message='Error parsing CSV line: 5. [ID,NAME]', entity=null, code=SUCCESS}
[INFO ] 2021-11-28 17:06:42.428 [main] DataProviderCsv - No matching entry found
[INFO ] 2021-11-28 17:07:51.125 [main] DataProviderCsvTest - updateUser test Success
[INFO ] 2021-11-28 17:07:51.140 [main] DataProviderCsv - updateUser [start] user to update User(id=0, name=null)
[INFO ] 2021-11-28 17:07:51.141 [main] DataProviderCsv - Empty user
[INFO ] 2021-11-29 10:01:16.757 [main] DataProviderCsvTest - AppendUser test Success
[INFO ] 2021-11-29 10:01:16.782 [main] DataProviderCsv - appendUser [start] appending user User(id=4, name=Daria)
[INFO ] 2021-11-29 10:01:16.783 [main] DataProviderCsv - getUserById [start] id:4
[ERROR] 2021-11-29 10:01:16.986 [main] DataProviderCsv - getUserById : Error parsing CSV line: 5. [ID,NAME]
[INFO ] 2021-11-29 10:01:16.988 [main] DataProviderCsv - getUserById [end]
[INFO ] 2021-11-29 10:01:16.997 [main] DataProviderCsv - append User [end] appended User(id=4, name=Daria)
[INFO ] 2021-11-29 10:02:55.539 [main] DataProviderCsvTest - AppendUser test Success
[INFO ] 2021-11-29 10:02:55.559 [main] DataProviderCsv - appendUser [start] appending user User(id=4, name=Daria)
[INFO ] 2021-11-29 10:02:55.560 [main] DataProviderCsv - getUserById [start] id:4
[ERROR] 2021-11-29 10:02:55.714 [main] DataProviderCsv - getUserById : Error parsing CSV line: 5. []
[INFO ] 2021-11-29 10:02:55.719 [main] DataProviderCsv - getUserById [end]
[INFO ] 2021-11-29 10:02:55.732 [main] DataProviderCsv - append User [end] appended User(id=4, name=Daria)
[INFO ] 2021-11-29 10:24:46.919 [main] DataProviderCsvTest - AppendUser test Success
[INFO ] 2021-11-29 10:24:46.937 [main] DataProviderCsv - appendUser [start] appending user User(id=4, name=Daria)
[INFO ] 2021-11-29 10:24:46.937 [main] DataProviderCsv - getUserById [start] id:4
[ERROR] 2021-11-29 10:24:47.087 [main] DataProviderCsv - getUserById : Error parsing CSV line: 5. []
[INFO ] 2021-11-29 10:24:47.089 [main] DataProviderCsv - getUserById [end]
[INFO ] 2021-11-29 10:24:47.099 [main] DataProviderCsv - append User [end] appended User(id=4, name=Daria)
[INFO ] 2021-11-29 10:30:13.065 [main] DataProviderCsvTest - AppendUser test Success
[INFO ] 2021-11-29 10:30:13.084 [main] DataProviderCsv - appendUser [start] appending user User(id=4, name=Daria)
[INFO ] 2021-11-29 10:30:13.084 [main] DataProviderCsv - getUserById [start] id:4
[ERROR] 2021-11-29 10:30:13.253 [main] DataProviderCsv - getUserById : Error parsing CSV line: 1. [ID,NAME]
[INFO ] 2021-11-29 10:30:13.255 [main] DataProviderCsv - getUserById [end]
[INFO ] 2021-11-29 10:30:13.267 [main] DataProviderCsv - append User [end] appended User(id=4, name=Daria)
[INFO ] 2021-11-29 10:31:08.212 [main] DataProviderCsvTest - AppendUser test Success
[INFO ] 2021-11-29 10:31:08.228 [main] DataProviderCsv - appendUser [start] appending user User(id=4, name=Daria)
[INFO ] 2021-11-29 10:31:08.228 [main] DataProviderCsv - getUserById [start] id:4
[INFO ] 2021-11-29 10:31:08.357 [main] DataProviderCsv - getUserById [end]
[INFO ] 2021-11-29 10:31:08.357 [main] DataProviderCsv - Entry already exists Result{message='null', entity=User(id=4, name=Daria), code=SUCCESS}
[INFO ] 2021-11-29 10:31:27.038 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-29 10:31:27.048 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-29 10:31:27.183 [main] DataProviderCsv - appendUsers created:[User(id=1, name=Ivan), User(id=2, name=Maria), User(id=3, name=Dmitriy)]
[INFO ] 2021-11-29 10:31:27.184 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-29 10:31:38.781 [main] DataProviderCsvTest - AppendUser test Success
[INFO ] 2021-11-29 10:31:38.799 [main] DataProviderCsv - appendUser [start] appending user User(id=4, name=Daria)
[INFO ] 2021-11-29 10:31:38.799 [main] DataProviderCsv - getUserById [start] id:4
[INFO ] 2021-11-29 10:31:38.944 [main] DataProviderCsv - getUserById [end]
[INFO ] 2021-11-29 10:31:38.957 [main] DataProviderCsv - append User [end] appended User(id=4, name=Daria)
[INFO ] 2021-11-29 10:32:02.108 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-29 10:32:02.118 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-29 10:32:02.253 [main] DataProviderCsv - appendUsers created:[User(id=1, name=Ivan), User(id=2, name=Maria), User(id=3, name=Dmitriy)]
[INFO ] 2021-11-29 10:32:02.254 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-29 10:32:21.748 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-29 10:32:21.758 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-29 10:32:21.896 [main] DataProviderCsv - appendUsers created:[User(id=1, name=Ivan), User(id=2, name=Maria), User(id=3, name=Dmitriy)]
[INFO ] 2021-11-29 10:32:21.896 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-29 10:32:39.784 [main] DataProviderCsvTest - getUserById test Success
[INFO ] 2021-11-29 10:32:39.800 [main] DataProviderCsv - getUserById [start] id:2
[INFO ] 2021-11-29 10:32:39.932 [main] DataProviderCsv - getUserById [end]
[INFO ] 2021-11-29 10:32:39.937 [main] DataProviderCsvTest - User(id=2, name=Maria)
[INFO ] 2021-11-29 10:33:01.653 [main] DataProviderCsvTest - deleteUserById test Success
[INFO ] 2021-11-29 10:33:01.661 [main] DataProviderCsv - deleteUserById [start] id:1
[INFO ] 2021-11-29 10:33:01.683 [main] DataProviderCsv - ./src/main/resources/data/csv/user.csv has been deleted
[INFO ] 2021-11-29 10:33:01.684 [main] DataProviderCsv - ./src/main/resources/data/csv/newuser.csv has been written and renamed to ./src/main/resources/data/csv/user.csv
[INFO ] 2021-11-29 10:33:18.624 [main] DataProviderCsvTest - updateUser test Success
[INFO ] 2021-11-29 10:33:18.641 [main] DataProviderCsv - updateUser [start] user to update User(id=2, name=Boris)
[INFO ] 2021-11-29 10:33:18.642 [main] DataProviderCsv - getUserById [start] id:2
[INFO ] 2021-11-29 10:33:18.767 [main] DataProviderCsv - getUserById [end]
[INFO ] 2021-11-29 10:33:18.768 [main] DataProviderCsv - Entry found by given id Result{message='null', entity=User(id=2, name=Maria), code=SUCCESS}
[INFO ] 2021-11-29 10:33:18.768 [main] DataProviderCsv - deleteUserById [start] id:2
[INFO ] 2021-11-29 10:33:18.773 [main] DataProviderCsv - ./src/main/resources/data/csv/user.csv has been deleted
[INFO ] 2021-11-29 10:33:18.775 [main] DataProviderCsv - ./src/main/resources/data/csv/newuser.csv has been written and renamed to ./src/main/resources/data/csv/user.csv
[INFO ] 2021-11-29 10:33:18.775 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-29 10:33:18.792 [main] DataProviderCsv - appendUsers created:[User(id=2, name=Boris)]
[INFO ] 2021-11-29 10:33:18.793 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-29 10:33:18.793 [main] DataProviderCsv - Entry successfully updated
[INFO ] 2021-11-29 10:33:37.159 [main] DataProviderCsvTest - AppendUser test Success
[INFO ] 2021-11-29 10:33:37.174 [main] DataProviderCsv - appendUser [start] appending user User(id=4, name=Daria)
[INFO ] 2021-11-29 10:33:37.174 [main] DataProviderCsv - getUserById [start] id:4
[INFO ] 2021-11-29 10:33:37.330 [main] DataProviderCsv - getUserById [end]
[INFO ] 2021-11-29 10:33:37.342 [main] DataProviderCsv - append User [end] appended User(id=4, name=Daria)
[INFO ] 2021-11-29 10:34:37.152 [main] DataProviderCsvTest - AppendUser test Fail
[INFO ] 2021-11-29 10:34:37.175 [main] DataProviderCsv - appendUser [start] appending user User(id=0, name=null)
[INFO ] 2021-11-29 10:34:37.176 [main] DataProviderCsv - Empty user
[INFO ] 2021-11-29 10:34:53.098 [main] DataProviderCsvTest - getUserById test Fail
[INFO ] 2021-11-29 10:34:53.115 [main] DataProviderCsv - getUserById [start] id:5
[INFO ] 2021-11-29 10:34:53.295 [main] DataProviderCsv - getUserById [end]
[INFO ] 2021-11-29 10:34:53.299 [main] DataProviderCsvTest - User(id=0, name=Empty)
[INFO ] 2021-11-29 10:35:15.504 [main] DataProviderCsvTest - deleteUserById test Fail
[INFO ] 2021-11-29 10:35:15.516 [main] DataProviderCsv - deleteUserById [start] id:2
[INFO ] 2021-11-29 10:35:15.537 [main] DataProviderCsv - ./src/main/resources/data/csv/user.csv has been deleted
[INFO ] 2021-11-29 10:35:15.539 [main] DataProviderCsv - ./src/main/resources/data/csv/newuser.csv has been written and renamed to ./src/main/resources/data/csv/user.csv
[INFO ] 2021-11-29 10:35:42.100 [main] DataProviderCsvTest - deleteUserById test Fail
[INFO ] 2021-11-29 10:35:42.117 [main] DataProviderCsv - deleteUserById [start] id:2
[INFO ] 2021-11-29 10:35:42.140 [main] DataProviderCsv - ./src/main/resources/data/csv/user.csv has been deleted
[INFO ] 2021-11-29 10:35:42.141 [main] DataProviderCsv - ./src/main/resources/data/csv/newuser.csv has been written and renamed to ./src/main/resources/data/csv/user.csv
[INFO ] 2021-11-29 10:36:00.229 [main] DataProviderCsvTest - updateUser test Success
[INFO ] 2021-11-29 10:36:00.249 [main] DataProviderCsv - updateUser [start] user to update User(id=0, name=null)
[INFO ] 2021-11-29 10:36:00.250 [main] DataProviderCsv - Empty user
[INFO ] 2021-11-29 10:36:24.836 [main] DataProviderCsvTest - AppendUsers test Success
[INFO ] 2021-11-29 10:36:24.845 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-29 10:36:24.965 [main] DataProviderCsv - appendUsers created:[User(id=1, name=Ivan), User(id=2, name=Maria), User(id=3, name=Dmitriy)]
[INFO ] 2021-11-29 10:36:24.966 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-29 10:50:40.534 [main] DataProviderCsvTest - AppendUser test Success
[INFO ] 2021-11-29 10:50:40.549 [main] DataProviderCsv - appendUser [start] appending user User(id=4, name=Daria)
[INFO ] 2021-11-29 10:50:40.549 [main] DataProviderCsv - getUserById [start] id:4
[INFO ] 2021-11-29 10:50:40.675 [main] DataProviderCsv - getUserById [end]
[INFO ] 2021-11-29 10:50:40.687 [main] DataProviderCsv - append User [end] appended User(id=4, name=Daria)
[INFO ] 2021-11-29 10:50:57.862 [main] DataProviderCsvTest - AppendUser test Fail
[INFO ] 2021-11-29 10:50:57.878 [main] DataProviderCsv - appendUser [start] appending user User(id=0, name=null)
[INFO ] 2021-11-29 10:50:57.878 [main] DataProviderCsv - Empty user
[INFO ] 2021-11-29 10:53:52.557 [main] DataProviderCsvTest - getUserById test Success
[INFO ] 2021-11-29 10:53:52.570 [main] DataProviderCsv - getUserById [start] id:2
[INFO ] 2021-11-29 10:53:52.690 [main] DataProviderCsv - getUserById [end]
[INFO ] 2021-11-29 10:53:52.695 [main] DataProviderCsvTest - User(id=2, name=Maria)
[INFO ] 2021-11-29 10:55:01.045 [main] DataProviderCsvTest - deleteUserById test Success
[INFO ] 2021-11-29 10:55:01.055 [main] DataProviderCsv - deleteUserById [start] id:1
[INFO ] 2021-11-29 10:55:01.075 [main] DataProviderCsv - ./src/main/resources/data/csv/user.csv has been deleted
[INFO ] 2021-11-29 10:55:01.077 [main] DataProviderCsv - ./src/main/resources/data/csv/newuser.csv has been written and renamed to ./src/main/resources/data/csv/user.csv
[INFO ] 2021-11-29 10:55:47.387 [main] DataProviderCsvTest - updateUser test Success
[INFO ] 2021-11-29 10:55:47.404 [main] DataProviderCsv - updateUser [start] user to update User(id=2, name=Boris)
[INFO ] 2021-11-29 10:55:47.404 [main] DataProviderCsv - getUserById [start] id:2
[INFO ] 2021-11-29 10:55:47.555 [main] DataProviderCsv - getUserById [end]
[INFO ] 2021-11-29 10:55:47.556 [main] DataProviderCsv - Entry found by given id Result{message='null', entity=User(id=2, name=Maria), code=SUCCESS}
[INFO ] 2021-11-29 10:55:47.557 [main] DataProviderCsv - deleteUserById [start] id:2
[INFO ] 2021-11-29 10:55:47.563 [main] DataProviderCsv - ./src/main/resources/data/csv/user.csv has been deleted
[INFO ] 2021-11-29 10:55:47.564 [main] DataProviderCsv - ./src/main/resources/data/csv/newuser.csv has been written and renamed to ./src/main/resources/data/csv/user.csv
[INFO ] 2021-11-29 10:55:47.565 [main] DataProviderCsv - appendUsers [start]
[INFO ] 2021-11-29 10:55:47.583 [main] DataProviderCsv - appendUsers created:[User(id=2, name=Boris)]
[INFO ] 2021-11-29 10:55:47.584 [main] DataProviderCsv - appendUsers [end]
[INFO ] 2021-11-29 10:55:47.584 [main] DataProviderCsv - Entry successfully updated
[INFO ] 2021-11-30 13:41:53.050 [main] DataProviderCsvTest - AppendUser test Fail
[INFO ] 2021-11-30 13:41:53.065 [main] DataProviderCsv - appendUser [start] appending user User(id=0, name=null)
[INFO ] 2021-11-30 13:41:53.066 [main] DataProviderCsv - Empty user
[INFO ] 2021-11-30 13:44:13.690 [main] DataProviderCsvTest - getUserById test Fail
[INFO ] 2021-11-30 13:44:13.703 [main] DataProviderCsv - getUserById [start] id:5
[INFO ] 2021-11-30 13:44:13.845 [main] DataProviderCsv - getUserById [end]
[INFO ] 2021-11-30 13:44:13.848 [main] DataProviderCsvTest - User(id=0, name=Empty)
[INFO ] 2021-11-30 13:44:13.848 [main] DataProviderCsvTest - getUserById test Fail - Success
[INFO ] 2021-11-30 13:45:25.383 [main] DataProviderCsvTest - deleteUserById test Fail
[INFO ] 2021-11-30 13:45:25.393 [main] DataProviderCsv - deleteUserById [start] id:5
[INFO ] 2021-11-30 13:45:25.411 [main] DataProviderCsv - ./src/main/resources/data/csv/user.csv has been deleted
[INFO ] 2021-11-30 13:45:25.413 [main] DataProviderCsv - ./src/main/resources/data/csv/newuser.csv has been written and renamed to ./src/main/resources/data/csv/user.csv
[INFO ] 2021-11-30 13:45:25.414 [main] DataProviderCsvTest - deleteUserById test Fail - Success
[INFO ] 2021-11-30 13:46:54.662 [main] DataProviderCsvTest - updateUser test Success
[INFO ] 2021-11-30 13:46:54.677 [main] DataProviderCsv - updateUser [start] user to update User(id=0, name=null)
[INFO ] 2021-11-30 13:46:54.677 [main] DataProviderCsv - Empty user
[INFO ] 2021-11-30 13:46:54.680 [main] DataProviderCsvTest - updateUser test Fail - Success
[INFO ] 2021-12-05 18:32:38.139 [main] DataProviderXml - appendUser[1]: writing user User(id=1, name=Ivan)
[INFO ] 2021-12-05 18:32:38.236 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-05 18:32:58.910 [main] DataProviderXml - appendUser[1]: writing user User(id=1, name=Ivan)
[INFO ] 2021-12-05 18:32:59.013 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-05 18:33:17.255 [main] DataProviderXml - appendUser[1]: writing user User(id=2, name=Daria)
[INFO ] 2021-12-05 18:33:17.368 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-05 18:48:50.725 [main] DataProviderXml - appendUser[1]: writing user User(id=2, name=Daria)
[INFO ] 2021-12-05 18:48:50.909 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-05 18:50:16.758 [main] DataProviderXml - appendUser[1]: writing user User(id=2, name=Daria)
[INFO ] 2021-12-05 18:50:16.854 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-06 10:05:17.129 [main] DataProviderCsvTest - AppendUser test Fail
[INFO ] 2021-12-06 10:05:17.144 [main] DataProviderCsv - appendUser [start] appending user User(id=0, name=null)
[INFO ] 2021-12-06 10:05:17.144 [main] DataProviderCsv - Empty user
[INFO ] 2021-12-06 10:05:17.147 [main] DataProviderCsvTest - appendUser test Fail - Success
[INFO ] 2021-12-06 10:05:34.078 [main] DataProviderCsvTest - getUserById test Fail
[INFO ] 2021-12-06 10:05:34.088 [main] DataProviderCsv - getUserById [start] id:5
[INFO ] 2021-12-06 10:05:34.274 [main] DataProviderCsv - getUserById [end]
[INFO ] 2021-12-06 10:05:34.279 [main] DataProviderCsvTest - User(id=0, name=Empty)
[INFO ] 2021-12-06 10:05:34.279 [main] DataProviderCsvTest - getUserById test Fail - Success
[INFO ] 2021-12-06 10:05:52.631 [main] DataProviderCsvTest - deleteUserById test Fail
[INFO ] 2021-12-06 10:05:52.641 [main] DataProviderCsv - deleteUserById [start] id:5
[INFO ] 2021-12-06 10:05:52.659 [main] DataProviderCsv - ./src/main/resources/data/csv/user.csv has been deleted
[INFO ] 2021-12-06 10:05:52.660 [main] DataProviderCsv - ./src/main/resources/data/csv/newuser.csv has been written and renamed to ./src/main/resources/data/csv/user.csv
[INFO ] 2021-12-06 10:05:52.661 [main] DataProviderCsvTest - deleteUserById test Fail - Success
[INFO ] 2021-12-06 10:06:19.311 [main] DataProviderCsvTest - updateUser test Success
[INFO ] 2021-12-06 10:06:19.326 [main] DataProviderCsv - updateUser [start] user to update User(id=0, name=null)
[INFO ] 2021-12-06 10:06:19.326 [main] DataProviderCsv - Empty user
[INFO ] 2021-12-06 10:06:19.329 [main] DataProviderCsvTest - updateUser test Fail - Success
[INFO ] 2021-12-06 10:06:49.523 [main] DataProviderCsvTest - AppendUser test Success
[INFO ] 2021-12-06 10:06:49.544 [main] DataProviderCsv - appendUser [start] appending user User(id=4, name=Daria)
[INFO ] 2021-12-06 10:06:49.544 [main] DataProviderCsv - getUserById [start] id:4
[INFO ] 2021-12-06 10:06:49.696 [main] DataProviderCsv - getUserById [end]
[INFO ] 2021-12-06 10:06:49.697 [main] DataProviderCsv - Entry already exists Result{message='null', entity=User(id=4, name=Daria), code=SUCCESS}
[INFO ] 2021-12-06 10:07:48.740 [main] DataProviderXml - appendUser[1]: writing user User(id=3, name=Ivan)
[INFO ] 2021-12-06 10:07:48.870 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-07 17:15:09.830 [main] DataProviderXml - appendUser[1]: writing user User(id=1, name=Ivan)
[INFO ] 2021-12-07 17:15:09.975 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-07 17:16:23.291 [main] DataProviderXml - appendUser[1]: writing user User(id=1, name=Ivan)
[INFO ] 2021-12-07 17:16:23.423 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-07 17:18:16.189 [main] DataProviderXml - appendUser[1]: writing user User(id=1, name=Ivan)
[INFO ] 2021-12-07 17:18:16.325 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-07 17:20:24.737 [main] DataProviderXml - appendUser[1]: writing user User(id=1, name=Ivan)
[INFO ] 2021-12-07 17:20:24.882 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-07 17:21:38.158 [main] DataProviderXml - appendUser[1]: writing user User(id=1, name=Ivan)
[INFO ] 2021-12-07 17:21:38.284 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-07 17:21:38.286 [main] DataProviderXml - appendUser[1]: writing user User(id=2, name=Boris)
[INFO ] 2021-12-07 17:21:38.288 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-07 17:21:38.288 [main] DataProviderXml - appendUser[1]: writing user User(id=3, name=Egor)
[INFO ] 2021-12-07 17:21:38.290 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-07 17:36:17.775 [main] DataProviderXml - appendUser[1]: writing user User(id=1, name=Ivan)
[INFO ] 2021-12-07 17:36:17.883 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-07 17:36:17.885 [main] DataProviderXml - appendUser[1]: writing user User(id=2, name=Boris)
[INFO ] 2021-12-07 17:36:17.887 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-07 17:36:17.887 [main] DataProviderXml - appendUser[1]: writing user User(id=3, name=Egor)
[INFO ] 2021-12-07 17:36:17.889 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-07 17:36:38.354 [main] DataProviderXml - Wrapper{userList=[User(id=3, name=Egor)]}
[INFO ] 2021-12-07 17:45:13.017 [main] DataProviderXml - appendUser[1]: writing user User(id=1, name=Ivan)
[ERROR] 2021-12-07 17:45:13.080 [main] DataProviderXml - appendUser[2]: exception received ParseError at [row,col]:[1,1]
Message: Premature end of file.
[INFO ] 2021-12-07 17:45:13.082 [main] DataProviderXml - appendUser[1]: writing user User(id=2, name=Boris)
[ERROR] 2021-12-07 17:45:13.083 [main] DataProviderXml - appendUser[2]: exception received ParseError at [row,col]:[1,1]
Message: Premature end of file.
[INFO ] 2021-12-07 17:45:13.083 [main] DataProviderXml - appendUser[1]: writing user User(id=3, name=Egor)
[ERROR] 2021-12-07 17:45:13.084 [main] DataProviderXml - appendUser[2]: exception received ParseError at [row,col]:[1,1]
Message: Premature end of file.
[INFO ] 2021-12-07 17:47:13.052 [main] DataProviderXml - appendUser[1]: writing user User(id=1, name=Ivan)
[ERROR] 2021-12-07 17:47:13.105 [main] DataProviderXml - appendUser[2]: exception received ParseError at [row,col]:[1,1]
Message: Premature end of file.
[INFO ] 2021-12-07 17:47:13.108 [main] DataProviderXml - appendUser[1]: writing user User(id=2, name=Boris)
[ERROR] 2021-12-07 17:47:13.109 [main] DataProviderXml - appendUser[2]: exception received ParseError at [row,col]:[1,1]
Message: Premature end of file.
[INFO ] 2021-12-07 17:47:13.109 [main] DataProviderXml - appendUser[1]: writing user User(id=3, name=Egor)
[ERROR] 2021-12-07 17:47:13.110 [main] DataProviderXml - appendUser[2]: exception received ParseError at [row,col]:[1,1]
Message: Premature end of file.
[INFO ] 2021-12-07 17:48:46.929 [main] DataProviderXml - appendUser[1]: writing user User(id=1, name=Ivan)
[ERROR] 2021-12-07 17:48:46.985 [main] DataProviderXml - appendUser[2]: exception received ParseError at [row,col]:[1,1]
Message: Premature end of file.
[INFO ] 2021-12-07 17:48:46.987 [main] DataProviderXml - appendUser[1]: writing user User(id=2, name=Boris)
[ERROR] 2021-12-07 17:48:46.988 [main] DataProviderXml - appendUser[2]: exception received ParseError at [row,col]:[1,1]
Message: Premature end of file.
[INFO ] 2021-12-07 17:48:46.988 [main] DataProviderXml - appendUser[1]: writing user User(id=3, name=Egor)
[ERROR] 2021-12-07 17:48:46.989 [main] DataProviderXml - appendUser[2]: exception received ParseError at [row,col]:[1,1]
Message: Premature end of file.
[INFO ] 2021-12-07 17:57:36.855 [main] DataProviderXml - appendUser[1]: writing user User(id=1, name=Ivan)
[INFO ] 2021-12-07 17:57:36.972 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-07 17:57:36.974 [main] DataProviderXml - appendUser[1]: writing user User(id=2, name=Boris)
[INFO ] 2021-12-07 17:57:36.987 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-07 17:57:36.987 [main] DataProviderXml - appendUser[1]: writing user User(id=3, name=Egor)
[INFO ] 2021-12-07 17:57:36.989 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-07 17:58:59.104 [main] DataProviderXml - appendUser[1]: writing user User(id=1, name=Ivan)
[INFO ] 2021-12-07 17:58:59.214 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-07 17:58:59.215 [main] DataProviderXml - appendUser[1]: writing user User(id=2, name=Boris)
[INFO ] 2021-12-07 17:58:59.231 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-07 17:58:59.231 [main] DataProviderXml - appendUser[1]: writing user User(id=3, name=Egor)
[INFO ] 2021-12-07 17:58:59.235 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-07 17:59:09.896 [main] DataProviderXml - Wrapper{userList=[User(id=1, name=Ivan), User(id=2, name=Boris), User(id=3, name=Egor)]}
[INFO ] 2021-12-07 18:06:03.702 [main] DataProviderXml - appendUser[1]: writing user User(id=1, name=Ivan)
[INFO ] 2021-12-07 18:06:03.824 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-07 18:06:03.825 [main] DataProviderXml - appendUser[1]: writing user User(id=2, name=Boris)
[INFO ] 2021-12-07 18:06:03.840 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-07 18:06:03.841 [main] DataProviderXml - appendUser[1]: writing user User(id=3, name=Egor)
[INFO ] 2021-12-07 18:06:03.845 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-07 18:13:15.804 [main] DataProviderXml - Wrapper{userList=[User(id=1, name=Ivan), User(id=2, name=Boris), User(id=3, name=Egor)]}
[INFO ] 2021-12-07 18:13:15.812 [main] DataProviderXml - Result{message='null', entity=User(id=3, name=Egor), code=ENTRY_NOT_FOUND}
[INFO ] 2021-12-07 18:14:46.512 [main] DataProviderXml - Wrapper{userList=[User(id=1, name=Ivan), User(id=2, name=Boris), User(id=3, name=Egor)]}
[INFO ] 2021-12-07 18:14:46.518 [main] DataProviderXml - Result{message='null', entity=User(id=3, name=Egor), code=SUCCESS}
[INFO ] 2021-12-07 18:30:42.164 [main] DataProviderXmlTest - appendUser test Success
[INFO ] 2021-12-07 18:30:42.178 [main] DataProviderXml - appendUser[1]: writing user User(id=1, name=Ivan)
[INFO ] 2021-12-07 18:30:42.287 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-07 18:30:42.289 [main] DataProviderXml - appendUser[1]: writing user User(id=2, name=Boris)
[INFO ] 2021-12-07 18:30:42.294 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-07 18:30:42.294 [main] DataProviderXml - appendUser[1]: writing user User(id=3, name=Egor)
[INFO ] 2021-12-07 18:30:42.299 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-07 18:33:05.377 [main] DataProviderXmlTest - appendUser test Fail
[INFO ] 2021-12-07 18:33:05.393 [main] DataProviderXml - appendUser[1]: writing user User(id=0, name=null)
[ERROR] 2021-12-07 18:33:05.515 [main] DataProviderXml - appendUser[2]: exception received Value for @org.simpleframework.xml.Element(name="", type=void.class, data=false, required=true) on field 'name' private java.lang.String ru.sfedu.log4jproject.model.beans.User.name is null in class ru.sfedu.log4jproject.model.beans.User
[INFO ] 2021-12-07 18:36:29.434 [main] DataProviderXmlTest - getUserById test Success
[INFO ] 2021-12-07 18:36:29.443 [main] DataProviderXml - getUserById[1]: finding by id=3
[INFO ] 2021-12-07 18:36:29.550 [main] DataProviderXml - getUserById[2]: searching result Result{message='null', entity=User(id=3, name=Egor), code=SUCCESS}
[INFO ] 2021-12-07 18:37:54.754 [main] DataProviderXmlTest - getUserById test Success
[INFO ] 2021-12-07 18:37:54.762 [main] DataProviderXml - getUserById[1]: searching for id=3
[INFO ] 2021-12-07 18:37:54.865 [main] DataProviderXml - getUserById[2]: searching result Result{message='null', entity=User(id=3, name=Egor), code=SUCCESS}
[INFO ] 2021-12-07 18:38:48.824 [main] DataProviderXmlTest - getUserById test Fail
[INFO ] 2021-12-07 18:38:48.833 [main] DataProviderXml - getUserById[1]: searching for id=5
[INFO ] 2021-12-07 18:38:48.936 [main] DataProviderXml - getUserById[2]: searching result Result{message='null', entity=null, code=ENTRY_NOT_FOUND}
[INFO ] 2021-12-12 14:01:32.573 [main] DataProviderXmlTest - deleteUserById test Success
[INFO ] 2021-12-12 14:01:32.583 [main] DataProviderXml - deleteUserById[1]: deleting user by id=2
[INFO ] 2021-12-12 14:01:32.748 [main] DataProviderXml - deleteUserById[2]: entry successfully deleted
[INFO ] 2021-12-12 14:02:21.671 [main] DataProviderXmlTest - deleteUserById test Fail
[INFO ] 2021-12-12 14:02:21.679 [main] DataProviderXml - deleteUserById[1]: deleting user by id=14
[INFO ] 2021-12-12 14:02:21.828 [main] DataProviderXml - deleteUserById[2]: entry successfully deleted
[INFO ] 2021-12-12 14:17:54.692 [main] DataProviderXmlTest - updateUser test Success
[INFO ] 2021-12-12 14:17:54.706 [main] DataProviderXml - updateUser[1]: updating userUser(id=3, name=Lisa)
[INFO ] 2021-12-12 14:17:54.708 [main] DataProviderXml - deleteUserById[1]: deleting user by id=3
[INFO ] 2021-12-12 14:17:54.819 [main] DataProviderXml - deleteUserById[2]: entry deleted
[INFO ] 2021-12-12 14:17:54.821 [main] DataProviderXml - appendUser[1]: writing user User(id=3, name=Lisa)
[INFO ] 2021-12-12 14:17:54.825 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-12 14:17:54.825 [main] DataProviderXml - updateUser[2]: user updated
[INFO ] 2021-12-12 14:18:41.897 [main] DataProviderXmlTest - updateUser test Fail
[INFO ] 2021-12-12 14:18:41.910 [main] DataProviderXml - updateUser[1]: updating userUser(id=15, name=Steve)
[INFO ] 2021-12-12 14:18:41.912 [main] DataProviderXml - deleteUserById[1]: deleting user by id=15
[INFO ] 2021-12-12 14:18:42.018 [main] DataProviderXml - deleteUserById[2]: entry deleted
[INFO ] 2021-12-12 14:18:42.020 [main] DataProviderXml - appendUser[1]: writing user User(id=15, name=Steve)
[INFO ] 2021-12-12 14:18:42.025 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-12 14:18:42.025 [main] DataProviderXml - updateUser[2]: user updated
[INFO ] 2021-12-12 14:19:09.365 [main] DataProviderXmlTest - updateUser test Fail
[INFO ] 2021-12-12 14:19:09.377 [main] DataProviderXml - updateUser[1]: updating userUser(id=0, name=null)
[ERROR] 2021-12-12 14:19:09.378 [main] Validator - Invalid user
[ERROR] 2021-12-12 14:19:09.379 [main] DataProviderXml - updateUser[2]: exception received Invalid user
[INFO ] 2021-12-13 10:04:28.176 [main] DataProviderXmlTest - appendUser test Fail
[INFO ] 2021-12-13 10:04:28.187 [main] DataProviderXml - appendUser[1]: writing user User(id=0, name=null)
[ERROR] 2021-12-13 10:04:28.187 [main] DataProviderXml - appendUser[2]: exception received Cannot invoke "String.isEmpty()" because the return value of "ru.sfedu.log4jproject.model.beans.User.getName()" is null
[INFO ] 2021-12-13 10:05:49.012 [main] DataProviderXmlTest - appendUser test Success
[INFO ] 2021-12-13 10:05:49.024 [main] DataProviderXml - appendUser[1]: writing user User(id=5, name=Fedya)
[ERROR] 2021-12-13 10:05:49.028 [main] Validator - Invalid user
[ERROR] 2021-12-13 10:05:49.028 [main] DataProviderXml - appendUser[2]: exception received Invalid user
[INFO ] 2021-12-13 10:06:18.733 [main] DataProviderXmlTest - appendUser test Success
[INFO ] 2021-12-13 10:06:18.745 [main] DataProviderXml - appendUser[1]: writing user User(id=5, name=Fedya)
[INFO ] 2021-12-13 10:06:18.877 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-13 10:06:36.336 [main] DataProviderXmlTest - appendUser test Fail
[INFO ] 2021-12-13 10:06:36.352 [main] DataProviderXml - appendUser[1]: writing user User(id=0, name=null)
[ERROR] 2021-12-13 10:06:36.356 [main] Validator - Invalid user
[ERROR] 2021-12-13 10:06:36.356 [main] DataProviderXml - appendUser[2]: exception received Invalid user
[INFO ] 2021-12-13 10:07:00.511 [main] DataProviderXmlTest - getUserById test Success
[INFO ] 2021-12-13 10:07:00.519 [main] DataProviderXml - getUserById[1]: searching for id=15
[INFO ] 2021-12-13 10:07:00.631 [main] DataProviderXml - getUserById[2]: searching result Result{message='null', entity=User(id=15, name=Steve), code=ENTRY_NOT_FOUND}
[INFO ] 2021-12-13 10:07:21.291 [main] DataProviderXmlTest - getUserById test Fail
[INFO ] 2021-12-13 10:07:21.303 [main] DataProviderXml - getUserById[1]: searching for id=6
[INFO ] 2021-12-13 10:07:21.431 [main] DataProviderXml - getUserById[2]: searching result Result{message='null', entity=null, code=ENTRY_NOT_FOUND}
[INFO ] 2021-12-13 10:07:45.683 [main] DataProviderXmlTest - deleteUserById test Success
[INFO ] 2021-12-13 10:07:45.691 [main] DataProviderXml - deleteUserById[1]: deleting user by id=15
[INFO ] 2021-12-13 10:07:45.843 [main] DataProviderXml - deleteUserById[2]: entry deleted
[INFO ] 2021-12-13 10:08:28.796 [main] DataProviderXmlTest - appendUser test Success
[INFO ] 2021-12-13 10:08:28.808 [main] DataProviderXml - appendUser[1]: writing user User(id=5, name=Fedya)
[INFO ] 2021-12-13 10:08:28.928 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-13 10:08:28.948 [main] DataProviderXmlTest - appendUser test Fail
[INFO ] 2021-12-13 10:08:28.948 [main] DataProviderXml - appendUser[1]: writing user User(id=0, name=null)
[ERROR] 2021-12-13 10:08:28.948 [main] Validator - Invalid user
[ERROR] 2021-12-13 10:08:28.948 [main] DataProviderXml - appendUser[2]: exception received Invalid user
[INFO ] 2021-12-13 10:08:28.948 [main] DataProviderXmlTest - deleteUserById test Success
[INFO ] 2021-12-13 10:08:28.948 [main] DataProviderXml - deleteUserById[1]: deleting user by id=15
[INFO ] 2021-12-13 10:08:28.956 [main] DataProviderXml - deleteUserById[2]: entry deleted
[INFO ] 2021-12-13 10:08:28.956 [main] DataProviderXmlTest - updateUser test Fail
[INFO ] 2021-12-13 10:08:28.956 [main] DataProviderXml - updateUser[1]: updating userUser(id=0, name=null)
[ERROR] 2021-12-13 10:08:28.956 [main] Validator - Invalid user
[ERROR] 2021-12-13 10:08:28.960 [main] DataProviderXml - updateUser[2]: exception received Invalid user
[INFO ] 2021-12-13 10:08:28.960 [main] DataProviderXmlTest - getUserById test Success
[INFO ] 2021-12-13 10:08:28.960 [main] DataProviderXml - getUserById[1]: searching for id=15
[INFO ] 2021-12-13 10:08:28.964 [main] DataProviderXml - getUserById[2]: searching result Result{message='null', entity=null, code=ENTRY_NOT_FOUND}
[INFO ] 2021-12-13 10:08:28.964 [main] DataProviderXmlTest - updateUser test Success
[INFO ] 2021-12-13 10:08:28.964 [main] DataProviderXml - updateUser[1]: updating userUser(id=3, name=Lisa)
[INFO ] 2021-12-13 10:08:28.964 [main] DataProviderXml - deleteUserById[1]: deleting user by id=3
[INFO ] 2021-12-13 10:08:28.968 [main] DataProviderXml - deleteUserById[2]: entry deleted
[INFO ] 2021-12-13 10:08:28.968 [main] DataProviderXml - appendUser[1]: writing user User(id=3, name=Lisa)
[INFO ] 2021-12-13 10:08:28.976 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-13 10:08:28.976 [main] DataProviderXml - updateUser[2]: user updated
[INFO ] 2021-12-13 10:08:28.976 [main] DataProviderXmlTest - deleteUserById test Fail
[INFO ] 2021-12-13 10:08:28.976 [main] DataProviderXml - deleteUserById[1]: deleting user by id=14
[INFO ] 2021-12-13 10:08:28.980 [main] DataProviderXml - deleteUserById[2]: entry deleted
[INFO ] 2021-12-13 10:08:28.984 [main] DataProviderXmlTest - getUserById test Fail
[INFO ] 2021-12-13 10:08:28.984 [main] DataProviderXml - getUserById[1]: searching for id=6
[INFO ] 2021-12-13 10:08:28.984 [main] DataProviderXml - getUserById[2]: searching result Result{message='null', entity=null, code=ENTRY_NOT_FOUND}
[INFO ] 2021-12-18 16:50:48.361 [main] DataProviderH2 - appendUser[]: Exception received Таблица "USERS" не найдена
Table "USERS" not found; SQL statement:
SELECT ID, NAME FROM USERS [42102-200]
[INFO ] 2021-12-18 16:52:25.837 [main] DataProviderH2 - rs3: org.h2.result.LocalResultImpl@4cc6fa2a columns: 2 rows: 1 pos: 0
[INFO ] 2021-12-18 16:53:28.347 [main] DataProviderH2 - 1
[INFO ] 2021-12-18 16:53:28.352 [main] DataProviderH2 - Ivan
[ERROR] 2021-12-18 17:09:51.378 [main] Validator - Invalid user
[INFO ] 2021-12-18 17:09:51.383 [main] DataProviderH2 - appendUser[]: Exception received Invalid user
[INFO ] 2021-12-18 17:10:14.778 [main] DataProviderH2 - appendUser[]: Exception received Столбец "TANISHA" не найден
Column "TANISHA" not found; SQL statement:
INSERT INTO USERS VALUES (2, Tanisha) [42122-200]
[INFO ] 2021-12-18 17:10:55.904 [main] DataProviderH2 - appendUser[]: Exception received Метод разрешен только для запросов. Используйте execute или executeUpdate вместо executeQuery
Method is only allowed for a query. Use execute or executeUpdate instead of executeQuery; SQL statement:
INSERT INTO USERS VALUES (2, 'Tanisha') [90002-200]
[INFO ] 2021-12-18 17:12:27.334 [main] DataProviderH2 - appendUser[]: Exception received Нарушение уникального индекса или первичного ключа: "PRIMARY KEY ON PUBLIC.USERS(ID)"
Unique index or primary key violation: "PRIMARY KEY ON PUBLIC.USERS(ID)"; SQL statement:
INSERT INTO USERS VALUES (2, 'Tanisha'); [23505-200]
[INFO ] 2021-12-18 17:17:24.171 [main] DataProviderH2 - appendUser[]: Exception received Нарушение уникального индекса или первичного ключа: "PRIMARY KEY ON PUBLIC.USERS(ID)"
Unique index or primary key violation: "PRIMARY KEY ON PUBLIC.USERS(ID)"; SQL statement:
INSERT INTO USERS VALUES (2, 'Tanisha'); [23505-200]
[INFO ] 2021-12-18 17:18:57.218 [main] DataProviderH2 - appendUser[]: Exception received Нарушение уникального индекса или первичного ключа: "PRIMARY KEY ON PUBLIC.USERS(ID)"
Unique index or primary key violation: "PRIMARY KEY ON PUBLIC.USERS(ID)"; SQL statement:
INSERT INTO USERS VALUES (3, 'Bob'); [23505-200]
[INFO ] 2021-12-18 17:19:56.445 [main] DataProviderH2 - appendUser[]: Exception received Нарушение уникального индекса или первичного ключа: "PRIMARY KEY ON PUBLIC.USERS(ID)"
Unique index or primary key violation: "PRIMARY KEY ON PUBLIC.USERS(ID)"; SQL statement:
INSERT INTO USERS VALUES (0, 'Bob'); [23505-200]
[ERROR] 2021-12-18 17:24:25.942 [main] Validator - Invalid user
[INFO ] 2021-12-18 17:24:25.948 [main] DataProviderH2 - appendUser[]: Exception received Invalid user
[INFO ] 2021-12-18 17:24:38.391 [main] DataProviderH2 - appendUser[]: Exception received Нарушение уникального индекса или первичного ключа: "PRIMARY KEY ON PUBLIC.USERS(ID)"
Unique index or primary key violation: "PRIMARY KEY ON PUBLIC.USERS(ID)"; SQL statement:
INSERT INTO USERS VALUES (3, 'Bob') [23505-200]
[INFO ] 2021-12-18 17:26:28.405 [main] DataProviderH2 - appendUser[]: Exception received Синтаксическая ошибка в выражении SQL "INSERT INTO USERS VALUE[*] ('Bob')"; ожидалось "., (, DIRECT, SORTED, DEFAULT, VALUES, SET, (, WITH, SELECT, TABLE, VALUES"
Syntax error in SQL statement "INSERT INTO USERS VALUE[*] ('Bob')"; expected "., (, DIRECT, SORTED, DEFAULT, VALUES, SET, (, WITH, SELECT, TABLE, VALUES"; SQL statement:
INSERT INTO USERS VALUE ('Bob') [42001-200]
[INFO ] 2021-12-18 17:26:47.705 [main] DataProviderH2 - appendUser[]: Exception received Синтаксическая ошибка в выражении SQL "INSERT INTO USERS VALUES (,[*]'Bob')"; ожидалось "), DEFAULT, NOT, EXISTS, INTERSECTS, UNIQUE"
Syntax error in SQL statement "INSERT INTO USERS VALUES (,[*]'Bob')"; expected "), DEFAULT, NOT, EXISTS, INTERSECTS, UNIQUE"; SQL statement:
INSERT INTO USERS VALUES (,'Bob') [42001-200]
[INFO ] 2021-12-18 17:27:16.033 [main] DataProviderH2 - appendUser[]: Exception received Неверное количество столбцов
Column count does not match; SQL statement:
INSERT INTO USERS VALUES ('Bob') [21002-200]
[INFO ] 2021-12-18 17:30:24.947 [main] DataProviderH2 - 0:Bob
[INFO ] 2021-12-18 17:30:24.950 [main] DataProviderH2 - 1:Ivan
[INFO ] 2021-12-18 17:30:24.950 [main] DataProviderH2 - 2:Tanisha
[INFO ] 2021-12-18 17:30:24.950 [main] DataProviderH2 - 3:Bob
[INFO ] 2021-12-18 17:30:24.950 [main] DataProviderH2 - 101:Bob
[INFO ] 2021-12-18 17:37:39.775 [main] DataProviderH2 - appendUser[]: Exception received База данных уже используется: "Locked by another process: C:/Users/legion/IdeaProjects/Log4jproject/src/main/resources/data/jdbc/Users.lock.db". Возможные решения: закрыть все другие соединения; использовать режим сервера
Database may be already in use: "Locked by another process: C:/Users/legion/IdeaProjects/Log4jproject/src/main/resources/data/jdbc/Users.lock.db". Possible solutions: close all other connection(s); use the server mode [90020-200]
[INFO ] 2021-12-18 17:38:04.805 [main] DataProviderH2 - 1:Ivan
[INFO ] 2021-12-18 17:38:04.810 [main] DataProviderH2 - 2:Tanisha
[ERROR] 2021-12-18 17:46:36.761 [main] Validator - Invalid user
[INFO ] 2021-12-18 17:46:36.768 [main] DataProviderH2 - appendUser[]: Exception received Invalid user
[INFO ] 2021-12-18 17:46:51.815 [main] DataProviderH2 - 3
[INFO ] 2021-12-18 17:48:21.308 [main] DataProviderH2 - appendUser[]: Exception received База данных уже используется: "Locked by another process: C:/Users/legion/IdeaProjects/Log4jproject/src/main/resources/data/jdbc/Users.lock.db". Возможные решения: закрыть все другие соединения; использовать режим сервера
Database may be already in use: "Locked by another process: C:/Users/legion/IdeaProjects/Log4jproject/src/main/resources/data/jdbc/Users.lock.db". Possible solutions: close all other connection(s); use the server mode [90020-200]
[INFO ] 2021-12-18 17:49:27.313 [main] DataProviderH2 - appendUser[]: Exception received База данных уже используется: "Locked by another process: C:/Users/legion/IdeaProjects/Log4jproject/src/main/resources/data/jdbc/Users.lock.db". Возможные решения: закрыть все другие соединения; использовать режим сервера
Database may be already in use: "Locked by another process: C:/Users/legion/IdeaProjects/Log4jproject/src/main/resources/data/jdbc/Users.lock.db". Possible solutions: close all other connection(s); use the server mode [90020-200]
[INFO ] 2021-12-18 18:05:54.517 [main] DataProviderH2 - getUserById[1]: getting user by id 2
[INFO ] 2021-12-18 18:05:56.855 [main] DataProviderH2 - getUserById[1]: received user User(id=2, name=Tanisha)
[INFO ] 2021-12-18 18:06:30.685 [main] DataProviderH2 - getUserById[1]: getting user by id 67
[INFO ] 2021-12-18 18:06:31.003 [main] DataProviderH2 - getUserById[1]: received user User(id=0, name=null)
[INFO ] 2021-12-18 18:24:11.447 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-18 18:24:11.457 [main] DataProviderH2 - appendUser[1]: appending user User(id=25, name=Fred)
[INFO ] 2021-12-18 18:24:11.459 [main] IdGenerator - generateForH2[1]: generating new id on jdbc:h2:C:\Users\legion\IdeaProjects\Log4jproject\src\main\resources\data\jdbc\Users;MV_STORE=false
[INFO ] 2021-12-18 18:24:11.786 [main] IdGenerator - generateForH2[2]: generated id 4
[INFO ] 2021-12-18 18:24:11.865 [main] DataProviderH2 - appendUser[2]: user appended
[INFO ] 2021-12-18 18:25:06.313 [main] DataProviderH2 - appendUser test Fail
[INFO ] 2021-12-18 18:25:06.324 [main] DataProviderH2 - appendUser[1]: appending user User(id=0, name=null)
[ERROR] 2021-12-18 18:25:06.325 [main] Validator - Invalid user
[INFO ] 2021-12-18 18:25:06.326 [main] DataProviderH2 - appendUser[2]: Exception received Invalid user
[INFO ] 2021-12-18 18:26:51.137 [main] DataProviderH2 - getUserById test success
[INFO ] 2021-12-18 18:26:51.142 [main] DataProviderH2 - getUserById[1]: getting user by id 2
[INFO ] 2021-12-18 18:26:53.467 [main] DataProviderH2 - getUserById[1]: received user User(id=2, name=Tanisha)
[INFO ] 2021-12-18 18:27:37.723 [main] DataProviderH2 - getUserById test Fail
[INFO ] 2021-12-18 18:27:37.729 [main] DataProviderH2 - getUserById[1]: getting user by id 580
[INFO ] 2021-12-18 18:27:38.026 [main] DataProviderH2 - getUserById[1]: received user User(id=0, name=null)
[INFO ] 2021-12-18 18:28:33.114 [main] DataProviderH2 - getUserById test Fail
[INFO ] 2021-12-18 18:28:33.122 [main] DataProviderH2 - getUserById[1]: getting user by id 580
[ERROR] 2021-12-18 18:28:33.415 [main] Validator - Invalid user
[INFO ] 2021-12-18 18:28:33.415 [main] DataProviderH2 - getUserById[]: exception received Invalid user
[INFO ] 2021-12-18 18:38:02.134 [main] DataProviderH2 - deleteUserById test Success
[INFO ] 2021-12-18 18:38:02.140 [main] DataProviderH2 - deleteUserById[1]: deleting user by id 2
[INFO ] 2021-12-18 18:38:02.504 [main] DataProviderH2 - deleteUserById[2]: user deleted
[INFO ] 2021-12-18 18:38:42.857 [main] DataProviderH2 - deleteUserById test Fail
[INFO ] 2021-12-18 18:38:42.863 [main] DataProviderH2 - deleteUserById[1]: deleting user by id 35
[INFO ] 2021-12-18 18:38:45.230 [main] DataProviderH2 - deleteUserById[2]: user deleted
[INFO ] 2021-12-18 18:58:13.770 [main] DataProviderH2 - updateUser test Success
[INFO ] 2021-12-18 18:58:13.782 [main] DataProviderH2 - updateUser[1]: updating user User(id=2, name=Jane)
[INFO ] 2021-12-18 18:58:16.150 [main] DataProviderH2 - update[]: exception received Столбец "JANE" не найден
Column "JANE" not found; SQL statement:
UPDATE USERS SET NAME = Jane WHERE ID = 2 [42122-200]
[INFO ] 2021-12-18 18:58:36.124 [main] DataProviderH2 - updateUser test Success
[INFO ] 2021-12-18 18:58:36.136 [main] DataProviderH2 - updateUser[1]: updating user User(id=2, name=Jane)
[INFO ] 2021-12-18 18:58:36.457 [main] DataProviderH2 - updateUser[2]: user updated
[INFO ] 2021-12-18 18:58:57.187 [main] DataProviderH2 - updateUser test Success
[INFO ] 2021-12-18 18:58:57.199 [main] DataProviderH2 - updateUser[1]: updating user User(id=3, name=Jane)
[INFO ] 2021-12-18 18:58:59.320 [main] DataProviderH2 - update[]: exception received База данных уже используется: "Locked by another process: C:/Users/legion/IdeaProjects/Log4jproject/src/main/resources/data/jdbc/Users.lock.db". Возможные решения: закрыть все другие соединения; использовать режим сервера
Database may be already in use: "Locked by another process: C:/Users/legion/IdeaProjects/Log4jproject/src/main/resources/data/jdbc/Users.lock.db". Possible solutions: close all other connection(s); use the server mode [90020-200]
[INFO ] 2021-12-18 18:59:08.468 [main] DataProviderH2 - updateUser test Success
[INFO ] 2021-12-18 18:59:08.478 [main] DataProviderH2 - updateUser[1]: updating user User(id=3, name=Jane)
[INFO ] 2021-12-18 18:59:10.838 [main] DataProviderH2 - updateUser[2]: user updated
[INFO ] 2021-12-18 19:02:13.546 [main] DataProviderH2 - updateUser test Success
[INFO ] 2021-12-18 19:02:13.556 [main] DataProviderH2 - updateUser[1]: updating user User(id=3, name=Jade)
[INFO ] 2021-12-18 19:02:15.857 [main] DataProviderH2 - getUserById[1]: getting user by id 3
[INFO ] 2021-12-18 19:02:15.859 [main] DataProviderH2 - getUserById[2]: received user User(id=3, name=Jane)
[INFO ] 2021-12-18 19:02:15.877 [main] DataProviderH2 - updateUser[2]: user updated
[INFO ] 2021-12-18 19:02:59.945 [main] DataProviderH2 - updateUser test Success
[INFO ] 2021-12-18 19:02:59.955 [main] DataProviderH2 - updateUser[1]: updating user User(id=2, name=Michael)
[INFO ] 2021-12-18 19:03:02.318 [main] DataProviderH2 - getUserById[1]: getting user by id 2
[ERROR] 2021-12-18 19:03:02.320 [main] Validator - Invalid user
[INFO ] 2021-12-18 19:03:02.320 [main] DataProviderH2 - getUserById[]: exception received Invalid user
[INFO ] 2021-12-18 19:03:02.322 [main] DataProviderH2 - appendUser[1]: appending user User(id=2, name=Michael)
[INFO ] 2021-12-18 19:03:02.323 [main] IdGenerator - generateForH2[1]: generating new id on jdbc:h2:C:\Users\legion\IdeaProjects\Log4jproject\src\main\resources\data\jdbc\Users;MV_STORE=false
[INFO ] 2021-12-18 19:03:02.324 [main] IdGenerator - generateForH2[2]: generated id 5
[INFO ] 2021-12-18 19:03:02.328 [main] DataProviderH2 - appendUser[2]: user appended
[INFO ] 2021-12-18 19:03:02.328 [main] DataProviderH2 - updateUser[2]: user updated
[INFO ] 2021-12-18 19:04:04.600 [main] DataProviderH2 - updateUser test Success
[INFO ] 2021-12-18 19:04:04.612 [main] DataProviderH2 - updateUser[1]: updating user User(id=2, name=Michael)
[INFO ] 2021-12-18 19:04:06.934 [main] DataProviderH2 - getUserById[1]: getting user by id 2
[ERROR] 2021-12-18 19:04:06.937 [main] Validator - Invalid user
[INFO ] 2021-12-18 19:04:06.937 [main] DataProviderH2 - getUserById[]: exception received Invalid user
[INFO ] 2021-12-18 19:04:06.938 [main] DataProviderH2 - appendUser[1]: appending user User(id=2, name=Michael)
[INFO ] 2021-12-18 19:04:06.939 [main] IdGenerator - generateForH2[1]: generating new id on jdbc:h2:C:\Users\legion\IdeaProjects\Log4jproject\src\main\resources\data\jdbc\Users;MV_STORE=false
[INFO ] 2021-12-18 19:04:06.941 [main] IdGenerator - generateForH2[2]: generated id 6
[INFO ] 2021-12-18 19:04:06.944 [main] DataProviderH2 - appendUser[2]: user appended
[INFO ] 2021-12-18 19:04:06.944 [main] DataProviderH2 - updateUser[2]: user updated
[INFO ] 2021-12-18 19:04:38.681 [main] DataProviderH2 - deleteUserById test Success
[INFO ] 2021-12-18 19:04:38.688 [main] DataProviderH2 - deleteUserById[1]: deleting user by id 6
[INFO ] 2021-12-18 19:04:41.049 [main] DataProviderH2 - deleteUserById[2]: user deleted
[INFO ] 2021-12-18 19:05:00.550 [main] DataProviderH2 - updateUser test Fail
[INFO ] 2021-12-18 19:05:00.561 [main] DataProviderH2 - updateUser[1]: updating user User(id=0, name=null)
[ERROR] 2021-12-18 19:05:00.563 [main] Validator - Invalid user
[INFO ] 2021-12-18 19:05:00.563 [main] DataProviderH2 - update[]: exception received Invalid user
[INFO ] 2021-12-20 09:59:58.512 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-20 09:59:58.523 [main] DataProviderH2 - appendUser[1]: appending user User(id=25, name=Fred)
[INFO ] 2021-12-20 09:59:58.526 [main] IdGenerator - generateForH2[1]: generating new id on jdbc:h2:C:\Users\legion\IdeaProjects\Log4jproject\src\main\resources\data\jdbc\Users;MV_STORE=false
[INFO ] 2021-12-20 09:59:58.923 [main] IdGenerator - generateForH2[2]: generated id 6
[INFO ] 2021-12-20 09:59:58.990 [main] DataProviderH2 - appendUser[2]: user appended
[INFO ] 2021-12-20 09:59:59.018 [main] DataProviderH2 - appendUser test Fail
[INFO ] 2021-12-20 09:59:59.019 [main] DataProviderH2 - appendUser[1]: appending user User(id=0, name=null)
[ERROR] 2021-12-20 09:59:59.019 [main] Validator - Invalid user
[INFO ] 2021-12-20 09:59:59.020 [main] DataProviderH2 - appendUser[]: Exception received Invalid user
[INFO ] 2021-12-20 09:59:59.023 [main] DataProviderH2 - deleteUserById test Success
[INFO ] 2021-12-20 09:59:59.023 [main] DataProviderH2 - deleteUserById[1]: deleting user by id 6
[INFO ] 2021-12-20 09:59:59.089 [main] DataProviderH2 - deleteUserById[2]: user deleted
[INFO ] 2021-12-20 09:59:59.091 [main] DataProviderH2 - updateUser test Success
[INFO ] 2021-12-20 09:59:59.091 [main] DataProviderH2 - updateUser[1]: updating user User(id=2, name=Michael)
[INFO ] 2021-12-20 09:59:59.138 [main] DataProviderH2 - getUserById[1]: getting user by id 2
[ERROR] 2021-12-20 09:59:59.140 [main] Validator - Invalid user
[INFO ] 2021-12-20 09:59:59.140 [main] DataProviderH2 - getUserById[]: exception received Invalid user
[INFO ] 2021-12-20 09:59:59.140 [main] DataProviderH2 - appendUser[1]: appending user User(id=2, name=Michael)
[INFO ] 2021-12-20 09:59:59.141 [main] IdGenerator - generateForH2[1]: generating new id on jdbc:h2:C:\Users\legion\IdeaProjects\Log4jproject\src\main\resources\data\jdbc\Users;MV_STORE=false
[INFO ] 2021-12-20 09:59:59.142 [main] IdGenerator - generateForH2[2]: generated id 6
[INFO ] 2021-12-20 09:59:59.143 [main] DataProviderH2 - appendUser[2]: user appended
[INFO ] 2021-12-20 09:59:59.144 [main] DataProviderH2 - updateUser[2]: user updated
[INFO ] 2021-12-20 09:59:59.146 [main] DataProviderH2 - updateUser test Fail
[INFO ] 2021-12-20 09:59:59.146 [main] DataProviderH2 - updateUser[1]: updating user User(id=0, name=null)
[ERROR] 2021-12-20 09:59:59.146 [main] Validator - Invalid user
[INFO ] 2021-12-20 09:59:59.146 [main] DataProviderH2 - update[]: exception received Invalid user
[INFO ] 2021-12-20 09:59:59.148 [main] DataProviderH2 - getUserById test success
[INFO ] 2021-12-20 09:59:59.149 [main] DataProviderH2 - getUserById[1]: getting user by id 2
[ERROR] 2021-12-20 09:59:59.150 [main] Validator - Invalid user
[INFO ] 2021-12-20 09:59:59.151 [main] DataProviderH2 - getUserById[]: exception received Invalid user
[INFO ] 2021-12-20 09:59:59.153 [main] DataProviderH2 - deleteUserById test Fail
[INFO ] 2021-12-20 09:59:59.153 [main] DataProviderH2 - deleteUserById[1]: deleting user by id 35
[INFO ] 2021-12-20 09:59:59.155 [main] DataProviderH2 - deleteUserById[2]: user deleted
[INFO ] 2021-12-20 09:59:59.157 [main] DataProviderH2 - getUserById test Fail
[INFO ] 2021-12-20 09:59:59.157 [main] DataProviderH2 - getUserById[1]: getting user by id 580
[ERROR] 2021-12-20 09:59:59.159 [main] Validator - Invalid user
[INFO ] 2021-12-20 09:59:59.159 [main] DataProviderH2 - getUserById[]: exception received Invalid user
[INFO ] 2021-12-20 10:20:56.671 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-20 10:20:56.685 [main] DataProviderH2 - appendUser[1]: appending user User(id=22, name=Lisa)
[INFO ] 2021-12-20 10:20:56.688 [main] IdGenerator - generateForH2[1]: generating new id on h2_url
[INFO ] 2021-12-20 10:20:56.693 [main] DataProviderH2 - appendUser[]: Exception received No suitable driver found for h2_url
[INFO ] 2021-12-20 10:23:44.678 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-20 10:23:44.692 [main] DataProviderH2 - appendUser[1]: appending user User(id=22, name=Lisa)
[INFO ] 2021-12-20 10:23:44.693 [main] DataProviderH2 - jdbc:h2:C:\Users\legion\IdeaProjects\Log4jproject\src\main\resources\data\jdbc\Users;MV_STORE=false
[INFO ] 2021-12-20 10:23:44.696 [main] IdGenerator - generateForH2[1]: generating new id on h2_url
[INFO ] 2021-12-20 10:23:44.700 [main] DataProviderH2 - appendUser[]: Exception received No suitable driver found for h2_url
[INFO ] 2021-12-20 10:25:47.021 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-20 10:25:47.039 [main] DataProviderH2 - appendUser[1]: appending user User(id=22, name=Lisa)
[INFO ] 2021-12-20 10:25:47.042 [main] DataProviderH2 - jdbc:h2:./src\main\resources\data\jdbc\Users;MV_STORE=false
[INFO ] 2021-12-20 10:25:47.044 [main] IdGenerator - generateForH2[1]: generating new id on h2_url
[INFO ] 2021-12-20 10:25:47.051 [main] DataProviderH2 - appendUser[]: Exception received No suitable driver found for h2_url
[INFO ] 2021-12-20 10:26:12.428 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-20 10:26:12.438 [main] DataProviderH2 - appendUser[1]: appending user User(id=22, name=Lisa)
[INFO ] 2021-12-20 10:26:12.442 [main] DataProviderH2 - jdbc:h2:./src/main/resources/data/jdbc/Users;MV_STORE=false
[INFO ] 2021-12-20 10:26:12.445 [main] IdGenerator - generateForH2[1]: generating new id on h2_url
[INFO ] 2021-12-20 10:26:12.449 [main] DataProviderH2 - appendUser[]: Exception received No suitable driver found for h2_url
[INFO ] 2021-12-20 10:27:09.449 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-20 10:27:09.462 [main] DataProviderH2 - appendUser[1]: appending user User(id=22, name=Lisa)
[INFO ] 2021-12-20 10:27:09.465 [main] DataProviderH2 - jdbc:h2:C:\Users\legion\IdeaProjects\Log4jproject\src\main\resources\data\jdbc\Users;MV_STORE=false
[INFO ] 2021-12-20 10:27:09.468 [main] IdGenerator - generateForH2[1]: generating new id on h2_url
[INFO ] 2021-12-20 10:27:09.474 [main] DataProviderH2 - appendUser[]: Exception received No suitable driver found for h2_url
[INFO ] 2021-12-20 10:28:18.625 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-20 10:28:18.639 [main] DataProviderH2 - appendUser[1]: appending user User(id=22, name=Lisa)
[INFO ] 2021-12-20 10:28:18.641 [main] DataProviderH2 - jdbc:h2:C:\Users\legion\IdeaProjects\Log4jproject\src\main\resources\data\jdbc\Users;MV_STORE=false
[INFO ] 2021-12-20 10:28:18.643 [main] IdGenerator - generateForH2[1]: generating new id on h2_url
[INFO ] 2021-12-20 10:28:18.648 [main] DataProviderH2 - appendUser[]: Exception received No suitable driver found for h2_url
[INFO ] 2021-12-20 10:30:14.203 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-20 10:30:14.215 [main] DataProviderH2 - appendUser[1]: appending user User(id=22, name=Lisa)
[INFO ] 2021-12-20 10:30:14.217 [main] DataProviderH2 - jdbc:h2:C:\Users\legion\IdeaProjects\Log4jproject\src\main\resources\data\jdbc\Users;MV_STORE=false
[INFO ] 2021-12-20 10:30:14.219 [main] IdGenerator - generateForH2[1]: generating new id on h2_url
[INFO ] 2021-12-20 10:30:14.223 [main] DataProviderH2 - appendUser[]: Exception received No suitable driver found for h2_url
[INFO ] 2021-12-20 10:32:18.848 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-20 10:32:18.860 [main] DataProviderH2 - appendUser[1]: appending user User(id=22, name=Lisa)
[INFO ] 2021-12-20 10:32:18.863 [main] DataProviderH2 - jdbc:h2:C:\Users\legion\IdeaProjects\Log4jproject\src\main\resources\data\jdbc\Users;MV_STORE=false
[INFO ] 2021-12-20 10:32:18.865 [main] IdGenerator - generateForH2[1]: generating new id on h2_url
[INFO ] 2021-12-20 10:32:18.871 [main] DataProviderH2 - appendUser[]: Exception received No suitable driver found for h2_url
[INFO ] 2021-12-20 10:33:32.043 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-20 10:33:32.054 [main] DataProviderH2 - appendUser[1]: appending user User(id=22, name=Lisa)
[INFO ] 2021-12-20 10:33:32.056 [main] DataProviderH2 - null
[INFO ] 2021-12-20 10:33:32.058 [main] IdGenerator - generateForH2[1]: generating new id on jdbc:h2:C:\Users\legion\IdeaProjects\Log4jproject\src\main\resources\data\jdbc\Users;MV_STORE=false
[INFO ] 2021-12-20 10:33:32.678 [main] DataProviderH2 - appendUser[]: Exception received Неверное имя пользователя или пароль
Wrong user name or password [28000-200]
[INFO ] 2021-12-20 10:35:49.709 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-20 10:35:49.721 [main] DataProviderH2 - appendUser[1]: appending user User(id=22, name=Lisa)
[INFO ] 2021-12-20 10:35:49.723 [main] DataProviderH2 - null
[INFO ] 2021-12-20 10:35:49.724 [main] IdGenerator - generateForH2[1]: generating new id on jdbc:h2:C:\Users\legion\IdeaProjects\Log4jproject\src\main\resources\data\jdbc\Users;MV_STORE=false
[INFO ] 2021-12-20 10:35:50.340 [main] DataProviderH2 - appendUser[]: Exception received Неверное имя пользователя или пароль
Wrong user name or password [28000-200]
[INFO ] 2021-12-20 10:37:07.748 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-20 10:37:07.759 [main] DataProviderH2 - appendUser[1]: appending user User(id=22, name=Lisa)
[INFO ] 2021-12-20 10:37:07.761 [main] DataProviderH2 - jdbc:h2:C:/Users/legion/IdeaProjects/Log4jproject/src/main/resources/data/jdbc/Users;MV_STORE=false
[INFO ] 2021-12-20 10:37:07.761 [main] DataProviderH2 - NextGen
[INFO ] 2021-12-20 10:37:07.763 [main] IdGenerator - generateForH2[1]: generating new id on h2_url
[INFO ] 2021-12-20 10:37:07.768 [main] DataProviderH2 - appendUser[]: Exception received No suitable driver found for h2_url
[INFO ] 2021-12-20 10:39:06.068 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-20 10:39:06.081 [main] DataProviderH2 - appendUser[1]: appending user User(id=22, name=Lisa)
[INFO ] 2021-12-20 10:39:06.083 [main] DataProviderH2 - jdbc:h2:C:/Users/legion/IdeaProjects/Log4jproject/src/main/resources/data/jdbc/Users;MV_STORE=false
[INFO ] 2021-12-20 10:39:06.084 [main] DataProviderH2 - NextGen
[INFO ] 2021-12-20 10:39:06.084 [main] DataProviderH2 - qwerty
[INFO ] 2021-12-20 10:39:06.087 [main] IdGenerator - generateForH2[1]: generating new id on h2_url
[INFO ] 2021-12-20 10:39:06.092 [main] DataProviderH2 - appendUser[]: Exception received No suitable driver found for h2_url
[INFO ] 2021-12-20 10:41:13.562 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-20 10:41:13.574 [main] DataProviderH2 - appendUser[1]: appending user User(id=22, name=Lisa)
[INFO ] 2021-12-20 10:41:13.576 [main] DataProviderH2 - jdbc:h2:C:/Users/legion/IdeaProjects/Log4jproject/src/main/resources/data/jdbc/Users;MV_STORE=false
[INFO ] 2021-12-20 10:41:13.577 [main] DataProviderH2 - NextGen
[INFO ] 2021-12-20 10:41:13.577 [main] DataProviderH2 - qwerty
[INFO ] 2021-12-20 10:41:13.579 [main] IdGenerator - generateForH2[1]: generating new id on h2_url
[INFO ] 2021-12-20 10:41:13.585 [main] DataProviderH2 - appendUser[]: Exception received No suitable driver found for h2_url
[INFO ] 2021-12-20 10:42:00.477 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-20 10:42:00.489 [main] DataProviderH2 - appendUser[1]: appending user User(id=22, name=Lisa)
[INFO ] 2021-12-20 10:42:00.491 [main] DataProviderH2 - null
[INFO ] 2021-12-20 10:42:00.491 [main] DataProviderH2 - null
[INFO ] 2021-12-20 10:42:00.491 [main] DataProviderH2 - null
[INFO ] 2021-12-20 10:42:00.493 [main] IdGenerator - generateForH2[1]: generating new id on jdbc:h2:C:/Users/legion/IdeaProjects/Log4jproject/src/main/resources/data/jdbc/Users;MV_STORE=false
[INFO ] 2021-12-20 10:42:00.805 [main] IdGenerator - generateForH2[2]: generated id 7
[INFO ] 2021-12-20 10:42:00.851 [main] DataProviderH2 - appendUser[]: Exception received Cannot invoke "java.sql.Connection.close()" because "ru.sfedu.log4jproject.api.DataProviderH2.con" is null
[INFO ] 2021-12-20 10:43:28.744 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-20 10:43:28.756 [main] DataProviderH2 - appendUser[1]: appending user User(id=22, name=Pete)
[INFO ] 2021-12-20 10:43:28.758 [main] DataProviderH2 - null
[INFO ] 2021-12-20 10:43:28.758 [main] DataProviderH2 - null
[INFO ] 2021-12-20 10:43:28.758 [main] DataProviderH2 - null
[INFO ] 2021-12-20 10:43:28.760 [main] IdGenerator - generateForH2[1]: generating new id on jdbc:h2:C:/Users/legion/IdeaProjects/Log4jproject/src/main/resources/data/jdbc/Users;MV_STORE=false
[INFO ] 2021-12-20 10:43:29.076 [main] IdGenerator - generateForH2[2]: generated id 8
[INFO ] 2021-12-20 10:43:29.149 [main] DataProviderH2 - appendUser[2]: user appended
[INFO ] 2021-12-20 10:44:33.220 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-20 10:44:33.235 [main] DataProviderH2 - appendUser[1]: appending user User(id=22, name=Pete)
[INFO ] 2021-12-20 10:44:33.238 [main] DataProviderH2 - null
[INFO ] 2021-12-20 10:44:33.238 [main] DataProviderH2 - null
[INFO ] 2021-12-20 10:44:33.238 [main] DataProviderH2 - null
[INFO ] 2021-12-20 10:44:33.241 [main] IdGenerator - generateForH2[1]: generating new id on jdbc:h2:C:/Users/legion/IdeaProjects/Log4jproject/src/main/resources/data/jdbc/Users;MV_STORE=false
[INFO ] 2021-12-20 10:44:33.579 [main] IdGenerator - generateForH2[2]: generated id 9
[INFO ] 2021-12-20 10:44:33.660 [main] DataProviderH2 - appendUser[2]: user appended
[INFO ] 2021-12-20 10:46:33.871 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-20 10:46:33.887 [main] DataProviderH2 - appendUser[1]: appending user User(id=22, name=Drake)
[INFO ] 2021-12-20 10:46:33.889 [main] DataProviderH2 - null
[INFO ] 2021-12-20 10:46:33.889 [main] DataProviderH2 - NextGen
[INFO ] 2021-12-20 10:46:33.890 [main] DataProviderH2 - null
[INFO ] 2021-12-20 10:46:33.892 [main] IdGenerator - generateForH2[1]: generating new id on jdbc:h2:C:/Users/legion/IdeaProjects/Log4jproject/src/main/resources/data/jdbc/Users;MV_STORE=false
[INFO ] 2021-12-20 10:46:34.573 [main] DataProviderH2 - appendUser[]: Exception received Неверное имя пользователя или пароль
Wrong user name or password [28000-200]
[INFO ] 2021-12-20 10:47:38.531 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-20 10:47:38.546 [main] DataProviderH2 - appendUser[1]: appending user User(id=22, name=Drake)
[INFO ] 2021-12-20 10:47:38.548 [main] DataProviderH2 - jdbc:h2:C:/Users/legion/IdeaProjects/Log4jproject/src/main/resources/data/jdbc/Users;MV_STORE=false
[INFO ] 2021-12-20 10:47:38.549 [main] DataProviderH2 - NextGen
[INFO ] 2021-12-20 10:47:38.549 [main] DataProviderH2 - qwerty
[INFO ] 2021-12-20 10:47:38.552 [main] IdGenerator - generateForH2[1]: generating new id on url
[INFO ] 2021-12-20 10:47:38.560 [main] DataProviderH2 - appendUser[]: Exception received No suitable driver found for url
[INFO ] 2021-12-20 10:48:00.841 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-20 10:48:00.857 [main] DataProviderH2 - appendUser[1]: appending user User(id=22, name=Drake)
[INFO ] 2021-12-20 10:48:00.859 [main] DataProviderH2 - jdbc:h2:C:/Users/legion/IdeaProjects/Log4jproject/src/main/resources/data/jdbc/Users;MV_STORE=false
[INFO ] 2021-12-20 10:48:00.859 [main] DataProviderH2 - NextGen
[INFO ] 2021-12-20 10:48:00.859 [main] DataProviderH2 - qwerty
[INFO ] 2021-12-20 10:48:00.862 [main] IdGenerator - generateForH2[1]: generating new id on url
[INFO ] 2021-12-20 10:48:00.869 [main] DataProviderH2 - appendUser[]: Exception received No suitable driver found for url
[INFO ] 2021-12-20 10:51:59.992 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-20 10:52:00.004 [main] DataProviderH2 - appendUser[1]: appending user User(id=22, name=Drake)
[INFO ] 2021-12-20 10:52:00.006 [main] DataProviderH2 - jdbc:h2:C:/Users/legion/IdeaProjects/Log4jproject/src/main/resources/data/jdbc/Users;MV_STORE=false
[INFO ] 2021-12-20 10:52:00.006 [main] DataProviderH2 - NextGen
[INFO ] 2021-12-20 10:52:00.007 [main] DataProviderH2 - qwerty
[INFO ] 2021-12-20 10:52:00.009 [main] IdGenerator - generateForH2[1]: generating new id on url
[INFO ] 2021-12-20 10:52:00.014 [main] DataProviderH2 - appendUser[]: Exception received No suitable driver found for url
[INFO ] 2021-12-20 10:53:15.573 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-20 10:53:15.585 [main] DataProviderH2 - appendUser[1]: appending user User(id=22, name=Drake)
[INFO ] 2021-12-20 10:53:15.587 [main] DataProviderH2 - jdbc:h2:C://Users//legion//IdeaProjects//Log4jproject//src//main//resources//data//jdbc//Users;MV_STORE=false
[INFO ] 2021-12-20 10:53:15.587 [main] DataProviderH2 - NextGen
[INFO ] 2021-12-20 10:53:15.587 [main] DataProviderH2 - qwerty
[INFO ] 2021-12-20 10:53:15.590 [main] IdGenerator - generateForH2[1]: generating new id on url
[INFO ] 2021-12-20 10:53:15.596 [main] DataProviderH2 - appendUser[]: Exception received No suitable driver found for url
[INFO ] 2021-12-20 10:54:17.939 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-20 10:54:17.949 [main] DataProviderH2 - appendUser[1]: appending user User(id=22, name=Drake)
[INFO ] 2021-12-20 10:54:17.951 [main] DataProviderH2 - jdbc:h2:C:/Users/legion/IdeaProjects/Log4jproject/src/main/resources/data/jdbc/Users;MV_STORE=false
[INFO ] 2021-12-20 10:54:17.951 [main] DataProviderH2 - NextGen
[INFO ] 2021-12-20 10:54:17.951 [main] DataProviderH2 - qwerty
[INFO ] 2021-12-20 10:54:17.953 [main] IdGenerator - generateForH2[1]: generating new id on url
[INFO ] 2021-12-20 10:54:17.958 [main] DataProviderH2 - appendUser[]: Exception received No suitable driver found for url
[INFO ] 2021-12-20 10:55:11.784 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-20 10:55:11.794 [main] DataProviderH2 - appendUser[1]: appending user User(id=22, name=Drake)
[INFO ] 2021-12-20 10:55:11.796 [main] DataProviderH2 - null
[INFO ] 2021-12-20 10:55:11.797 [main] DataProviderH2 - NextGen
[INFO ] 2021-12-20 10:55:11.797 [main] DataProviderH2 - qwerty
[INFO ] 2021-12-20 10:55:11.798 [main] IdGenerator - generateForH2[1]: generating new id on jdbc:h2:C:/Users/legion/IdeaProjects/Log4jproject/src/main/resources/data/jdbc/Users;MV_STORE=false
[INFO ] 2021-12-20 10:55:12.479 [main] DataProviderH2 - appendUser[]: Exception received Неверное имя пользователя или пароль
Wrong user name or password [28000-200]
[INFO ] 2021-12-20 10:55:42.649 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-20 10:55:42.662 [main] DataProviderH2 - appendUser[1]: appending user User(id=22, name=Drake)
[INFO ] 2021-12-20 10:55:42.663 [main] DataProviderH2 - null
[INFO ] 2021-12-20 10:55:42.664 [main] DataProviderH2 - null
[INFO ] 2021-12-20 10:55:42.664 [main] DataProviderH2 - qwerty
[INFO ] 2021-12-20 10:55:42.666 [main] IdGenerator - generateForH2[1]: generating new id on jdbc:h2:C:/Users/legion/IdeaProjects/Log4jproject/src/main/resources/data/jdbc/Users;MV_STORE=false
[INFO ] 2021-12-20 10:55:43.345 [main] DataProviderH2 - appendUser[]: Exception received Неверное имя пользователя или пароль
Wrong user name or password [28000-200]
[INFO ] 2021-12-20 10:56:01.279 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-20 10:56:01.292 [main] DataProviderH2 - appendUser[1]: appending user User(id=22, name=Drake)
[INFO ] 2021-12-20 10:56:01.294 [main] DataProviderH2 - null
[INFO ] 2021-12-20 10:56:01.294 [main] DataProviderH2 - null
[INFO ] 2021-12-20 10:56:01.294 [main] DataProviderH2 - qwerty
[INFO ] 2021-12-20 10:56:01.296 [main] IdGenerator - generateForH2[1]: generating new id on jdbc:h2:C:/Users/legion/IdeaProjects/Log4jproject/src/main/resources/data/jdbc/Users;MV_STORE=false
[INFO ] 2021-12-20 10:56:01.927 [main] DataProviderH2 - appendUser[]: Exception received Неверное имя пользователя или пароль
Wrong user name or password [28000-200]
[INFO ] 2021-12-20 10:56:20.703 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-20 10:56:20.714 [main] DataProviderH2 - appendUser[1]: appending user User(id=22, name=Drake)
[INFO ] 2021-12-20 10:56:20.716 [main] DataProviderH2 - null
[INFO ] 2021-12-20 10:56:20.716 [main] DataProviderH2 - null
[INFO ] 2021-12-20 10:56:20.717 [main] DataProviderH2 - null
[INFO ] 2021-12-20 10:56:20.719 [main] IdGenerator - generateForH2[1]: generating new id on jdbc:h2:C:/Users/legion/IdeaProjects/Log4jproject/src/main/resources/data/jdbc/Users;MV_STORE=false
[INFO ] 2021-12-20 10:56:21.062 [main] IdGenerator - generateForH2[2]: generated id 10
[INFO ] 2021-12-20 10:56:21.142 [main] DataProviderH2 - appendUser[2]: user appended
[INFO ] 2021-12-20 10:56:53.526 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-20 10:56:53.539 [main] DataProviderH2 - appendUser[1]: appending user User(id=22, name=Drake)
[INFO ] 2021-12-20 10:56:53.540 [main] DataProviderH2 - null
[INFO ] 2021-12-20 10:56:53.541 [main] DataProviderH2 - null
[INFO ] 2021-12-20 10:56:53.541 [main] DataProviderH2 - qwerty
[INFO ] 2021-12-20 10:56:53.542 [main] IdGenerator - generateForH2[1]: generating new id on jdbc:h2:C:/Users/legion/IdeaProjects/Log4jproject/src/main/resources/data/jdbc/Users;MV_STORE=false
[INFO ] 2021-12-20 10:56:54.160 [main] DataProviderH2 - appendUser[]: Exception received Неверное имя пользователя или пароль
Wrong user name or password [28000-200]
[INFO ] 2021-12-20 10:57:47.693 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-20 10:57:47.709 [main] DataProviderH2 - appendUser[1]: appending user User(id=22, name=Drake)
[INFO ] 2021-12-20 10:57:47.711 [main] DataProviderH2 - null
[INFO ] 2021-12-20 10:57:47.712 [main] DataProviderH2 - null
[INFO ] 2021-12-20 10:57:47.712 [main] DataProviderH2 - qwerty
[INFO ] 2021-12-20 10:57:47.714 [main] IdGenerator - generateForH2[1]: generating new id on jdbc:h2:C:/Users/legion/IdeaProjects/Log4jproject/src/main/resources/data/jdbc/Users;MV_STORE=false
[INFO ] 2021-12-20 10:57:48.400 [main] DataProviderH2 - appendUser[]: Exception received Неверное имя пользователя или пароль
Wrong user name or password [28000-200]
[INFO ] 2021-12-20 11:00:38.893 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-20 11:00:38.905 [main] DataProviderH2 - appendUser[1]: appending user User(id=22, name=Drake)
[INFO ] 2021-12-20 11:00:38.906 [main] DataProviderH2 - null
[INFO ] 2021-12-20 11:00:38.906 [main] DataProviderH2 - null
[INFO ] 2021-12-20 11:00:38.906 [main] DataProviderH2 - null
[INFO ] 2021-12-20 11:00:38.909 [main] IdGenerator - generateForH2[1]: generating new id on jdbc:h2:C:/Users/legion/IdeaProjects/Log4jproject/src/main/resources/data/jdbc/Users;MV_STORE=false
[INFO ] 2021-12-20 11:00:39.235 [main] IdGenerator - generateForH2[2]: generated id 11
[INFO ] 2021-12-20 11:00:39.314 [main] DataProviderH2 - appendUser[2]: user appended
[INFO ] 2021-12-20 11:02:27.538 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-20 11:02:27.551 [main] DataProviderH2 - appendUser[1]: appending user User(id=22, name=Drake)
[INFO ] 2021-12-20 11:02:27.553 [main] DataProviderH2 - null
[INFO ] 2021-12-20 11:02:27.553 [main] DataProviderH2 - null
[INFO ] 2021-12-20 11:02:27.553 [main] DataProviderH2 - null
[INFO ] 2021-12-20 11:02:27.555 [main] IdGenerator - generateForH2[1]: generating new id on jdbc:h2:C:/Users/legion/IdeaProjects/Log4jproject/src/main/resources/data/jdbc/Users;MV_STORE=false
[INFO ] 2021-12-20 11:02:27.907 [main] IdGenerator - generateForH2[2]: generated id 12
[INFO ] 2021-12-20 11:02:27.907 [main] DataProviderH2 - appendUser[]: Exception received The url cannot be null
[INFO ] 2021-12-20 11:03:19.528 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-20 11:03:19.541 [main] DataProviderH2 - appendUser[1]: appending user User(id=22, name=Drake)
[INFO ] 2021-12-20 11:03:19.542 [main] DataProviderH2 - jdbc:h2:C:/Users/legion/IdeaProjects/Log4jproject/src/main/resources/data/jdbc/Users;MV_STORE=false
[INFO ] 2021-12-20 11:03:19.543 [main] DataProviderH2 - NextGen
[INFO ] 2021-12-20 11:03:19.543 [main] DataProviderH2 - qwerty
[INFO ] 2021-12-20 11:03:19.545 [main] IdGenerator - generateForH2[1]: generating new id on url
[INFO ] 2021-12-20 11:03:19.550 [main] DataProviderH2 - appendUser[]: Exception received No suitable driver found for url
[INFO ] 2021-12-20 11:04:03.851 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-20 11:04:03.865 [main] DataProviderH2 - appendUser[1]: appending user User(id=22, name=Drake)
[INFO ] 2021-12-20 11:04:03.868 [main] IdGenerator - generateForH2[1]: generating new id on url
[INFO ] 2021-12-20 11:04:03.872 [main] DataProviderH2 - appendUser[]: Exception received No suitable driver found for url
[INFO ] 2021-12-20 11:04:35.216 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-20 11:04:35.227 [main] DataProviderH2 - appendUser[1]: appending user User(id=22, name=Drake)
[INFO ] 2021-12-20 11:04:35.230 [main] DataProviderH2 - jdbc:h2:C:/Users/legion/IdeaProjects/Log4jproject/src/main/resources/data/jdbc/Users;MV_STORE=false
[INFO ] 2021-12-20 11:04:35.230 [main] DataProviderH2 - NextGen
[INFO ] 2021-12-20 11:04:35.230 [main] DataProviderH2 - qwerty
[INFO ] 2021-12-20 11:04:35.566 [main] DataProviderH2 - appendUser[2]: user appended
[INFO ] 2021-12-20 11:07:37.718 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-20 11:07:37.741 [main] DataProviderH2 - appendUser[1]: appending user User(id=25, name=Noah)
[INFO ] 2021-12-20 11:07:37.746 [main] IdGenerator - generateForH2[1]: generating new id on url
[INFO ] 2021-12-20 11:07:38.134 [main] IdGenerator - generateForH2[2]: generated id 23
[INFO ] 2021-12-20 11:07:38.223 [main] DataProviderH2 - appendUser[2]: user appended
[INFO ] 2021-12-20 11:09:54.670 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-20 11:09:54.683 [main] DataProviderH2 - appendUser[1]: appending user User(id=25, name=Phil)
[INFO ] 2021-12-20 11:09:54.685 [main] IdGenerator - generateForH2[1]: generating new id on url
[INFO ] 2021-12-20 11:09:55.031 [main] IdGenerator - generateForH2[2]: generated id 24
[INFO ] 2021-12-20 11:09:55.105 [main] DataProviderH2 - appendUser[2]: user appended
[INFO ] 2021-12-20 11:11:29.274 [main] DataProviderH2 - getUserById test success
[INFO ] 2021-12-20 11:11:29.282 [main] DataProviderH2 - getUserById[1]: getting user by id 11
[INFO ] 2021-12-20 11:11:29.596 [main] DataProviderH2 - getUserById[2]: received user User(id=11, name=Drake)
[INFO ] 2021-12-20 11:12:47.635 [main] DataProviderH2 - deleteUserById test Success
[INFO ] 2021-12-20 11:12:47.640 [main] DataProviderH2 - deleteUserById[1]: deleting user by id 22
[INFO ] 2021-12-20 11:12:47.975 [main] DataProviderH2 - deleteUserById[2]: user deleted
[INFO ] 2021-12-21 18:25:41.558 [main] DataProviderCsvTest - AppendUser test Success
[INFO ] 2021-12-21 18:25:41.577 [main] DataProviderCsvOld - appendUser [start] appending user User(id=15, name=Jane)
[INFO ] 2021-12-21 18:25:41.577 [main] DataProviderCsvOld - getUserById [start] id:15
[INFO ] 2021-12-21 18:25:41.739 [main] DataProviderCsvOld - getUserById [end]
[INFO ] 2021-12-21 18:25:41.748 [main] DataProviderCsvOld - append User [end] appended User(id=15, name=Jane)
[INFO ] 2021-12-21 18:30:02.251 [main] DataProviderCsvTest - AppendUser test Fail
[INFO ] 2021-12-21 18:30:02.265 [main] DataProviderCsvOld - appendUser [start] appending user User(id=0, name=null)
[INFO ] 2021-12-21 18:30:02.265 [main] DataProviderCsvOld - Empty user
[INFO ] 2021-12-21 18:30:02.269 [main] DataProviderCsvTest - appendUser test Fail - Success
[INFO ] 2021-12-21 18:32:18.495 [main] DataProviderCsvTest - getUserById test Success
[INFO ] 2021-12-21 18:32:18.504 [main] DataProviderCsvOld - getUserById [start] id:3
[INFO ] 2021-12-21 18:32:18.622 [main] DataProviderCsvOld - getUserById [end]
[INFO ] 2021-12-21 18:32:18.626 [main] DataProviderCsvTest - User(id=3, name=Dmitriy)
[INFO ] 2021-12-21 18:32:31.279 [main] DataProviderCsvTest - getUserById test Fail
[INFO ] 2021-12-21 18:32:31.289 [main] DataProviderCsvOld - getUserById [start] id:5
[INFO ] 2021-12-21 18:32:31.395 [main] DataProviderCsvOld - getUserById [end]
[INFO ] 2021-12-21 18:32:31.398 [main] DataProviderCsvTest - User(id=0, name=Empty)
[INFO ] 2021-12-21 18:32:31.398 [main] DataProviderCsvTest - getUserById test Fail - Success
[INFO ] 2021-12-21 18:32:57.487 [main] DataProviderCsvTest - deleteUserById test Success
[INFO ] 2021-12-21 18:32:57.498 [main] DataProviderCsvOld - deleteUserById [start] id:15
[INFO ] 2021-12-21 18:32:57.524 [main] DataProviderCsvOld - ./src/main/resources/data/csv/user.csv has been deleted
[INFO ] 2021-12-21 18:32:57.526 [main] DataProviderCsvOld - ./src/main/resources/data/csv/newuser.csv has been written and renamed to ./src/main/resources/data/csv/user.csv
[INFO ] 2021-12-21 18:33:10.251 [main] DataProviderCsvTest - deleteUserById test Fail
[INFO ] 2021-12-21 18:33:10.259 [main] DataProviderCsvOld - deleteUserById [start] id:5
[INFO ] 2021-12-21 18:33:10.280 [main] DataProviderCsvOld - ./src/main/resources/data/csv/user.csv has been deleted
[INFO ] 2021-12-21 18:33:10.282 [main] DataProviderCsvOld - ./src/main/resources/data/csv/newuser.csv has been written and renamed to ./src/main/resources/data/csv/user.csv
[INFO ] 2021-12-21 18:33:10.283 [main] DataProviderCsvTest - deleteUserById test Fail - Success
[INFO ] 2021-12-21 18:40:02.235 [main] DataProviderCsvTest - updateUser test Success
[INFO ] 2021-12-21 18:40:02.249 [main] DataProviderCsvOld - updateUser [start] user to update User(id=4, name=Jane)
[INFO ] 2021-12-21 18:40:02.249 [main] DataProviderCsvOld - getUserById [start] id:4
[INFO ] 2021-12-21 18:40:02.422 [main] DataProviderCsvOld - getUserById [end]
[INFO ] 2021-12-21 18:40:02.423 [main] DataProviderCsvOld - Entry found by given id Result{message='null', entity=User(id=4, name=Daria), code=SUCCESS}
[INFO ] 2021-12-21 18:40:02.423 [main] DataProviderCsvOld - deleteUserById [start] id:4
[INFO ] 2021-12-21 18:40:02.427 [main] DataProviderCsvOld - ./src/main/resources/data/csv/user.csv has been deleted
[INFO ] 2021-12-21 18:40:02.428 [main] DataProviderCsvOld - ./src/main/resources/data/csv/newuser.csv has been written and renamed to ./src/main/resources/data/csv/user.csv
[INFO ] 2021-12-21 18:40:02.429 [main] DataProviderCsvOld - appendUsers [start]
[INFO ] 2021-12-21 18:40:02.444 [main] DataProviderCsvOld - appendUsers created:[User(id=4, name=Jane)]
[INFO ] 2021-12-21 18:40:02.444 [main] DataProviderCsvOld - appendUsers [end]
[INFO ] 2021-12-21 18:40:02.445 [main] DataProviderCsvOld - Entry successfully updated
[INFO ] 2021-12-21 18:40:26.063 [main] DataProviderCsvTest - updateUser test Success
[INFO ] 2021-12-21 18:40:26.080 [main] DataProviderCsvOld - updateUser [start] user to update User(id=0, name=null)
[INFO ] 2021-12-21 18:40:26.080 [main] DataProviderCsvOld - Empty user
[INFO ] 2021-12-21 18:40:26.082 [main] DataProviderCsvTest - updateUser test Fail - Success
[INFO ] 2021-12-21 18:48:42.402 [main] DataProviderCsvTest - AppendUser test Success
[INFO ] 2021-12-21 18:48:42.420 [main] DataProviderCsv - appendUser[1]: appending user User(id=15, name=Jane)
[INFO ] 2021-12-21 18:48:42.422 [main] DataProviderCsv - getUserById[1]: id:15
[INFO ] 2021-12-21 18:48:42.555 [main] DataProviderCsv - getUserById[2]: Got user User(id=0, name=null)
[INFO ] 2021-12-21 18:48:42.556 [main] DataProviderCsv - getUserById[3]: No entry found
[INFO ] 2021-12-21 18:48:42.560 [main] IdGenerator - generateUserCsvId[1]: Generating id for file user
[INFO ] 2021-12-21 18:48:42.566 [main] IdGenerator - generateUserCsvId[2]: Generated id 5
[INFO ] 2021-12-21 18:48:42.577 [main] DataProviderCsv - appendUser[2]: appended User(id=5, name=Jane)
[INFO ] 2021-12-21 18:48:42.603 [main] DataProviderCsvTest - AppendUser test Fail
[INFO ] 2021-12-21 18:48:42.603 [main] DataProviderCsv - appendUser[1]: appending user User(id=0, name=null)
[ERROR] 2021-12-21 18:48:42.604 [main] Validator - Invalid user
[ERROR] 2021-12-21 18:48:42.604 [main] DataProviderCsv - appendUser: Invalid user
[INFO ] 2021-12-21 18:48:42.604 [main] DataProviderCsvTest - appendUser test Fail - Success
[INFO ] 2021-12-21 18:48:42.607 [main] DataProviderCsvTest - deleteUserById test Success
[INFO ] 2021-12-21 18:48:42.607 [main] DataProviderCsv - deleteUserById[1]: id:15
[INFO ] 2021-12-21 18:48:42.609 [main] CsvUtil - swapFiles[1]: swapping for file user
[INFO ] 2021-12-21 18:48:42.609 [main] CsvUtil - swapFiles[2]: .\src\main\resources\data\csv\user.csv has been deleted
[INFO ] 2021-12-21 18:48:42.611 [main] CsvUtil - swapFiles[3]: .\src\main\resources\data\csv\userNew.csv has been written and renamed to .\src\main\resources\data\csv\user.csv
[INFO ] 2021-12-21 18:48:42.614 [main] DataProviderCsvTest - updateUser test Success
[INFO ] 2021-12-21 18:48:42.614 [main] DataProviderCsv - updateUser[1]: user to update User(id=4, name=Jane)
[INFO ] 2021-12-21 18:48:42.614 [main] DataProviderCsv - deleteUserById[1]: id:4
[INFO ] 2021-12-21 18:48:42.615 [main] CsvUtil - swapFiles[1]: swapping for file user
[INFO ] 2021-12-21 18:48:42.616 [main] CsvUtil - swapFiles[2]: .\src\main\resources\data\csv\user.csv has been deleted
[INFO ] 2021-12-21 18:48:42.617 [main] CsvUtil - swapFiles[3]: .\src\main\resources\data\csv\userNew.csv has been written and renamed to .\src\main\resources\data\csv\user.csv
[INFO ] 2021-12-21 18:48:42.617 [main] DataProviderCsv - appendUser[1]: appending user User(id=4, name=Jane)
[INFO ] 2021-12-21 18:48:42.618 [main] DataProviderCsv - getUserById[1]: id:4
[INFO ] 2021-12-21 18:48:42.623 [main] DataProviderCsv - getUserById[2]: Got user User(id=0, name=null)
[INFO ] 2021-12-21 18:48:42.623 [main] DataProviderCsv - getUserById[3]: No entry found
[INFO ] 2021-12-21 18:48:42.624 [main] IdGenerator - generateUserCsvId[1]: Generating id for file user
[INFO ] 2021-12-21 18:48:42.629 [main] IdGenerator - generateUserCsvId[2]: Generated id 6
[INFO ] 2021-12-21 18:48:42.632 [main] DataProviderCsv - appendUser[2]: appended User(id=6, name=Jane)
[INFO ] 2021-12-21 18:48:42.632 [main] DataProviderCsv - updateUser[2]: Entry successfully updated
[INFO ] 2021-12-21 18:48:42.634 [main] DataProviderCsvTest - updateUser test Success
[INFO ] 2021-12-21 18:48:42.635 [main] DataProviderCsv - updateUser[1]: user to update User(id=0, name=null)
[ERROR] 2021-12-21 18:48:42.635 [main] Validator - Invalid user
[ERROR] 2021-12-21 18:48:42.635 [main] DataProviderCsv - updateUser: Exception received Invalid user
[INFO ] 2021-12-21 18:48:42.635 [main] DataProviderCsvTest - updateUser test Fail - Success
[INFO ] 2021-12-21 18:48:42.637 [main] DataProviderCsvTest - getUserById test Success
[INFO ] 2021-12-21 18:48:42.638 [main] DataProviderCsv - getUserById[1]: id:3
[INFO ] 2021-12-21 18:48:42.643 [main] DataProviderCsv - getUserById[2]: Got user User(id=3, name=Dmitriy)
[INFO ] 2021-12-21 18:48:42.643 [main] DataProviderCsvTest - User(id=3, name=Dmitriy)
[INFO ] 2021-12-21 18:48:42.646 [main] DataProviderCsvTest - deleteUserById test Fail
[INFO ] 2021-12-21 18:48:42.646 [main] DataProviderCsv - deleteUserById[1]: id:5
[INFO ] 2021-12-21 18:48:42.648 [main] CsvUtil - swapFiles[1]: swapping for file user
[ERROR] 2021-12-21 18:48:42.649 [main] CsvUtil - swapFiles: exception received File is not deleted
[INFO ] 2021-12-21 18:48:42.649 [main] DataProviderCsvTest - deleteUserById test Fail - Success
[INFO ] 2021-12-21 18:48:42.652 [main] DataProviderCsvTest - getUserById test Fail
[INFO ] 2021-12-21 18:48:42.652 [main] DataProviderCsv - getUserById[1]: id:5
[INFO ] 2021-12-21 18:48:42.657 [main] DataProviderCsv - getUserById[2]: Got user User(id=5, name=Jane)
[INFO ] 2021-12-21 18:48:42.658 [main] DataProviderCsvTest - User(id=5, name=Jane)
[INFO ] 2021-12-21 18:48:42.658 [main] DataProviderCsvTest - getUserById test Fail - Success
[INFO ] 2021-12-22 18:11:46.258 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-22 18:11:46.270 [main] DataProviderH2 - appendUser[1]: appending user User(id=51, name=Duke)
[INFO ] 2021-12-22 18:11:46.273 [main] IdGenerator - generateForH2[1]: generating new id on url
[INFO ] 2021-12-22 18:11:46.650 [main] IdGenerator - generateForH2[2]: generated id 25
[INFO ] 2021-12-22 18:11:46.724 [main] DataProviderH2 - appendUser[2]: user appended
[INFO ] 2021-12-22 18:19:05.479 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-22 18:19:05.489 [main] DataProviderH2 - appendUser[1]: appending user User(id=51, name=Leah)
[INFO ] 2021-12-22 18:19:05.492 [main] IdGenerator - generateForH2[1]: generating new id on database
[INFO ] 2021-12-22 18:19:05.826 [main] IdGenerator - generateForH2[2]: generated id 26
[INFO ] 2021-12-22 18:19:05.905 [main] DataProviderH2 - appendUser[2]: user appended
[INFO ] 2021-12-22 18:22:09.429 [main] DataProviderH2 - getUserById test success
[INFO ] 2021-12-22 18:22:09.434 [main] DataProviderH2 - getUserById[1]: getting user by id 4
[INFO ] 2021-12-22 18:22:09.704 [main] DataProviderH2 - getUserById[2]: received user User(id=4, name=Fred)
[INFO ] 2021-12-22 18:22:21.275 [main] DataProviderH2 - getUserById test Fail
[INFO ] 2021-12-22 18:22:21.282 [main] DataProviderH2 - getUserById[1]: getting user by id 580
[ERROR] 2021-12-22 18:22:21.560 [main] Validator - Invalid user
[INFO ] 2021-12-22 18:22:21.560 [main] DataProviderH2 - getUserById[]: exception received Invalid user
[INFO ] 2021-12-22 18:22:44.477 [main] DataProviderH2 - deleteUserById test Success
[INFO ] 2021-12-22 18:22:44.483 [main] DataProviderH2 - deleteUserById[1]: deleting user by id 6
[INFO ] 2021-12-22 18:22:44.820 [main] DataProviderH2 - deleteUserById[2]: user deleted
[INFO ] 2021-12-22 18:23:08.798 [main] DataProviderH2 - deleteUserById test Fail
[INFO ] 2021-12-22 18:23:08.803 [main] DataProviderH2 - deleteUserById[1]: deleting user by id 35
[INFO ] 2021-12-22 18:23:09.104 [main] DataProviderH2 - deleteUserById[2]: user deleted
[INFO ] 2021-12-22 18:23:51.564 [main] DataProviderH2 - updateUser test Success
[INFO ] 2021-12-22 18:23:51.574 [main] DataProviderH2 - updateUser[1]: updating user User(id=9, name=Ada)
[INFO ] 2021-12-22 18:23:51.859 [main] DataProviderH2 - getUserById[1]: getting user by id 9
[INFO ] 2021-12-22 18:23:51.862 [main] DataProviderH2 - getUserById[2]: received user User(id=9, name=Pete)
[INFO ] 2021-12-22 18:23:51.868 [main] DataProviderH2 - updateUser[2]: user updated
[INFO ] 2021-12-22 18:24:23.238 [main] DataProviderH2 - updateUser test Fail
[INFO ] 2021-12-22 18:24:23.249 [main] DataProviderH2 - updateUser[1]: updating user User(id=0, name=null)
[ERROR] 2021-12-22 18:24:23.250 [main] Validator - Invalid user
[INFO ] 2021-12-22 18:24:23.251 [main] DataProviderH2 - update[]: exception received Invalid user
[INFO ] 2021-12-22 18:24:42.964 [main] DataProviderH2 - updateUser test Fail
[INFO ] 2021-12-22 18:24:42.982 [main] DataProviderH2 - updateUser[1]: updating user User(id=56, name=Tony)
[INFO ] 2021-12-22 18:24:43.307 [main] DataProviderH2 - getUserById[1]: getting user by id 56
[ERROR] 2021-12-22 18:24:43.309 [main] Validator - Invalid user
[INFO ] 2021-12-22 18:24:43.309 [main] DataProviderH2 - getUserById[]: exception received Invalid user
[INFO ] 2021-12-22 18:24:43.311 [main] DataProviderH2 - appendUser[1]: appending user User(id=56, name=Tony)
[INFO ] 2021-12-22 18:24:43.312 [main] IdGenerator - generateForH2[1]: generating new id on database
[INFO ] 2021-12-22 18:24:43.313 [main] IdGenerator - generateForH2[2]: generated id 27
[INFO ] 2021-12-22 18:24:43.317 [main] DataProviderH2 - appendUser[2]: user appended
[INFO ] 2021-12-22 18:24:43.317 [main] DataProviderH2 - updateUser[2]: user updated
[INFO ] 2021-12-22 20:53:03.478 [main] MongoProvider - workWithMongo[1]: working with {password=root, dbname=admin, port=27017, host=localhost, login=root, table=users}
[INFO ] 2021-12-22 20:53:03.630 [main] MongoProvider - save[1]: command = UPDATING, type = XML, object = User(id=1, name=Louren), dbName = {}
[ERROR] 2021-12-22 20:53:03.633 [main] MongoProvider - save[2]: Exception received can't serialize class ru.sfedu.log4jproject.model.beans.User
[INFO ] 2021-12-22 21:00:10.101 [main] MongoProvider - workWithMongo[1]: working with {password=root, dbname=admin, port=27017, host=localhost, login=root, table=users}
[INFO ] 2021-12-22 21:00:10.220 [main] MongoProvider - save[1]: command = UPDATING, type = XML, object = SUCCESS, dbName = {}
[ERROR] 2021-12-22 21:00:10.225 [main] MongoProvider - save[2]: Exception received can't serialize class ru.sfedu.log4jproject.model.CodeType
[INFO ] 2021-12-22 21:04:31.201 [main] MongoProvider - workWithMongo[1]: working with {password=root, dbname=admin, port=27017, host=localhost, login=root, table=users}
[INFO ] 2021-12-22 21:04:31.317 [main] MongoProvider - save[1]: command = UPDATING, type = XML, object = Success, dbName = {}
[INFO ] 2021-12-22 21:05:48.183 [main] MongoProvider - workWithMongo[1]: working with {password=root, dbname=admin, port=27017, host=localhost, login=root, table=users}
[INFO ] 2021-12-22 21:05:48.313 [main] MongoProvider - save[1]: command = UPDATING, type = XML, object = User(id=1, name=Susan), dbName = {}
[INFO ] 2021-12-22 21:07:50.810 [main] MongoProvider - workWithMongo[1]: working with {password=root, dbname=admin, port=27017, host=localhost, login=root, table=users}
[INFO ] 2021-12-22 21:07:50.940 [main] MongoProvider - save[1]: command = UPDATING, type = XML, object = Result{message='Some message', entity=User(id=1, name=Susan), code=SUCCESS}, dbName = {}
[INFO ] 2021-12-23 17:59:23.935 [main] MongoProvider - workWithMongo[1]: working with {password=root, dbname=admin, port=27017, host=localhost, login=root, table=users}
[INFO ] 2021-12-23 17:59:24.076 [main] MongoProvider - save[1]: command = UPDATING, type = XML, object = Result{message='Some message', entity=User(id=1, name=Susan), code=SUCCESS}
[INFO ] 2021-12-23 18:05:14.338 [main] MongoProvider - workWithMongo[1]: working with {password=root, dbname=admin, port=27017, host=localhost, login=root, table=users}
[INFO ] 2021-12-23 18:05:14.470 [main] MongoProvider - save[1]: command = UPDATING, type = XML, object = Result{message='null', entity=null, code=null}
[INFO ] 2021-12-23 18:06:19.465 [main] MongoProvider - workWithMongo[1]: working with {password=root, dbname=admin, port=27017, host=localhost, login=root, table=users}
[INFO ] 2021-12-23 18:06:19.615 [main] MongoProvider - save[1]: command = UPDATING, type = XML, object = User(id=0, name=null)
[INFO ] 2021-12-24 21:28:39.304 [main] DataProviderCsvTest - AppendUser test Success
[INFO ] 2021-12-24 21:28:39.318 [main] DataProviderCsv - appendUser[1]: appending user User(id=15, name=Norman)
[INFO ] 2021-12-24 21:28:39.319 [main] DataProviderCsv - getUserById[1]: id:15
[INFO ] 2021-12-24 21:28:39.450 [main] DataProviderCsv - getUserById[2]: Got user User(id=0, name=null)
[INFO ] 2021-12-24 21:28:39.451 [main] DataProviderCsv - getUserById[3]: No entry found
[INFO ] 2021-12-24 21:28:39.455 [main] IdGenerator - generateUserCsvId[1]: Generating id for file user
[INFO ] 2021-12-24 21:28:39.460 [main] IdGenerator - generateUserCsvId[2]: Generated id 7
[INFO ] 2021-12-24 21:28:39.472 [main] DataProviderCsv - appendUser[2]: appended User(id=7, name=Norman)
[INFO ] 2021-12-24 21:28:57.444 [main] DataProviderXmlTest - appendUser test Fail
[INFO ] 2021-12-24 21:28:57.457 [main] DataProviderXml - appendUser[1]: writing user User(id=0, name=null)
[ERROR] 2021-12-24 21:28:57.461 [main] Validator - Invalid user
[ERROR] 2021-12-24 21:28:57.461 [main] DataProviderXml - appendUser[2]: exception received Invalid user
[INFO ] 2021-12-24 21:29:14.940 [main] DataProviderXmlTest - getUserById test Success
[INFO ] 2021-12-24 21:29:14.947 [main] DataProviderXml - getUserById[1]: searching for id=5
[INFO ] 2021-12-24 21:29:15.053 [main] DataProviderXml - getUserById[2]: searching result Optional[User(id=5, name=Fedya)]
[INFO ] 2021-12-24 21:29:29.415 [main] DataProviderXmlTest - getUserById test Fail
[INFO ] 2021-12-24 21:29:29.423 [main] DataProviderXml - getUserById[1]: searching for id=1
[INFO ] 2021-12-24 21:29:29.535 [main] DataProviderXml - getUserById[2]: searching result Optional[User(id=1, name=Ivan)]
[INFO ] 2021-12-24 21:30:14.125 [main] DataProviderCsvTest - AppendUser test Fail
[INFO ] 2021-12-24 21:30:14.136 [main] DataProviderCsv - appendUser[1]: appending user User(id=0, name=null)
[ERROR] 2021-12-24 21:30:14.138 [main] Validator - Invalid user
[ERROR] 2021-12-24 21:30:14.138 [main] DataProviderCsv - appendUser: Invalid user
[INFO ] 2021-12-24 21:30:14.140 [main] DataProviderCsvTest - appendUser test Fail - Success
[INFO ] 2021-12-24 21:30:33.446 [main] DataProviderCsvTest - getUserById test Success
[INFO ] 2021-12-24 21:30:33.454 [main] DataProviderCsv - getUserById[1]: id:6
[INFO ] 2021-12-24 21:30:33.581 [main] DataProviderCsv - getUserById[2]: Got user User(id=6, name=Jane)
[INFO ] 2021-12-24 21:30:33.584 [main] DataProviderCsvTest - User(id=6, name=Jane)
[INFO ] 2021-12-24 21:30:57.754 [main] DataProviderCsvTest - deleteUserById test Success
[INFO ] 2021-12-24 21:30:57.771 [main] DataProviderCsv - deleteUserById[1]: id:5
[INFO ] 2021-12-24 21:30:57.795 [main] CsvUtil - swapFiles[1]: swapping for file user
[INFO ] 2021-12-24 21:30:57.796 [main] CsvUtil - swapFiles[2]: .\src\main\resources\data\csv\user.csv has been deleted
[INFO ] 2021-12-24 21:30:57.797 [main] CsvUtil - swapFiles[3]: .\src\main\resources\data\csv\userNew.csv has been written and renamed to .\src\main\resources\data\csv\user.csv
[INFO ] 2021-12-24 21:31:23.955 [main] DataProviderCsvTest - updateUser test Success
[INFO ] 2021-12-24 21:31:23.965 [main] DataProviderCsv - updateUser[1]: user to update User(id=3, name=Lewis)
[INFO ] 2021-12-24 21:31:23.967 [main] DataProviderCsv - deleteUserById[1]: id:3
[INFO ] 2021-12-24 21:31:23.985 [main] CsvUtil - swapFiles[1]: swapping for file user
[INFO ] 2021-12-24 21:31:23.986 [main] CsvUtil - swapFiles[2]: .\src\main\resources\data\csv\user.csv has been deleted
[INFO ] 2021-12-24 21:31:23.987 [main] CsvUtil - swapFiles[3]: .\src\main\resources\data\csv\userNew.csv has been written and renamed to .\src\main\resources\data\csv\user.csv
[INFO ] 2021-12-24 21:31:23.989 [main] DataProviderCsv - appendUser[1]: appending user User(id=3, name=Lewis)
[INFO ] 2021-12-24 21:31:23.990 [main] DataProviderCsv - getUserById[1]: id:3
[INFO ] 2021-12-24 21:31:24.099 [main] DataProviderCsv - getUserById[2]: Got user User(id=0, name=null)
[INFO ] 2021-12-24 21:31:24.101 [main] DataProviderCsv - getUserById[3]: No entry found
[INFO ] 2021-12-24 21:31:24.102 [main] IdGenerator - generateUserCsvId[1]: Generating id for file user
[INFO ] 2021-12-24 21:31:24.111 [main] IdGenerator - generateUserCsvId[2]: Generated id 8
[INFO ] 2021-12-24 21:31:24.118 [main] DataProviderCsv - appendUser[2]: appended User(id=8, name=Lewis)
[INFO ] 2021-12-24 21:31:24.119 [main] DataProviderCsv - updateUser[2]: Entry successfully updated
[INFO ] 2021-12-24 21:32:33.737 [main] DataProviderXmlTest - appendUser test Success
[INFO ] 2021-12-24 21:32:33.750 [main] DataProviderXml - appendUser[1]: writing user User(id=1, name=Ivan)
[INFO ] 2021-12-24 21:32:33.882 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-24 21:34:02.829 [main] DataProviderXmlTest - appendUser test Success
[INFO ] 2021-12-24 21:34:02.843 [main] DataProviderXml - appendUser[1]: writing user User(id=2, name=Bean)
[INFO ] 2021-12-24 21:34:02.954 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-24 21:36:25.099 [main] DataProviderXmlTest - appendUser test Success
[INFO ] 2021-12-24 21:36:25.116 [main] DataProviderXml - appendUser[1]: writing user User(id=2, name=Bean)
[INFO ] 2021-12-24 21:36:25.289 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-24 21:44:57.245 [main] DataProviderXmlTest - appendUser test Success
[INFO ] 2021-12-24 21:44:57.257 [main] DataProviderXml - appendUser[1]: writing user User(id=2, name=Bean)
[INFO ] 2021-12-24 21:44:57.261 [main] IdGenerator - generateUserXmlId[1]: Generating id for file user
[INFO ] 2021-12-24 21:44:57.375 [main] IdGenerator - generateUserXmlId[2]: Generated id 3
[INFO ] 2021-12-24 21:44:57.391 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-24 21:45:23.656 [main] DataProviderXmlTest - appendUser test Fail
[INFO ] 2021-12-24 21:45:23.675 [main] DataProviderXml - appendUser[1]: writing user User(id=0, name=null)
[ERROR] 2021-12-24 21:45:23.680 [main] Validator - Invalid user
[ERROR] 2021-12-24 21:45:23.681 [main] DataProviderXml - appendUser[2]: exception received Invalid user
[INFO ] 2021-12-24 21:45:40.509 [main] DataProviderXmlTest - getUserById test Success
[INFO ] 2021-12-24 21:45:40.520 [main] DataProviderXml - getUserById[1]: searching for id=2
[INFO ] 2021-12-24 21:45:40.622 [main] DataProviderXml - getUserById[2]: searching result Optional[User(id=2, name=Bean)]
[INFO ] 2021-12-24 21:45:47.737 [main] DataProviderXmlTest - getUserById test Fail
[INFO ] 2021-12-24 21:45:47.744 [main] DataProviderXml - getUserById[1]: searching for id=1
[INFO ] 2021-12-24 21:45:47.859 [main] DataProviderXml - getUserById[2]: searching result Optional[User(id=0, name=null)]
[INFO ] 2021-12-24 21:46:11.531 [main] DataProviderXmlTest - deleteUserById test Success
[INFO ] 2021-12-24 21:46:11.543 [main] DataProviderXml - deleteUserById[1]: deleting user by id=3
[INFO ] 2021-12-24 21:46:11.674 [main] DataProviderXml - deleteUserById[2]: entry deleted
[INFO ] 2021-12-24 21:46:21.807 [main] DataProviderXmlTest - deleteUserById test Fail
[INFO ] 2021-12-24 21:46:21.818 [main] DataProviderXml - deleteUserById[1]: deleting user by id=14
[INFO ] 2021-12-24 21:46:21.944 [main] DataProviderXml - deleteUserById[2]: entry deleted
[INFO ] 2021-12-24 21:46:41.473 [main] DataProviderXmlTest - updateUser test Success
[INFO ] 2021-12-24 21:46:41.490 [main] DataProviderXml - updateUser[1]: updating userUser(id=2, name=Shaun)
[INFO ] 2021-12-24 21:46:41.492 [main] DataProviderXml - deleteUserById[1]: deleting user by id=2
[INFO ] 2021-12-24 21:46:41.604 [main] DataProviderXml - deleteUserById[2]: entry deleted
[INFO ] 2021-12-24 21:46:41.606 [main] DataProviderXml - appendUser[1]: writing user User(id=2, name=Shaun)
[INFO ] 2021-12-24 21:46:41.608 [main] IdGenerator - generateUserXmlId[1]: Generating id for file user
[ERROR] 2021-12-24 21:46:41.610 [main] DataProviderXml - appendUser[2]: exception received Unable to satisfy @org.simpleframework.xml.ElementList(entry="", data=false, inline=true, name="", type=void.class, required=true, empty=true) on field 'userList' private java.util.List ru.sfedu.log4jproject.model.Wrapper.userList for class ru.sfedu.log4jproject.model.Wrapper at line 1
[INFO ] 2021-12-24 21:46:41.610 [main] DataProviderXml - updateUser[2]: user updated
[INFO ] 2021-12-24 21:50:04.004 [main] DataProviderXmlTest - updateUser test Success
[INFO ] 2021-12-24 21:50:04.018 [main] DataProviderXml - updateUser[1]: updating userUser(id=2, name=Shaun)
[INFO ] 2021-12-24 21:50:04.020 [main] DataProviderXml - deleteUserById[1]: deleting user by id=2
[ERROR] 2021-12-24 21:50:04.060 [main] DataProviderXml - deleteUserById[2]: exception received ParseError at [row,col]:[1,1]
Message: Premature end of file.
[INFO ] 2021-12-24 21:50:04.063 [main] DataProviderXml - appendUser[1]: writing user User(id=2, name=Shaun)
[INFO ] 2021-12-24 21:50:04.064 [main] IdGenerator - generateUserXmlId[1]: Generating id for file user
[ERROR] 2021-12-24 21:50:04.064 [main] DataProviderXml - appendUser[2]: exception received ParseError at [row,col]:[1,1]
Message: Premature end of file.
[INFO ] 2021-12-24 21:50:04.065 [main] DataProviderXml - updateUser[2]: user updated
[INFO ] 2021-12-24 21:54:21.767 [main] DataProviderXmlTest - updateUser test Success
[INFO ] 2021-12-24 21:54:21.780 [main] DataProviderXml - updateUser[1]: updating userUser(id=2, name=Shaun)
[INFO ] 2021-12-24 21:54:21.782 [main] DataProviderXml - getUserById[1]: searching for id=2
[ERROR] 2021-12-24 21:54:21.836 [main] DataProviderXml - getUserById[2]: Exception received ParseError at [row,col]:[1,1]
Message: Premature end of file.
[INFO ] 2021-12-24 21:54:21.839 [main] DataProviderXml - deleteUserById[1]: deleting user by id=2
[ERROR] 2021-12-24 21:54:21.840 [main] DataProviderXml - deleteUserById[2]: exception received ParseError at [row,col]:[1,1]
Message: Premature end of file.
[INFO ] 2021-12-24 21:54:21.840 [main] DataProviderXml - appendUser[1]: writing user User(id=2, name=Shaun)
[INFO ] 2021-12-24 21:54:21.842 [main] IdGenerator - generateUserXmlId[1]: Generating id for file user
[ERROR] 2021-12-24 21:54:21.842 [main] DataProviderXml - appendUser[2]: exception received ParseError at [row,col]:[1,1]
Message: Premature end of file.
[INFO ] 2021-12-24 21:54:21.843 [main] DataProviderXml - updateUser[2]: user updated
[INFO ] 2021-12-24 21:54:48.766 [main] DataProviderXmlTest - updateUser test Success
[INFO ] 2021-12-24 21:54:48.785 [main] DataProviderXml - updateUser[1]: updating userUser(id=2, name=Shaun)
[INFO ] 2021-12-24 21:54:48.786 [main] DataProviderXml - getUserById[1]: searching for id=2
[ERROR] 2021-12-24 21:54:48.845 [main] DataProviderXml - getUserById[2]: Exception received ParseError at [row,col]:[1,1]
Message: Premature end of file.
[INFO ] 2021-12-24 21:54:48.848 [main] DataProviderXml - appendUser[1]: writing user User(id=2, name=Shaun)
[INFO ] 2021-12-24 21:54:48.850 [main] IdGenerator - generateUserXmlId[1]: Generating id for file user
[ERROR] 2021-12-24 21:54:48.851 [main] DataProviderXml - appendUser[2]: exception received ParseError at [row,col]:[1,1]
Message: Premature end of file.
[INFO ] 2021-12-24 21:54:48.852 [main] DataProviderXml - updateUser[2]: user updated
[INFO ] 2021-12-24 21:57:22.481 [main] DataProviderXmlTest - appendUser test Success
[INFO ] 2021-12-24 21:57:22.496 [main] DataProviderXml - appendUser[1]: writing user User(id=2, name=Bean)
[INFO ] 2021-12-24 21:57:22.501 [main] IdGenerator - generateUserXmlId[1]: Generating id for file user
[ERROR] 2021-12-24 21:57:22.541 [main] DataProviderXml - appendUser[2]: exception received ParseError at [row,col]:[1,1]
Message: Premature end of file.
[INFO ] 2021-12-24 21:59:14.890 [main] DataProviderXmlTest - appendUser test Success
[INFO ] 2021-12-24 21:59:14.905 [main] DataProviderXml - appendUser[1]: writing user User(id=2, name=Bean)
[INFO ] 2021-12-24 21:59:14.909 [main] IdGenerator - generateUserXmlId[1]: Generating id for file user
[ERROR] 2021-12-24 21:59:14.948 [main] DataProviderXml - appendUser[2]: exception received ParseError at [row,col]:[1,1]
Message: Premature end of file.
[INFO ] 2021-12-24 22:05:55.845 [main] DataProviderXmlTest - appendUser test Success
[INFO ] 2021-12-24 22:05:55.857 [main] DataProviderXml - appendUser[1]: writing user User(id=2, name=Bean)
[INFO ] 2021-12-24 22:05:55.969 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-24 22:07:00.349 [main] DataProviderXmlTest - updateUser test Success
[INFO ] 2021-12-24 22:07:00.363 [main] DataProviderXml - updateUser[1]: updating userUser(id=2, name=Shaun)
[INFO ] 2021-12-24 22:07:00.365 [main] DataProviderXml - deleteUserById[1]: deleting user by id=2
[INFO ] 2021-12-24 22:07:00.502 [main] DataProviderXml - deleteUserById[2]: entry deleted
[INFO ] 2021-12-24 22:07:00.504 [main] DataProviderXml - appendUser[1]: writing user User(id=2, name=Shaun)
[ERROR] 2021-12-24 22:07:00.506 [main] DataProviderXml - appendUser[2]: exception received Unable to satisfy @org.simpleframework.xml.ElementList(entry="", data=false, inline=true, name="", type=void.class, required=true, empty=true) on field 'userList' private java.util.List ru.sfedu.log4jproject.model.Wrapper.userList for class ru.sfedu.log4jproject.model.Wrapper at line 1
[INFO ] 2021-12-24 22:07:00.507 [main] DataProviderXml - updateUser[2]: user updated
[INFO ] 2021-12-24 22:13:57.241 [main] DataProviderXmlTest - updateUser test Success
[INFO ] 2021-12-24 22:13:57.255 [main] DataProviderXml - updateUser[1]: updating userUser(id=2, name=Shaun)
[INFO ] 2021-12-24 22:13:57.257 [main] DataProviderXml - deleteUserById[1]: deleting user by id=2
[ERROR] 2021-12-24 22:13:57.313 [main] DataProviderXml - deleteUserById[2]: exception received ParseError at [row,col]:[1,1]
Message: Premature end of file.
[INFO ] 2021-12-24 22:13:57.315 [main] DataProviderXml - appendUser[1]: writing user User(id=2, name=Shaun)
[INFO ] 2021-12-24 22:13:57.405 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-24 22:13:57.405 [main] DataProviderXml - updateUser[2]: user updated
[INFO ] 2021-12-24 22:14:47.658 [main] DataProviderXmlTest - updateUser test Success
[INFO ] 2021-12-24 22:14:47.672 [main] DataProviderXml - updateUser[1]: updating userUser(id=2, name=Shaun)
[INFO ] 2021-12-24 22:14:47.674 [main] DataProviderXml - deleteUserById[1]: deleting user by id=2
[ERROR] 2021-12-24 22:14:47.715 [main] DataProviderXml - deleteUserById[2]: exception received ParseError at [row,col]:[1,1]
Message: Premature end of file.
[INFO ] 2021-12-24 22:14:47.718 [main] DataProviderXml - appendUser[1]: writing user User(id=2, name=Shaun)
[INFO ] 2021-12-24 22:14:47.719 [main] IdGenerator - generateUserXmlId[1]: Generating id for file user
[ERROR] 2021-12-24 22:14:47.720 [main] DataProviderXml - appendUser[2]: exception received ParseError at [row,col]:[1,1]
Message: Premature end of file.
[INFO ] 2021-12-24 22:14:47.720 [main] DataProviderXml - updateUser[2]: user updated
[INFO ] 2021-12-24 22:17:48.611 [main] DataProviderXmlTest - updateUser test Success
[INFO ] 2021-12-24 22:17:48.622 [main] DataProviderXml - updateUser[1]: updating userUser(id=2, name=Shaun)
[INFO ] 2021-12-24 22:17:48.624 [main] DataProviderXml - deleteUserById[1]: deleting user by id=2
[ERROR] 2021-12-24 22:17:48.662 [main] DataProviderXml - deleteUserById[2]: exception received ParseError at [row,col]:[1,1]
Message: Premature end of file.
[INFO ] 2021-12-24 22:17:48.665 [main] DataProviderXml - appendUser[1]: writing user User(id=2, name=Shaun)
[INFO ] 2021-12-24 22:17:48.666 [main] IdGenerator - generateUserXmlId[1]: Generating id for file user
[ERROR] 2021-12-24 22:17:48.667 [main] DataProviderXml - appendUser[2]: exception received ParseError at [row,col]:[1,1]
Message: Premature end of file.
[INFO ] 2021-12-24 22:17:48.667 [main] DataProviderXml - updateUser[2]: user updated
[INFO ] 2021-12-24 22:20:50.036 [main] DataProviderXmlTest - updateUser test Success
[INFO ] 2021-12-24 22:20:50.052 [main] DataProviderXml - updateUser[1]: updating userUser(id=2, name=Shaun)
[INFO ] 2021-12-24 22:20:50.053 [main] DataProviderXml - deleteUserById[1]: deleting user by id=2
[ERROR] 2021-12-24 22:20:50.103 [main] DataProviderXml - deleteUserById[2]: exception received ParseError at [row,col]:[1,1]
Message: Premature end of file.
[INFO ] 2021-12-24 22:20:50.105 [main] DataProviderXml - appendUser[1]: writing user User(id=2, name=Shaun)
[INFO ] 2021-12-24 22:20:50.107 [main] IdGenerator - generateUserXmlId[1]: Generating id for file user
[ERROR] 2021-12-24 22:20:50.108 [main] IdGenerator - generateUserXmlId[2]: error appeared, file is considered to be empty, given id - 1
[INFO ] 2021-12-24 22:20:50.179 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-24 22:20:50.180 [main] DataProviderXml - updateUser[2]: user updated
[INFO ] 2021-12-24 22:21:21.733 [main] DataProviderXmlTest - updateUser test Success
[INFO ] 2021-12-24 22:21:21.744 [main] DataProviderXml - updateUser[1]: updating userUser(id=5, name=Helen)
[INFO ] 2021-12-24 22:21:21.745 [main] DataProviderXml - deleteUserById[1]: deleting user by id=5
[INFO ] 2021-12-24 22:21:21.892 [main] DataProviderXml - deleteUserById[2]: entry deleted
[INFO ] 2021-12-24 22:21:21.894 [main] DataProviderXml - appendUser[1]: writing user User(id=5, name=Helen)
[INFO ] 2021-12-24 22:21:21.896 [main] IdGenerator - generateUserXmlId[1]: Generating id for file user
[INFO ] 2021-12-24 22:21:21.898 [main] IdGenerator - generateUserXmlId[2]: Generated id 2
[INFO ] 2021-12-24 22:21:21.902 [main] DataProviderXml - appendUser[2]: user successfully written
[INFO ] 2021-12-24 22:21:21.903 [main] DataProviderXml - updateUser[2]: user updated
[INFO ] 2021-12-24 22:23:49.341 [main] DataProviderXmlTest - updateUser test Fail
[INFO ] 2021-12-24 22:23:49.355 [main] DataProviderXml - updateUser[1]: updating userUser(id=0, name=null)
[ERROR] 2021-12-24 22:23:49.357 [main] Validator - Invalid user
[ERROR] 2021-12-24 22:23:49.357 [main] DataProviderXml - updateUser[2]: exception received Invalid user
[INFO ] 2021-12-25 00:48:10.993 [main] DataProviderH2 - appendUser test success
[INFO ] 2021-12-25 00:48:11.003 [main] DataProviderH2 - appendUser[1]: appending user User(id=51, name=Leah)
[INFO ] 2021-12-25 00:48:11.006 [main] IdGenerator - generateForH2[1]: generating new id on database
[INFO ] 2021-12-25 00:48:11.362 [main] IdGenerator - generateForH2[2]: generated id 28
[INFO ] 2021-12-25 00:48:11.432 [main] DataProviderH2 - appendUser[2]: user appended
[INFO ] 2021-12-25 01:05:54.100 [main] DataProviderH2 - deleteUserById test Success
[INFO ] 2021-12-25 01:05:54.106 [main] DataProviderH2 - deleteUserById[1]: deleting user by id 26
[INFO ] 2021-12-25 01:05:54.440 [main] DataProviderH2 - deleteUserById[2]: user deleted
[INFO ] 2021-12-25 01:20:07.034 [main] DataProviderH2 - getUserById test success
[INFO ] 2021-12-25 01:20:07.043 [main] DataProviderH2 - getUserById[1]: getting user by id 1
[INFO ] 2021-12-25 01:20:07.348 [main] DataProviderH2 - getUserById[2]: received user User(id=1, name=Ivan)
[INFO ] 2021-12-25 01:59:21.924 [main] DataProviderH2 - getUserById test success
[ERROR] 2021-12-25 01:59:22.243 [main] DataProviderH2 - DataProviderH2 Constructor: unable to create data provider. Exception received Неизвестный тип данных: "STRING"
Unknown data type: "STRING"; SQL statement:
CREATE TABLE IF NOT EXISTS Users(ID INTEGER NOT NULL AUTO_INCREMENT PRIMARY KEY,ID BIGINT NOT NULL,Name STRING NOT NULL) [50004-200]
[INFO ] 2021-12-25 01:59:22.243 [main] DataProviderH2 - getUserById[1]: getting user by id 1
[INFO ] 2021-12-25 01:59:22.263 [main] DataProviderH2 - getUserById[2]: received user User(id=1, name=Ivan)
[INFO ] 2021-12-25 02:01:21.733 [main] DataProviderH2 - getUserById test success
[INFO ] 2021-12-25 02:01:22.054 [main] DataProviderH2 - getUserById[1]: getting user by id 1
[INFO ] 2021-12-25 02:01:22.112 [main] DataProviderH2 - getUserById[2]: received user User(id=1, name=Ivan)
[INFO ] 2021-12-25 10:02:01.392 [main] MongoProvider - workWithMongo[1]: working with {password=root, dbname=admin, port=27017, host=localhost, login=root, table=users}
[INFO ] 2021-12-25 10:02:01.538 [main] MongoProvider - save[1]: command = UPDATING, type = XML, object = Result{message='Some message', entity=User(id=1, name=Sonic), code=SUCCESS}
[INFO ] 2021-12-25 10:02:38.379 [main] MongoProvider - workWithMongo[1]: working with {password=root, dbname=admin, port=27017, host=localhost, login=root, table=users}
[INFO ] 2021-12-25 10:02:38.514 [main] MongoProvider - save[1]: command = UPDATING, type = XML, object = Result{message='Some message', entity=User(id=1, name=Sonic), code=SUCCESS}
[INFO ] 2021-12-25 10:23:46.740 [main] MongoProvider - workWithMongo[1]: working with {password=root, dbname=admin, port=27017, host=localhost, login=root, table=users}
[INFO ] 2021-12-25 10:23:46.886 [main] MongoProvider - save[1]: command = DELETING, type = XML, object = Result{message='Some message', entity=User(id=1, name=Sonic), code=SUCCESS}
[INFO ] 2021-12-25 10:29:23.249 [main] MongoProvider - workWithMongo[1]: working with {password=root, dbname=admin, port=27017, host=localhost, login=root, table=users}
[INFO ] 2021-12-25 10:29:23.370 [main] MongoProvider - save[1]: command = DELETING, type = XML, object = Result{message='Some message', entity=User(id=15, name=TEST), code=SUCCESS}
[INFO ] 2021-12-25 10:33:23.942 [main] MongoProvider - workWithMongo[1]: working with {password=root, dbname=admin, port=27017, host=localhost, login=root, table=users}
[INFO ] 2021-12-25 10:33:24.102 [main] MongoProvider - save[1]: command = DELETING, type = XML, object = Result{message='Some message', entity=User(id=15, name=TEST), code=SUCCESS}
[INFO ] 2021-12-25 10:46:59.614 [main] DataProviderCsvTest - AppendUser test Success
[INFO ] 2021-12-25 10:46:59.625 [main] DataProviderCsv - appendUser[1]: appending user User(id=15, name=Frank)
[INFO ] 2021-12-25 10:46:59.626 [main] DataProviderCsv - getUserById[1]: id:15
[INFO ] 2021-12-25 10:46:59.732 [main] DataProviderCsv - getUserById[2]: Got user User(id=0, name=null)
[INFO ] 2021-12-25 10:46:59.734 [main] DataProviderCsv - getUserById[3]: No entry found
[INFO ] 2021-12-25 10:46:59.739 [main] MongoProvider - save[1]: command = GETTING, type = CSV, object = Result{message='No entry found', entity=null, code=ENTRY_NOT_FOUND}
[ERROR] 2021-12-25 10:46:59.747 [main] MongoProvider - save[2]: Exception received Cannot invoke "com.mongodb.DBCollection.insert(com.mongodb.DBObject[])" because "ru.sfedu.log4jproject.api.MongoProvider.table" is null
[INFO ] 2021-12-25 10:46:59.749 [main] IdGenerator - generateUserCsvId[1]: Generating id for file user
[INFO ] 2021-12-25 10:46:59.761 [main] IdGenerator - generateUserCsvId[2]: Generated id 9
[INFO ] 2021-12-25 10:46:59.769 [main] DataProviderCsv - appendUser[2]: appended User(id=9, name=Frank)
[INFO ] 2021-12-25 10:46:59.769 [main] MongoProvider - save[1]: command = APPENDING, type = CSV, object = Result{message='null', entity=null, code=SUCCESS}
[ERROR] 2021-12-25 10:46:59.770 [main] MongoProvider - save[2]: Exception received Cannot invoke "com.mongodb.DBCollection.insert(com.mongodb.DBObject[])" because "ru.sfedu.log4jproject.api.MongoProvider.table" is null
[INFO ] 2021-12-25 10:51:03.075 [main] DataProviderCsvTest - AppendUser test Success
[INFO ] 2021-12-25 10:51:03.086 [main] DataProviderCsv - appendUser[1]: appending user User(id=15, name=Frank)
[INFO ] 2021-12-25 10:51:03.087 [main] DataProviderCsv - getUserById[1]: id:15
[INFO ] 2021-12-25 10:51:03.199 [main] DataProviderCsv - getUserById[2]: Got user User(id=0, name=null)
[INFO ] 2021-12-25 10:51:03.200 [main] DataProviderCsv - getUserById[3]: No entry found
[INFO ] 2021-12-25 10:51:03.205 [main] MongoProvider - save[1]: command = GETTING, type = CSV, object = Result{message='No entry found', entity=null, code=ENTRY_NOT_FOUND}
[ERROR] 2021-12-25 10:51:03.209 [main] MongoProvider - save[2]: Exception received Cannot invoke "com.mongodb.DBCollection.insert(com.mongodb.DBObject[])" because "ru.sfedu.log4jproject.api.MongoProvider.table" is null
[INFO ] 2021-12-25 10:51:03.211 [main] IdGenerator - generateUserCsvId[1]: Generating id for file user
[INFO ] 2021-12-25 10:51:03.219 [main] IdGenerator - generateUserCsvId[2]: Generated id 10
[INFO ] 2021-12-25 10:51:03.236 [main] DataProviderCsv - appendUser[2]: appended User(id=10, name=Frank)
[INFO ] 2021-12-25 10:51:03.236 [main] MongoProvider - save[1]: command = APPENDING, type = CSV, object = Result{message='null', entity=null, code=SUCCESS}
[ERROR] 2021-12-25 10:51:03.236 [main] MongoProvider - save[2]: Exception received Cannot invoke "com.mongodb.DBCollection.insert(com.mongodb.DBObject[])" because "ru.sfedu.log4jproject.api.MongoProvider.table" is null
[INFO ] 2021-12-25 10:52:22.140 [main] DataProviderCsvTest - AppendUser test Success
[INFO ] 2021-12-25 10:52:22.171 [main] DataProviderCsv - appendUser[1]: appending user User(id=15, name=Frank)
[INFO ] 2021-12-25 10:52:22.173 [main] DataProviderCsv - getUserById[1]: id:15
[INFO ] 2021-12-25 10:52:22.306 [main] DataProviderCsv - getUserById[2]: Got user User(id=0, name=null)
[INFO ] 2021-12-25 10:52:22.307 [main] DataProviderCsv - getUserById[3]: No entry found
[INFO ] 2021-12-25 10:52:22.313 [main] MongoProvider - save[1]: command = GETTING, type = CSV, object = Result{message='No entry found', entity=null, code=ENTRY_NOT_FOUND}
[ERROR] 2021-12-25 10:52:22.319 [main] MongoProvider - save[2]: Exception received Cannot invoke "com.mongodb.DBCollection.insert(com.mongodb.DBObject[])" because "ru.sfedu.log4jproject.api.MongoProvider.table" is null
[INFO ] 2021-12-25 10:52:22.321 [main] IdGenerator - generateUserCsvId[1]: Generating id for file user
[INFO ] 2021-12-25 10:52:22.330 [main] IdGenerator - generateUserCsvId[2]: Generated id 11
[INFO ] 2021-12-25 10:52:22.338 [main] DataProviderCsv - appendUser[2]: appended User(id=11, name=Frank)
[INFO ] 2021-12-25 10:52:22.339 [main] MongoProvider - save[1]: command = APPENDING, type = CSV, object = Result{message='null', entity=null, code=SUCCESS}
[ERROR] 2021-12-25 10:52:22.339 [main] MongoProvider - save[2]: Exception received Cannot invoke "com.mongodb.DBCollection.insert(com.mongodb.DBObject[])" because "ru.sfedu.log4jproject.api.MongoProvider.table" is null
[INFO ] 2021-12-25 10:54:06.975 [main] DataProviderCsvTest - AppendUser test Success
[INFO ] 2021-12-25 10:54:06.987 [main] DataProviderCsv - appendUser[1]: appending user User(id=15, name=Frank)
[INFO ] 2021-12-25 10:54:06.989 [main] DataProviderCsv - getUserById[1]: id:15
[INFO ] 2021-12-25 10:54:07.106 [main] DataProviderCsv - getUserById[2]: Got user User(id=0, name=null)
[INFO ] 2021-12-25 10:54:07.107 [main] DataProviderCsv - getUserById[3]: No entry found
[INFO ] 2021-12-25 10:54:07.112 [main] MongoProvider - save[1]: command = GETTING, type = CSV, object = Result{message='No entry found', entity=null, code=ENTRY_NOT_FOUND}
[ERROR] 2021-12-25 10:54:07.117 [main] MongoProvider - save[2]: Exception received Cannot invoke "com.mongodb.DBCollection.insert(com.mongodb.DBObject[])" because "ru.sfedu.log4jproject.api.MongoProvider.table" is null
[INFO ] 2021-12-25 10:54:07.119 [main] IdGenerator - generateUserCsvId[1]: Generating id for file user
[INFO ] 2021-12-25 10:54:07.127 [main] IdGenerator - generateUserCsvId[2]: Generated id 12
[INFO ] 2021-12-25 10:54:07.143 [main] DataProviderCsv - appendUser[2]: appended User(id=12, name=Frank)
[INFO ] 2021-12-25 10:54:07.143 [main] MongoProvider - save[1]: command = APPENDING, type = CSV, object = Result{message='null', entity=null, code=SUCCESS}
[ERROR] 2021-12-25 10:54:07.144 [main] MongoProvider - save[2]: Exception received Cannot invoke "com.mongodb.DBCollection.insert(com.mongodb.DBObject[])" because "ru.sfedu.log4jproject.api.MongoProvider.table" is null
[INFO ] 2021-12-25 10:58:20.746 [main] DataProviderCsvTest - AppendUser test Success
[INFO ] 2021-12-25 10:58:20.758 [main] DataProviderCsv - appendUser[1]: appending user User(id=15, name=Frank)
[INFO ] 2021-12-25 10:58:20.759 [main] DataProviderCsv - getUserById[1]: id:15
[INFO ] 2021-12-25 10:58:20.883 [main] DataProviderCsv - getUserById[2]: Got user User(id=0, name=null)
[INFO ] 2021-12-25 10:58:20.885 [main] DataProviderCsv - getUserById[3]: No entry found
[INFO ] 2021-12-25 10:58:20.890 [main] MongoProvider - save[1]: command = GETTING, type = CSV, object = Result{message='No entry found', entity=null, code=ENTRY_NOT_FOUND}
[ERROR] 2021-12-25 10:58:20.895 [main] MongoProvider - save[2]: Exception received Cannot invoke "com.mongodb.DBCollection.insert(com.mongodb.DBObject[])" because "ru.sfedu.log4jproject.api.MongoProvider.table" is null
[INFO ] 2021-12-25 10:58:20.898 [main] IdGenerator - generateUserCsvId[1]: Generating id for file user
[INFO ] 2021-12-25 10:58:20.914 [main] IdGenerator - generateUserCsvId[2]: Generated id 13
[INFO ] 2021-12-25 10:58:20.922 [main] DataProviderCsv - appendUser[2]: appended User(id=13, name=Frank)
[INFO ] 2021-12-25 10:58:20.923 [main] MongoProvider - save[1]: command = APPENDING, type = CSV, object = Result{message='null', entity=null, code=SUCCESS}
[ERROR] 2021-12-25 10:58:20.923 [main] MongoProvider - save[2]: Exception received Cannot invoke "com.mongodb.DBCollection.insert(com.mongodb.DBObject[])" because "ru.sfedu.log4jproject.api.MongoProvider.table" is null
[INFO ] 2021-12-25 11:02:11.995 [main] MongoProvider - PROPERTIES {password=root, dbname=admin, port=27017, host=localhost, login=root, table=Users}
[INFO ] 2021-12-25 11:02:12.001 [main] MongoProvider - workWithMongo[1]: working with {password=root, dbname=admin, port=27017, host=localhost, login=root, table=Users}
[INFO ] 2021-12-25 11:02:12.158 [main] MongoProvider - save[1]: command = DELETING, type = XML, object = Result{message='Some message', entity=User(id=15, name=TEST), code=SUCCESS}
[INFO ] 2021-12-25 11:02:12.197 [main] MongoProvider - PROPERTIES {password=root, dbname=admin, port=27017, host=localhost, login=root, table=Users}
[INFO ] 2021-12-25 11:02:12.197 [main] MongoProvider - workWithMongo[1]: working with {password=root, dbname=admin, port=27017, host=localhost, login=root, table=Users}
[INFO ] 2021-12-25 11:02:12.203 [main] MongoProvider - save[1]: command = UPDATING, type = XML, object = User(id=0, name=null)
[INFO ] 2021-12-25 11:02:59.590 [main] MongoProvider - PROPERTIES {password=root, dbname=admin, port=27017, host=localhost, login=root, table=Users}
[INFO ] 2021-12-25 11:02:59.593 [main] MongoProvider - workWithMongo[1]: working with {password=root, dbname=admin, port=27017, host=localhost, login=root, table=Users}
[INFO ] 2021-12-25 11:02:59.709 [main] MongoProvider - save[1]: command = DELETING, type = XML, object = Result{message='Some message', entity=User(id=15, name=TEST), code=SUCCESS}
[INFO ] 2021-12-25 11:04:11.395 [main] DataProviderCsvTest - AppendUser test Success
[INFO ] 2021-12-25 11:04:11.407 [main] DataProviderCsv - appendUser[1]: appending user User(id=15, name=Frank)
[INFO ] 2021-12-25 11:04:11.408 [main] DataProviderCsv - getUserById[1]: id:15
[INFO ] 2021-12-25 11:04:11.570 [main] DataProviderCsv - getUserById[2]: Got user User(id=0, name=null)
[INFO ] 2021-12-25 11:04:11.571 [main] DataProviderCsv - getUserById[3]: No entry found
[INFO ] 2021-12-25 11:04:11.578 [main] MongoProvider - save[1]: command = GETTING, type = CSV, object = Result{message='No entry found', entity=null, code=ENTRY_NOT_FOUND}
[ERROR] 2021-12-25 11:04:11.583 [main] MongoProvider - save[2]: Exception received Cannot invoke "com.mongodb.DBCollection.insert(com.mongodb.DBObject[])" because "ru.sfedu.log4jproject.api.MongoProvider.table" is null
[INFO ] 2021-12-25 11:04:11.585 [main] IdGenerator - generateUserCsvId[1]: Generating id for file user
[INFO ] 2021-12-25 11:04:11.598 [main] IdGenerator - generateUserCsvId[2]: Generated id 14
[INFO ] 2021-12-25 11:04:11.606 [main] DataProviderCsv - appendUser[2]: appended User(id=14, name=Frank)
[INFO ] 2021-12-25 11:04:11.606 [main] MongoProvider - save[1]: command = APPENDING, type = CSV, object = Result{message='null', entity=null, code=SUCCESS}
[ERROR] 2021-12-25 11:04:11.606 [main] MongoProvider - save[2]: Exception received Cannot invoke "com.mongodb.DBCollection.insert(com.mongodb.DBObject[])" because "ru.sfedu.log4jproject.api.MongoProvider.table" is null
[INFO ] 2021-12-25 11:04:45.039 [main] DataProviderCsvTest - AppendUser test Success
[INFO ] 2021-12-25 11:04:45.054 [main] DataProviderCsv - appendUser[1]: appending user User(id=15, name=Frank)
[INFO ] 2021-12-25 11:04:45.056 [main] DataProviderCsv - getUserById[1]: id:15
[INFO ] 2021-12-25 11:04:45.181 [main] DataProviderCsv - getUserById[2]: Got user User(id=0, name=null)
[INFO ] 2021-12-25 11:04:45.182 [main] DataProviderCsv - getUserById[3]: No entry found
[INFO ] 2021-12-25 11:04:45.188 [main] MongoProvider - save[1]: command = GETTING, type = CSV, object = Result{message='No entry found', entity=null, code=ENTRY_NOT_FOUND}
[ERROR] 2021-12-25 11:04:45.193 [main] MongoProvider - save[2]: Exception received Cannot invoke "com.mongodb.DBCollection.insert(com.mongodb.DBObject[])" because "ru.sfedu.log4jproject.api.MongoProvider.table" is null
[INFO ] 2021-12-25 11:04:45.195 [main] IdGenerator - generateUserCsvId[1]: Generating id for file user
[INFO ] 2021-12-25 11:04:45.207 [main] IdGenerator - generateUserCsvId[2]: Generated id 15
[INFO ] 2021-12-25 11:04:45.215 [main] DataProviderCsv - appendUser[2]: appended User(id=15, name=Frank)
[INFO ] 2021-12-25 11:04:45.215 [main] MongoProvider - save[1]: command = APPENDING, type = CSV, object = Result{message='null', entity=null, code=SUCCESS}
[ERROR] 2021-12-25 11:04:45.216 [main] MongoProvider - save[2]: Exception received Cannot invoke "com.mongodb.DBCollection.insert(com.mongodb.DBObject[])" because "ru.sfedu.log4jproject.api.MongoProvider.table" is null
[INFO ] 2021-12-25 11:07:29.101 [main] DataProviderCsvTest - AppendUser test Success
[INFO ] 2021-12-25 11:07:29.111 [main] MongoProvider - PROPERTIES {password=root, dbname=admin, port=27017, host=localhost, login=root, table=Users}
[INFO ] 2021-12-25 11:07:29.111 [main] MongoProvider - workWithMongo[1]: working with {password=root, dbname=admin, port=27017, host=localhost, login=root, table=Users}
[INFO ] 2021-12-25 11:07:29.246 [main] DataProviderCsv - appendUser[1]: appending user User(id=15, name=Frank)
[INFO ] 2021-12-25 11:07:29.248 [main] DataProviderCsv - getUserById[1]: id:15
[INFO ] 2021-12-25 11:07:29.374 [main] DataProviderCsv - getUserById[2]: Got user User(id=15, name=Frank)
[INFO ] 2021-12-25 11:07:29.379 [main] MongoProvider - save[1]: command = GETTING, type = CSV, object = Result{message='null', entity=User(id=15, name=Frank), code=SUCCESS}
[ERROR] 2021-12-25 11:07:29.383 [main] DataProviderCsv - appendUser[2]: This user with this id already exists Result{message='null', entity=User(id=15, name=Frank), code=SUCCESS}
[INFO ] 2021-12-25 11:07:29.384 [main] MongoProvider - save[1]: command = APPENDING, type = CSV, object = Result{message='null', entity=null, code=SUCCESS}
[INFO ] 2021-12-25 11:09:57.225 [main] DataProviderCsvTest - AppendUser test Success
[INFO ] 2021-12-25 11:09:57.234 [main] MongoProvider - PROPERTIES {password=root, dbname=admin, port=27017, host=localhost, login=root, table=Users}
[INFO ] 2021-12-25 11:09:57.235 [main] MongoProvider - workWithMongo[1]: working with {password=root, dbname=admin, port=27017, host=localhost, login=root, table=Users}
[INFO ] 2021-12-25 11:09:57.353 [main] DataProviderCsv - appendUser[1]: appending user User(id=4, name=Frank)
[INFO ] 2021-12-25 11:09:57.355 [main] DataProviderCsv - getUserById[1]: id:4
[INFO ] 2021-12-25 11:09:57.467 [main] DataProviderCsv - getUserById[2]: Got user User(id=0, name=null)
[INFO ] 2021-12-25 11:09:57.468 [main] DataProviderCsv - getUserById[3]: No entry found
[INFO ] 2021-12-25 11:09:57.472 [main] MongoProvider - save[1]: command = GETTING, type = CSV, object = Result{message='No entry found', entity=null, code=ENTRY_NOT_FOUND}
[INFO ] 2021-12-25 11:09:57.480 [main] IdGenerator - generateUserCsvId[1]: Generating id for file user
[INFO ] 2021-12-25 11:09:57.495 [main] IdGenerator - generateUserCsvId[2]: Generated id 16
[INFO ] 2021-12-25 11:09:57.503 [main] DataProviderCsv - appendUser[2]: appended User(id=16, name=Frank)
[INFO ] 2021-12-25 11:09:57.503 [main] MongoProvider - save[1]: command = APPENDING, type = CSV, object = Result{message='null', entity=null, code=SUCCESS}
[INFO ] 2021-12-25 11:13:38.739 [main] DataProviderCsvTest - deleteUserById test Success
[INFO ] 2021-12-25 11:13:38.747 [main] MongoProvider - PROPERTIES {password=root, dbname=admin, port=27017, host=localhost, login=root, table=Users}
[INFO ] 2021-12-25 11:13:38.747 [main] MongoProvider - workWithMongo[1]: working with {password=root, dbname=admin, port=27017, host=localhost, login=root, table=Users}
[INFO ] 2021-12-25 11:13:38.853 [main] DataProviderCsv - deleteUserById[1]: id:13
[INFO ] 2021-12-25 11:13:38.869 [main] CsvUtil - swapFiles[1]: swapping for file user
[INFO ] 2021-12-25 11:13:38.870 [main] CsvUtil - swapFiles[2]: .\src\main\resources\data\csv\user.csv has been deleted
[INFO ] 2021-12-25 11:13:38.871 [main] CsvUtil - swapFiles[3]: .\src\main\resources\data\csv\userNew.csv has been written and renamed to .\src\main\resources\data\csv\user.csv
[INFO ] 2021-12-25 11:13:38.874 [main] MongoProvider - save[1]: command = DELETING, type = CSV, object = Result{message='Deleted entry by id 13', entity=null, code=SUCCESS}
[INFO ] 2022-03-16 08:46:32.003 [main] DataProviderHibernateTest - getDatabaseSize test Success
[INFO ] 2022-03-16 08:48:36.730 [main] DataProviderHibernateTest - getDatabaseSize test Success
[INFO ] 2022-03-16 08:48:38.273 [main] DataProviderHibernate - getDatabaseSize[1]: getting database size
[INFO ] 2022-03-16 08:48:38.354 [main] DataProviderHibernate - getDatabaseSize[2]: got result 0
[INFO ] 2022-03-16 08:51:58.258 [main] DataProviderHibernateTest - getDatabaseSize test Success
[INFO ] 2022-03-16 08:51:59.752 [main] DataProviderHibernate - getDatabaseSize[1]: getting database size
[ERROR] 2022-03-16 08:51:59.874 [main] DataProviderHibernate - getDataBaseSize[ERR]: org.hibernate.exception.SQLGrammarException: could not prepare statement
[INFO ] 2022-03-16 08:57:58.193 [main] DataProviderHibernateTest - getDatabaseSize test Success
[INFO ] 2022-03-16 08:57:59.727 [main] DataProviderHibernate - getDatabaseSize[1]: getting database size
[ERROR] 2022-03-16 08:57:59.839 [main] DataProviderHibernate - getDataBaseSize[ERR]: org.hibernate.exception.SQLGrammarException: could not prepare statement
[INFO ] 2022-03-16 09:03:19.878 [main] DataProviderHibernateTest - getDatabaseSize test Success
[INFO ] 2022-03-16 09:03:21.395 [main] DataProviderHibernate - getDatabaseSize[1]: getting database size
[ERROR] 2022-03-16 09:03:21.518 [main] DataProviderHibernate - getDataBaseSize[ERR]: org.hibernate.exception.SQLGrammarException: could not prepare statement
[INFO ] 2022-03-17 15:06:29.224 [main] DataProviderHibernateTest - getDatabaseSize test Success
[INFO ] 2022-03-17 15:06:30.997 [main] DataProviderHibernate - getDatabaseSize[1]: getting database size
[ERROR] 2022-03-17 15:06:31.142 [main] DataProviderHibernate - getDataBaseSize[ERR]: org.hibernate.exception.SQLGrammarException: could not prepare statement
[INFO ] 2022-03-28 15:04:16.682 [main] DataProviderHibernateTest - getDatabaseSize test Success
[INFO ] 2022-03-28 15:06:17.309 [main] DataProviderHibernateTest - getDatabaseSize test Success
[INFO ] 2022-03-28 15:09:40.093 [main] DataProviderHibernateTest - getDatabaseSize test Success
[INFO ] 2022-03-28 15:09:42.123 [main] DataProviderHibernate - getDatabaseSize[1]: getting database size
[INFO ] 2022-03-28 15:09:42.549 [main] DataProviderHibernate - getDatabaseSize[2]: got result 8553 kB
[ERROR] 2022-03-28 15:09:42.550 [main] DataProviderHibernate - getDataBaseSize[ERR]: class java.lang.String cannot be cast to class java.lang.Long (java.lang.String and java.lang.Long are in module java.base of loader 'bootstrap')
[INFO ] 2022-03-28 15:11:20.252 [main] DataProviderHibernateTest - getDatabaseSize test Success
[INFO ] 2022-03-28 15:11:22.353 [main] DataProviderHibernate - getDatabaseSize[1]: getting database size
[INFO ] 2022-03-28 15:11:22.655 [main] DataProviderHibernate - getDatabaseSize[2]: got result 8553 kB
[INFO ] 2022-03-28 15:12:58.397 [main] DataProviderHibernateTest - getDatabaseSize test Success
[INFO ] 2022-03-28 15:13:00.498 [main] DataProviderHibernate - getDatabaseSize[1]: getting database size
[INFO ] 2022-03-28 15:13:00.808 [main] DataProviderHibernate - getDatabaseSize[2]: got result 8553 kB
[INFO ] 2022-03-28 15:16:27.054 [main] DataProviderHibernateTest - getDatabaseVersion test Success
[INFO ] 2022-03-28 15:16:29.127 [main] DataProviderHibernate - getDatabaseVersion[1]: getting database version
[INFO ] 2022-03-28 15:16:29.288 [main] DataProviderHibernate - getDatabaseVersion[2]: got result PostgreSQL 14.2, compiled by Visual C++ build 1914, 64-bit
[INFO ] 2022-03-28 15:24:44.216 [main] DataProviderHibernateTest - getDatabaseSize test Success
[INFO ] 2022-03-28 15:24:46.301 [main] DataProviderHibernate - getDatabaseSize[1]: getting database size
[INFO ] 2022-03-28 15:24:46.598 [main] DataProviderHibernate - getDatabaseSize[2]: got result 8553 kB
[INFO ] 2022-03-28 15:24:46.600 [main] DataProviderHibernateTest - CREATE TABLE IF NOT EXISTS users(ID INTEGER NOT NULL AUTO_INCREMENT PRIMARY KEY,ID BIGINT NOT NULL,Name CHAR NOT NULL)
[INFO ] 2022-03-28 15:33:36.340 [main] DataProviderHibernateTest - getDatabaseSize test Success
[INFO ] 2022-03-28 15:33:38.178 [main] DataProviderHibernate - getDatabaseSize[1]: getting database size
[ERROR] 2022-03-28 15:33:38.323 [main] DataProviderHibernate - getDataBaseSize[ERR]: org.hibernate.exception.SQLGrammarException: could not extract ResultSet
[ERROR] 2022-03-28 15:33:38.324 [main] DataProviderHibernateTest - getDatabaseSize test Success - Failure
[INFO ] 2022-03-28 15:33:38.352 [main] DataProviderHibernateTest - getDatabaseVersion test Success
[INFO ] 2022-03-28 15:33:38.352 [main] DataProviderHibernate - getDatabaseVersion[1]: getting database version
[INFO ] 2022-03-28 15:33:38.363 [main] DataProviderHibernate - getDatabaseVersion[2]: got result PostgreSQL 14.2, compiled by Visual C++ build 1914, 64-bit
[INFO ] 2022-03-28 15:34:51.799 [main] DataProviderHibernateTest - getDatabaseSize test Success
[INFO ] 2022-03-28 15:34:53.340 [main] DataProviderHibernate - getDatabaseSize[1]: getting database size
[ERROR] 2022-03-28 15:34:53.458 [main] DataProviderHibernate - getDataBaseSize[ERR]: org.hibernate.exception.GenericJDBCException: could not extract ResultSet
[ERROR] 2022-03-28 15:34:53.458 [main] DataProviderHibernateTest - getDatabaseSize test Success - Failure
[INFO ] 2022-03-28 15:34:53.477 [main] DataProviderHibernateTest - getDatabaseVersion test Success
[INFO ] 2022-03-28 15:34:53.477 [main] DataProviderHibernate - getDatabaseVersion[1]: getting database version
[INFO ] 2022-03-28 15:34:53.489 [main] DataProviderHibernate - getDatabaseVersion[2]: got result PostgreSQL 14.2, compiled by Visual C++ build 1914, 64-bit
[INFO ] 2022-03-28 15:35:38.206 [main] DataProviderHibernateTest - getDatabaseSize test Success
[INFO ] 2022-03-28 15:35:39.740 [main] DataProviderHibernate - getDatabaseSize[1]: getting database size
[INFO ] 2022-03-28 15:35:40.005 [main] DataProviderHibernate - getDatabaseSize[2]: got result 8617 kB
[INFO ] 2022-03-28 15:35:40.015 [main] DataProviderHibernateTest - getDatabaseVersion test Success
[INFO ] 2022-03-28 15:35:40.016 [main] DataProviderHibernate - getDatabaseVersion[1]: getting database version
[INFO ] 2022-03-28 15:35:40.018 [main] DataProviderHibernate - getDatabaseVersion[2]: got result PostgreSQL 14.2, compiled by Visual C++ build 1914, 64-bit
[INFO ] 2022-03-28 15:38:05.210 [main] DataProviderHibernateTest - getEntriesCount test Success
[INFO ] 2022-03-28 15:38:06.991 [main] DataProviderHibernate - getEntriesCount[1]: getting entries count
[INFO ] 2022-03-28 15:38:07.126 [main] DataProviderHibernate - getEntriesCount[2]: got result 0
[INFO ] 2022-03-28 15:51:32.442 [main] DataProviderHibernateTest - getEntriesCount test Success
[INFO ] 2022-03-28 15:51:34.025 [main] DataProviderHibernate - getTables[1]: getting tables
[INFO ] 2022-03-28 15:51:34.146 [main] DataProviderHibernate - getTables[2]: got result users
[INFO ] 2022-03-28 16:20:13.510 [main] DataProviderHibernateTest - getEntriesCount test Success
[INFO ] 2022-03-28 16:20:13.887 [main] DataProviderHibernateTest - getEntriesCount test Success
[INFO ] 2022-03-28 16:20:13.893 [main] DataProviderHibernateTest - getDatabaseSize test Success
[INFO ] 2022-03-28 16:20:13.899 [main] DataProviderHibernateTest - getDatabaseVersion test Success
[INFO ] 2022-03-28 16:22:19.668 [main] DataProviderHibernateTest - getEntriesCount test Success
[INFO ] 2022-03-28 16:22:21.207 [main] DataProviderHibernate - getTables[1]: getting tables
[INFO ] 2022-03-28 16:22:21.330 [main] DataProviderHibernate - getTables[2]: got result users
[INFO ] 2022-03-28 16:22:21.339 [main] DataProviderHibernateTest - getEntriesCount test Success
[INFO ] 2022-03-28 16:22:21.340 [main] DataProviderHibernate - getEntriesCount[1]: getting entries count
[INFO ] 2022-03-28 16:22:21.345 [main] DataProviderHibernate - getEntriesCount[2]: got result 0
[INFO ] 2022-03-28 16:22:21.347 [main] DataProviderHibernateTest - getDatabaseSize test Success
[INFO ] 2022-03-28 16:22:21.347 [main] DataProviderHibernate - getDatabaseSize[1]: getting database size
[INFO ] 2022-03-28 16:22:21.450 [main] DataProviderHibernate - getDatabaseSize[2]: got result 8617 kB
[INFO ] 2022-03-28 16:22:21.453 [main] DataProviderHibernateTest - getDatabaseVersion test Success
[INFO ] 2022-03-28 16:22:21.453 [main] DataProviderHibernate - getDatabaseVersion[1]: getting database version
[INFO ] 2022-03-28 16:22:21.455 [main] DataProviderHibernate - getDatabaseVersion[2]: got result PostgreSQL 14.2, compiled by Visual C++ build 1914, 64-bit
[INFO ] 2022-03-30 08:06:08.643 [main] DataProviderHibernateTest - getEntriesCount test Success
[INFO ] 2022-03-30 08:06:11.010 [main] DataProviderHibernate - getTables[1]: getting tables
[INFO ] 2022-03-30 08:06:11.802 [main] DataProviderHibernate - getTables[2]: got result users
[INFO ] 2022-03-30 08:06:11.816 [main] DataProviderHibernateTest - getEntriesCount test Success
[INFO ] 2022-03-30 08:06:11.817 [main] DataProviderHibernate - getEntriesCount[1]: getting entries count
[INFO ] 2022-03-30 08:06:11.908 [main] DataProviderHibernate - getEntriesCount[2]: got result 0
[INFO ] 2022-03-30 08:06:11.911 [main] DataProviderHibernateTest - getDatabaseSize test Success
[INFO ] 2022-03-30 08:06:11.911 [main] DataProviderHibernate - getDatabaseSize[1]: getting database size
[INFO ] 2022-03-30 08:06:13.831 [main] DataProviderHibernate - getDatabaseSize[2]: got result 8617 kB
[INFO ] 2022-03-30 08:06:13.834 [main] DataProviderHibernateTest - getDatabaseVersion test Success
[INFO ] 2022-03-30 08:06:13.835 [main] DataProviderHibernate - getDatabaseVersion[1]: getting database version
[INFO ] 2022-03-30 08:06:13.837 [main] DataProviderHibernate - getDatabaseVersion[2]: got result PostgreSQL 14.2, compiled by Visual C++ build 1914, 64-bit
[INFO ] 2022-03-30 08:29:05.067 [main] DataProviderHibernateTest - getEntriesCount test Success
[INFO ] 2022-03-30 08:29:06.641 [main] DataProviderHibernate - getTables[1]: getting tables
[INFO ] 2022-03-30 08:29:06.800 [main] DataProviderHibernate - getTables[2]: got result users
[INFO ] 2022-03-30 08:29:06.810 [main] DataProviderHibernateTest - getEntriesCount test Success
[INFO ] 2022-03-30 08:29:06.811 [main] DataProviderHibernate - getEntriesCount[1]: getting entries count
[INFO ] 2022-03-30 08:29:06.814 [main] DataProviderHibernate - getEntriesCount[2]: got result 0
[INFO ] 2022-03-30 08:29:06.818 [main] DataProviderHibernateTest - getDatabaseSize test Success
[INFO ] 2022-03-30 08:29:06.818 [main] DataProviderHibernate - getDatabaseSize[1]: getting database size
[INFO ] 2022-03-30 08:29:06.869 [main] DataProviderHibernate - getDatabaseSize[2]: got result 8617 kB
[INFO ] 2022-03-30 08:29:06.872 [main] DataProviderHibernateTest - getDatabaseVersion test Success
[INFO ] 2022-03-30 08:29:06.873 [main] DataProviderHibernate - getDatabaseVersion[1]: getting database version
[INFO ] 2022-03-30 08:29:06.876 [main] DataProviderHibernate - getDatabaseVersion[2]: got result PostgreSQL 14.2, compiled by Visual C++ build 1914, 64-bit
[INFO ] 2022-04-12 19:49:15.597 [main] DataProviderHibernateTest - saveTestEntity test Success
[INFO ] 2022-04-12 19:49:17.356 [main] DataProviderHibernate - saveTestEntity[1]: adding entity TestEntity{id=0, name='Joker', desc='Favorite card', dateCreated=Tue Apr 12 19:49:17 MSK 2022, check=true}
[ERROR] 2022-04-12 19:49:17.430 [main] DataProviderHibernate - saveTestEntity: Unknown entity: ru.sfedu.log4jproject.model.beans.TestEntity
[INFO ] 2022-04-12 19:50:20.771 [main] DataProviderHibernateTest - saveTestEntity test Success
[INFO ] 2022-04-12 19:50:22.399 [main] DataProviderHibernate - saveTestEntity[1]: adding entity TestEntity{id=0, name='Joker', desc='Favorite card', dateCreated=Tue Apr 12 19:50:22 MSK 2022, check=true}
[ERROR] 2022-04-12 19:50:22.480 [main] DataProviderHibernate - saveTestEntity: Unknown entity: ru.sfedu.log4jproject.model.beans.TestEntity
[INFO ] 2022-04-12 19:52:30.086 [main] DataProviderHibernateTest - saveTestEntity test Success
[INFO ] 2022-04-12 19:52:31.565 [main] DataProviderHibernate - saveTestEntity[1]: adding entity TestEntity{id=0, name='Joker', desc='Favorite card', dateCreated=Tue Apr 12 19:52:31 MSK 2022, check=true}
[ERROR] 2022-04-12 19:52:31.646 [main] DataProviderHibernate - saveTestEntity: Unknown entity: ru.sfedu.log4jproject.model.beans.TestEntity
[INFO ] 2022-04-12 19:57:34.690 [main] DataProviderHibernateTest - saveTestEntity test Success
[INFO ] 2022-04-12 19:57:36.562 [main] DataProviderHibernate - saveTestEntity[1]: adding entity TestEntity{id=0, name='Joker', desc='Favorite card', dateCreated=Tue Apr 12 19:57:36 MSK 2022, check=true}
[ERROR] 2022-04-12 19:57:36.654 [main] DataProviderHibernate - saveTestEntity: Unknown entity: ru.sfedu.log4jproject.model.beans.TestEntity
[INFO ] 2022-04-12 19:57:44.820 [main] DataProviderHibernateTest - saveTestEntity test Success
[INFO ] 2022-04-12 19:57:46.442 [main] DataProviderHibernate - saveTestEntity[1]: adding entity TestEntity{id=0, name='Joker', desc='Favorite card', dateCreated=Tue Apr 12 19:57:46 MSK 2022, check=true}
[ERROR] 2022-04-12 19:57:46.545 [main] DataProviderHibernate - saveTestEntity: Unknown entity: ru.sfedu.log4jproject.model.beans.TestEntity
[INFO ] 2022-04-12 20:04:17.184 [main] DataProviderHibernateTest - saveTestEntity test Success
[INFO ] 2022-04-12 20:04:18.738 [main] DataProviderHibernate - saveTestEntity[1]: adding entity TestEntity{id=0, name='Joker', desc='Favorite card', dateCreated=2022-04-12, check=true}
[ERROR] 2022-04-12 20:04:18.812 [main] DataProviderHibernate - saveTestEntity: Unknown entity: ru.sfedu.log4jproject.model.beans.TestEntity
[INFO ] 2022-04-12 20:12:32.777 [main] DataProviderHibernateTest - saveTestEntity test Success
[INFO ] 2022-04-12 20:12:34.384 [main] DataProviderHibernate - saveTestEntity[1]: adding entity TestEntity{id=0, name='Joker', desc='Favorite card', dateCreated=2022-04-12, check=true}
[ERROR] 2022-04-12 20:12:34.466 [main] DataProviderHibernate - saveTestEntity: Unknown entity: ru.sfedu.log4jproject.model.beans.TestEntity, [org.hibernate.metamodel.internal.MetamodelImpl.entityPersister(MetamodelImpl.java:710), org.hibernate.internal.SessionImpl.getEntityPersister(SessionImpl.java:1648), org.hibernate.event.internal.AbstractSaveEventListener.saveWithGeneratedId(AbstractSaveEventListener.java:114), org.hibernate.event.internal.DefaultSaveOrUpdateEventListener.saveWithGeneratedOrRequestedId(DefaultSaveOrUpdateEventListener.java:194), org.hibernate.event.internal.DefaultSaveEventListener.saveWithGeneratedOrRequestedId(DefaultSaveEventListener.java:38), org.hibernate.event.internal.DefaultSaveOrUpdateEventListener.entityIsTransient(DefaultSaveOrUpdateEventListener.java:179), org.hibernate.event.internal.DefaultSaveEventListener.performSaveOrUpdate(DefaultSaveEventListener.java:32), org.hibernate.event.internal.DefaultSaveOrUpdateEventListener.onSaveOrUpdate(DefaultSaveOrUpdateEventListener.java:75), org.hibernate.event.service.internal.EventListenerGroupImpl.fireEventOnEachListener(EventListenerGroupImpl.java:107), org.hibernate.internal.SessionImpl.fireSave(SessionImpl.java:676), org.hibernate.internal.SessionImpl.save(SessionImpl.java:669), org.hibernate.internal.SessionImpl.save(SessionImpl.java:664), ru.sfedu.log4jproject.api.DataProviderHibernate.saveTestEntity(DataProviderHibernate.java:78), ru.sfedu.log4jproject.api.DataProviderHibernateTest.saveTestEntity(DataProviderHibernateTest.java:72), java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method), java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78), java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43), java.base/java.lang.reflect.Method.invoke(Method.java:567), org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:725), org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60), org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131), org.junit.jupiter.engine.extension.TimeoutExtension.intercept(TimeoutExtension.java:149), org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestableMethod(TimeoutExtension.java:140), org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestMethod(TimeoutExtension.java:84), org.junit.jupiter.engine.execution.ExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0(ExecutableInvoker.java:115), org.junit.jupiter.engine.execution.ExecutableInvoker.lambda$invoke$0(ExecutableInvoker.java:105), org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106), org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed(InvocationInterceptorChain.java:64), org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke(InvocationInterceptorChain.java:45), org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke(InvocationInterceptorChain.java:37), org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:104), org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:98), org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$7(TestMethodTestDescriptor.java:214), org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73), org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod(TestMethodTestDescriptor.java:210), org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:135), org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:66), org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:151), org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73), org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141), org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137), org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139), org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73), org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138), org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95), java.base/java.util.ArrayList.forEach(ArrayList.java:1511), org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41), org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155), org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73), org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141), org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137), org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139), org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73), org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138), org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95), java.base/java.util.ArrayList.forEach(ArrayList.java:1511), org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41), org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155), org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73), org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141), org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137), org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139), org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73), org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138), org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95), org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:35), org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57), org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:54), org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:107), org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:88), org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:54), org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:67), org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:52), org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:114), org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:86), org.junit.platform.launcher.core.DefaultLauncherSession$DelegatingLauncher.execute(DefaultLauncherSession.java:86), org.junit.platform.launcher.core.SessionPerRequestLauncher.execute(SessionPerRequestLauncher.java:53), com.intellij.junit5.JUnit5IdeaTestRunner.startRunnerWithArgs(JUnit5IdeaTestRunner.java:71), com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38), com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11), com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35), com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235), com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)]
[INFO ] 2022-04-12 20:15:15.919 [main] DataProviderHibernateTest - saveTestEntity test Success
[INFO ] 2022-04-12 20:15:17.674 [main] DataProviderHibernate - saveTestEntity[1]: adding entity TestEntity{id=0, name='Joker', desc='Favorite card', dateCreated=2022-04-12, check=true}
[ERROR] 2022-04-12 20:15:17.738 [main] DataProviderHibernate - saveTestEntity: Unknown entity: ru.sfedu.log4jproject.model.beans.TestEntity
[INFO ] 2022-04-12 20:16:36.630 [main] DataProviderHibernateTest - saveTestEntity test Success
[INFO ] 2022-04-12 20:16:38.165 [main] DataProviderHibernate - saveTestEntity[1]: adding entity TestEntity{id=0, name='Joker', desc='Favorite card', dateCreated=2022-04-12, check=true}
[ERROR] 2022-04-12 20:16:38.248 [main] DataProviderHibernate - saveTestEntity: Unknown entity: ru.sfedu.log4jproject.model.beans.TestEntity
[INFO ] 2022-04-12 20:17:39.296 [main] DataProviderHibernateTest - saveTestEntity test Success
[INFO ] 2022-04-12 20:17:40.885 [main] DataProviderHibernate - saveTestEntity[1]: adding entity TestEntity{id=0, name='Joker', desc='Favorite card', dateCreated=2022-04-12, check=true}
[ERROR] 2022-04-12 20:17:40.971 [main] DataProviderHibernate - saveTestEntity: Unknown entity: ru.sfedu.log4jproject.model.beans.TestEntity
[INFO ] 2022-04-12 20:21:00.102 [main] DataProviderHibernateTest - saveTestEntity test Success
[INFO ] 2022-04-12 20:21:01.634 [main] DataProviderHibernate - saveTestEntity[1]: adding entity TestEntity{id=0, name='Joker', desc='Favorite card', dateCreated=2022-04-12, check=true}
[ERROR] 2022-04-12 20:21:01.727 [main] DataProviderHibernate - saveTestEntity: Unknown entity: ru.sfedu.log4jproject.model.beans.TestEntity
[INFO ] 2022-04-12 20:31:26.514 [main] DataProviderHibernateTest - saveTestEntity test Success
[INFO ] 2022-04-12 20:31:28.064 [main] DataProviderHibernate - saveTestEntity[1]: adding entity TestEntity{id=0, name='Joker', desc='Favorite card', dateCreated=2022-04-12, check=true}
[ERROR] 2022-04-12 20:31:28.139 [main] DataProviderHibernate - saveTestEntity: Unknown entity: ru.sfedu.log4jproject.model.beans.TestEntity
[INFO ] 2022-04-12 20:37:44.508 [main] DataProviderHibernateTest - saveTestEntity test Success
[INFO ] 2022-04-12 20:37:46.075 [main] DataProviderHibernate - saveTestEntity[1]: adding entity TestEntity{id=0, name='Joker', desc='Favorite card', dateCreated=2022-04-12, check=true}
[ERROR] 2022-04-12 20:37:46.142 [main] DataProviderHibernate - saveTestEntity: Unknown entity: ru.sfedu.log4jproject.model.beans.TestEntity
[INFO ] 2022-04-12 20:44:53.191 [main] DataProviderHibernateTest - saveTestEntity test Success
[INFO ] 2022-04-12 20:44:54.705 [main] DataProviderHibernate - saveTestEntity[1]: adding entity TestEntity{id=0, name='Joker', desc='Favorite card', dateCreated=2022-04-12, check=true}
[ERROR] 2022-04-12 20:44:54.773 [main] DataProviderHibernate - saveTestEntity: Unknown entity: ru.sfedu.log4jproject.model.beans.TestEntity
[INFO ] 2022-04-25 21:23:56.239 [main] DataProviderHibernateTest - saveTestEntity test Success
[INFO ] 2022-04-25 21:23:57.890 [main] DataProviderHibernate - saveTestEntity[1]: adding entity TestEntity{id=0, name='Joker', desc='Favorite card', dateCreated=2022-04-25, check=true}
[ERROR] 2022-04-25 21:23:57.967 [main] DataProviderHibernate - saveTestEntity: Unknown entity: ru.sfedu.log4jproject.model.beans.TestEntity
[INFO ] 2022-04-25 21:39:42.899 [main] DataProviderHibernateTest - saveTestEntity test Success
[INFO ] 2022-04-25 21:39:44.567 [main] DataProviderHibernate - saveTestEntity[1]: adding entity TestEntity{id=0, name='Joker', desc='Favorite card', dateCreated=2022-04-25, check=true}
[ERROR] 2022-04-25 21:39:44.646 [main] DataProviderHibernate - saveTestEntity: Unknown entity: ru.sfedu.log4jproject.model.beans.TestEntity
[INFO ] 2022-04-25 21:56:24.342 [main] DataProviderHibernateTest - getEntriesCount test Success
[INFO ] 2022-04-25 21:56:25.811 [main] DataProviderHibernate - getTables[1]: getting tables
[INFO ] 2022-04-25 21:56:26.072 [main] DataProviderHibernate - getTables[2]: got result test
[INFO ] 2022-04-25 22:32:58.155 [main] DataProviderHibernateTest - saveTestEntity test Success
[INFO ] 2022-04-25 22:32:59.646 [main] DataProviderHibernate - saveTestEntity[1]: adding entity TestEntity{id=0, name='Joker', desc='Favorite card', dateCreated=2022-04-25, check=true}
[ERROR] 2022-04-25 22:32:59.723 [main] DataProviderHibernate - saveTestEntity: Unknown entity: ru.sfedu.log4jproject.model.beans.TestEntity
[INFO ] 2022-04-25 22:36:16.198 [main] DataProviderHibernateTest - saveTestEntity test Success
[INFO ] 2022-04-25 22:36:17.791 [main] DataProviderHibernate - saveTestEntity[1]: adding entity TestEntity{id=0, name='Joker', desc='Favorite card', dateCreated=2022-04-25, check=true}
[ERROR] 2022-04-25 22:36:17.867 [main] DataProviderHibernate - saveTestEntity: Unknown entity: ru.sfedu.log4jproject.model.beans.TestEntity
[INFO ] 2022-04-25 22:41:52.576 [main] DataProviderHibernateTest - saveTestEntity test Success
[INFO ] 2022-04-25 22:41:54.108 [main] DataProviderHibernate - saveTestEntity[1]: adding entity TestEntity{id=0, name='Joker', desc='Favorite card', dateCreated=2022-04-25, check=true}
[ERROR] 2022-04-25 22:41:54.187 [main] DataProviderHibernate - saveTestEntity: Unknown entity: ru.sfedu.log4jproject.model.beans.TestEntity
[INFO ] 2022-04-25 22:45:16.087 [main] DataProviderHibernateTest - saveTestEntity test Success
[INFO ] 2022-04-25 22:45:17.638 [main] DataProviderHibernate - saveTestEntity[1]: adding entity TestEntity{id=0, name='Joker', desc='Favorite card', dateCreated=2022-04-25, check=true}
[ERROR] 2022-04-25 22:45:17.715 [main] DataProviderHibernate - saveTestEntity: Unknown entity: ru.sfedu.log4jproject.model.beans.TestEntity
[INFO ] 2022-04-25 22:54:24.497 [main] DataProviderHibernateTest - saveTestEntity test Success
[INFO ] 2022-04-25 22:54:26.163 [main] DataProviderHibernate - saveTestEntity[1]: adding entity TestEntity{id=0, name='Joker', desc='Favorite card', dateCreated=2022-04-25, check=true}
[ERROR] 2022-04-25 22:54:26.231 [main] DataProviderHibernate - saveTestEntity: Unknown entity: ru.sfedu.log4jproject.model.beans.TestEntity
[INFO ] 2022-04-26 11:28:28.485 [main] DataProviderHibernateTest - saveTestEntity test Success
[INFO ] 2022-04-26 11:28:30.935 [main] DataProviderHibernate - saveTestEntity[1]: adding entity TestEntity{id=0, name='Joker', desc='Favorite card', dateCreated=2022-04-26, check=true}
[INFO ] 2022-04-26 11:28:31.007 [main] DataProviderHibernate - saveTestEntity[2]: entity is successfully saved
[INFO ] 2022-04-26 14:55:41.459 [main] DataProviderHibernateTest - getEntriesCount test Success
[INFO ] 2022-04-26 14:55:43.434 [main] DataProviderHibernate - getEntriesCount[1]: getting entries count
[ERROR] 2022-04-26 14:55:43.537 [main] DataProviderHibernate - getEntriesCount[ERR]: org.hibernate.exception.SQLGrammarException: could not extract ResultSet
[ERROR] 2022-04-26 14:55:43.538 [main] DataProviderHibernateTest - getEntriesCount test Success - Failure
[INFO ] 2022-04-26 14:57:29.191 [main] DataProviderHibernateTest - getEntriesCount test Success
[INFO ] 2022-04-26 14:57:31.180 [main] DataProviderHibernate - getEntriesCount[1]: getting entries count
[INFO ] 2022-04-26 14:57:31.278 [main] DataProviderHibernate - getEntriesCount[2]: got result 0
[INFO ] 2022-04-26 14:58:20.914 [main] DataProviderHibernateTest - saveTestEntitySuccess test Success
[INFO ] 2022-04-26 14:58:22.930 [main] DataProviderHibernate - saveTestEntity[1]: adding entity TestEntity{id=0, name='Joker', desc='Favorite card', dateCreated=2022-04-26, check=true}
[INFO ] 2022-04-26 14:58:23.006 [main] DataProviderHibernate - saveTestEntity[2]: entity is successfully saved
[INFO ] 2022-04-26 15:00:51.610 [main] DataProviderHibernateTest - saveTestEntitySuccess test Success
[INFO ] 2022-04-26 15:00:53.685 [main] DataProviderHibernate - saveTestEntity[1]: adding entity TestEntity{id=0, name='Joker', desc='Favorite card', dateCreated=2022-04-26, check=true}
[INFO ] 2022-04-26 15:00:53.762 [main] DataProviderHibernate - saveTestEntity[2]: entity is successfully saved
[INFO ] 2022-04-26 15:07:46.190 [main] DataProviderHibernateTest - saveTestEntitySuccess test Success
[INFO ] 2022-04-26 15:07:48.361 [main] DataProviderHibernate - saveTestEntity[1]: adding entity TestEntity{id=0, name='Joker', desc='Favorite card', dateCreated=2022-04-26, check=true}
[INFO ] 2022-04-26 15:07:48.436 [main] DataProviderHibernate - saveTestEntity[2]: entity is successfully saved
[INFO ] 2022-04-26 15:08:50.124 [main] DataProviderHibernateTest - saveTestEntitySuccess test Success
[INFO ] 2022-04-26 15:08:52.393 [main] DataProviderHibernate - saveTestEntity[1]: adding entity TestEntity{id=0, name='Joker', desc='Favorite card', dateCreated=2022-04-26, check=true}
[INFO ] 2022-04-26 15:08:52.467 [main] DataProviderHibernate - saveTestEntity[2]: entity is successfully saved
[INFO ] 2022-04-26 15:10:15.416 [main] DataProviderHibernateTest - saveTestEntitySuccess test Success
[INFO ] 2022-04-26 15:10:17.285 [main] DataProviderHibernate - saveTestEntity[1]: adding entity TestEntity{id=0, name='Joker', desc='Favorite card', dateCreated=2022-04-26, check=true}
[INFO ] 2022-04-26 15:10:17.356 [main] DataProviderHibernate - saveTestEntity[2]: entity is successfully saved
[INFO ] 2022-04-26 15:11:16.561 [main] DataProviderHibernateTest - saveTestEntitySuccess test Success
[INFO ] 2022-04-26 15:11:18.579 [main] DataProviderHibernate - saveTestEntity[1]: adding entity TestEntity{id=0, name='Joker', desc='Favorite card', dateCreated=2022-04-26, check=true}
[INFO ] 2022-04-26 15:11:18.671 [main] DataProviderHibernate - saveTestEntity[2]: entity is successfully saved
[INFO ] 2022-04-26 15:19:21.256 [main] DataProviderHibernateTest - saveTestEntitySuccess test Success
[INFO ] 2022-04-26 15:19:23.109 [main] DataProviderHibernate - saveTestEntity[1]: adding entity TestEntity{id=0, name='Joker', desc='Favorite card', dateCreated=2022-04-26, check=true}
[INFO ] 2022-04-26 15:19:23.188 [main] DataProviderHibernate - saveTestEntity[2]: entity is successfully saved
[INFO ] 2022-04-26 15:21:13.600 [main] DataProviderHibernateTest - saveTestEntitySuccess test Success
[INFO ] 2022-04-26 15:21:15.505 [main] DataProviderHibernate - saveTestEntity[1]: adding entity TestEntity{id=0, name='Joker', desc='Favorite card', dateCreated=2022-04-26, check=true}
[INFO ] 2022-04-26 15:21:15.602 [main] DataProviderHibernate - saveTestEntity[2]: entity is successfully saved
[INFO ] 2022-04-26 15:25:02.675 [main] DataProviderHibernateTest - saveTestEntitySuccess test Success
[INFO ] 2022-04-26 15:25:04.557 [main] DataProviderHibernate - saveTestEntity[1]: adding entity TestEntity{id=0, name='Joker', desc='Favorite card', dateCreated=2022-04-26, check=true}
[INFO ] 2022-04-26 15:25:04.637 [main] DataProviderHibernate - saveTestEntity[2]: entity is successfully saved
[INFO ] 2022-04-26 15:29:54.351 [main] DataProviderHibernateTest - saveTestEntitySuccess test Success
[INFO ] 2022-04-26 15:29:56.302 [main] DataProviderHibernate - saveTestEntity[1]: adding entity ru.sfedu.log4jproject.model.beans.TestEntity@7302ff13
[INFO ] 2022-04-26 15:29:56.385 [main] DataProviderHibernate - saveTestEntity[2]: entity is successfully saved
[INFO ] 2022-04-26 15:36:52.345 [main] DataProviderHibernateTest - saveTestEntitySuccess test Success
[INFO ] 2022-04-26 15:36:54.143 [main] DataProviderHibernate - saveTestEntity[1]: adding entity TestEntity{id=0, name='Joker', desc='Favorite card', dateCreated=2022-04-26, check=true}
[INFO ] 2022-04-26 15:36:54.228 [main] DataProviderHibernate - saveTestEntity[2]: entity is successfully saved
[INFO ] 2022-04-26 15:40:38.984 [main] DataProviderHibernateTest - saveTestEntitySuccess test Success
[INFO ] 2022-04-26 15:40:40.777 [main] DataProviderHibernate - saveTestEntity[1]: adding entity TestEntity{id=0, name='Joker', desc='Favorite card', dateCreated=2022-04-26, check=true}
[INFO ] 2022-04-26 15:40:40.872 [main] DataProviderHibernate - saveTestEntity[2]: entity is successfully saved
